<table class="table">
<tbody>
	<tr>
		<td>
			<span class="date"><big><strong>2017</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20170009 --><big><strong>Preprints</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://arxiv.org/abs/1701.02643">Bayesian log-Gaussian Cox process regression with applications to fMRI meta-analysis</a>.
			</span><br />
			<span class="authors">
				Pantelis Samartsidis, Tor D. Wager, Lisa Feldman Barrett, Shir Atzil, Simon B. Eickhoff, Thomas E. Nichols, and Timothy D. Johnson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Samartsidis2017').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Samartsidis2017" class="abstract">
			A typical neuroimaging study will produce a 3D brain statistic image that summarises the evidence for activation during the experiment. However, for practical reasons those images are rarely published; instead, authors only report the (x,y,z) locations of local maxima in the statistic image. Neuroimaging meta-analyses use these foci from multiple studies to find areas of consistent activation across the human brain. However, current methods in the field do not account for study-specific characteristics. Therefore, we propose a fully Bayesian model based on log-Gaussian Cox processes that allows for the inclusion of study-specific variables. We present an efficient MCMC scheme based on the Hamiltonian Monte Carlo algorithm to simulate draws from the posterior. Computational time is significantly reduced through a parallel implementation using a graphical processing unit card. We evaluate the method on both simulated and real data.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20170009 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://arxiv.org/abs/1701.02643">Bayesian log-Gaussian Cox process regression with applications to fMRI meta-analysis</a>.
			</span><br />
			<span class="authors">
				Pantelis Samartsidis, Tor D. Wager, Lisa Feldman Barrett, Shir Atzil, Simon B. Eickhoff, Thomas E. Nichols, and Timothy D. Johnson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Samartsidis2017').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Samartsidis2017" class="abstract">
			A typical neuroimaging study will produce a 3D brain statistic image that summarises the evidence for activation during the experiment. However, for practical reasons those images are rarely published; instead, authors only report the (x,y,z) locations of local maxima in the statistic image. Neuroimaging meta-analyses use these foci from multiple studies to find areas of consistent activation across the human brain. However, current methods in the field do not account for study-specific characteristics. Therefore, we propose a fully Bayesian model based on log-Gaussian Cox processes that allows for the inclusion of study-specific variables. We present an efficient MCMC scheme based on the Hamiltonian Monte Carlo algorithm to simulate draws from the posterior. Computational time is significantly reduced through a parallel implementation using a graphical processing unit card. We evaluate the method on both simulated and real data.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20170008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://arxiv.org/abs/1701.02942">A defense of using resting state fMRI as null data for estimating false positive rates</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, Anders Eklund, and Hans Knutsson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Cognitive Neuroscience 00(2017)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Nichols2017').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Nichols2017" class="abstract">
			A recent Editorial by Slotnick (2017) reconsiders the findings of our paper on the accuracy of false positive rate control with cluster inference in fMRI (Eklund et al, 2016), in particular criticising our use of resting state fMRI data as a source for null data in the evaluation of task fMRI methods. We defend this use of resting fMRI data, as while there is much structure in this data, we argue it is representative of task data noise and as such analysis software should be able to accommodate this noise. We also discuss a potential problem with Slotnick&#39;s own method.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20170008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.nature.com/doifinder/10.1038/nn.4500">Best practices in data analysis and sharing in neuroimaging using MRI</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, Samir Das, Simon B. Eickhoff, Alan C. Evans, Tristan Glatard, Michael Hanke, Nikolaus Kriegeskorte, Michael P. Milham, Russell A. Poldrack, Jean-Baptiste Poline, Erika Proal, Bertrand Thirion, David C. {Van Essen}, Tonya White, and B. T. Thomas Yeo.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature Neuroscience 20(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Nichols2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Nichols2016" class="abstract">
			Neuroimaging enables rich noninvasive measurements of human brain activity, but translating such data into neuroscientific insights and clinical applications requires complex analyses and collaboration among a diverse array of researchers. The open science movement is reshaping scientific culture and addressing the challenges of transparency and reproducibility of research. To advance open science in neuroimaging the Organization for Human Brain Mapping created the Committee on Best Practice in Data Analysis and Sharing (COBIDAS), charged with creating a report that collects best practice recommendations from experts and the entire brain imaging community. The purpose of this work is to elaborate the principles of open and reproducible research for neuroimaging using Magnetic Resonance Imaging (MRI), and then distill these principles to specific research practices. Many elements of a study are so varied that practice cannot be prescribed, but for these areas we detail the information that must be reported to fully understand and potentially replicate a study. For other elements of a study, like statistical modelling where specific poor practices can be identified, and the emerging areas of data sharing and reproducibility, we detail both good practice and reporting standards. For each of seven areas of a study we provide tabular listing of over 100 items to help plan, execute, report and share research in the most transparent fashion. Whether for individual scientists, or for editors and reviewers, we hope these guidelines serve as a benchmark, to raise the standards of practice and reporting in neuroimaging using MRI.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20170008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1038/nrn.2016.167 http://www.ncbi.nlm.nih.gov/pubmed/28053326">Scanning the horizon: towards transparent and reproducible neuroimaging research.</a>.
			</span><br />
			<span class="authors">
				Russell A Poldrack, Chris I Baker, Joke Durnez, Krzysztof J Gorgolewski, Paul M Matthews, Marcus R Munaf{\`{o}}, Thomas E Nichols, Jean-Baptiste Poline, Edward Vul, and Tal Yarkoni.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature reviews. Neuroscience</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Poldrack2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Poldrack2016" class="abstract">
			Functional neuroimaging techniques have transformed our ability to probe the neurobiological basis of behaviour and are increasingly being applied by the wider neuroscience community. However, concerns have recently been raised that the conclusions that are drawn from some human neuroimaging studies are either spurious or not generalizable. Problems such as low statistical power, flexibility in data analysis, software errors and a lack of direct replication apply to many fields, but perhaps particularly to functional MRI. Here, we discuss these problems, outline current and suggested best practices, and describe how we think the field should evolve to produce the most meaningful and reliable answers to neuroscientific questions.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20170008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811916304128 http://www.ncbi.nlm.nih.gov/pubmed/27666385">Generalized reduced rank latent factor regression for high dimensional tensor fields, and neuroimaging-genetic applications.</a>.
			</span><br />
			<span class="authors">
				Chenyang Tao, Thomas E. Nichols, Xue Hua, Christopher R K Ching, Edmund T. Rolls, Paul M. Thompson, Jianfeng Feng, and {Alzheimer's Disease Neuroimaging Initiative}.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 144(Pt A)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Tao2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Tao2016" class="abstract">
			We propose a generalized reduced rank latent factor regression model (GRRLF) for the analysis of tensor field responses and high dimensional covariates. The model is motivated by the need from imaging-genetic studies to identify genetic variants that are associated with brain imaging phenotypes, often in the form of high dimensional tensor fields. GRRLF identifies from the structure in the data the effective dimensionality of the data, and then jointly performs dimension reduction of the covariates, dynamic identification of latent factors, and nonparametric estimation of both covariate and latent response fields. After accounting for the latent and covariate effects, GRLLF performs a nonparametric test on the remaining factor of interest. GRRLF provides a better factorization of the signals compared with common solutions, and is less susceptible to overfitting because it exploits the effective dimensionality. The generality and the flexibility of GRRLF also allow various statistical models to be handled in a unified framework and solutions can be efficiently computed. Within the field of neuroimaging, it improves the sensitivity for weak signals and is a promising alternative to existing approaches. The operation of the framework is demonstrated with both synthetic datasets and a real-world neuroimaging example in which the effects of a set of genes on the structure of the brain at the voxel level were measured, and the results compared favorably with those from existing approaches.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2016</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20160009 --><big><strong>Preprints</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://biorxiv.org/content/early/2016/05/20/054262 http://www.humanbrainmapping.org/COBIDASreport">Best Practices in Data Analysis and Sharing in Neuroimaging using MRI</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, Samir Das, Simon B. Eickhoff, Alan C. Evans, Tristan Glatard, Michael Hanke, Nikolaus Kriegeskorte, Michael P. Milham, Russell A Poldrack, Jean-Baptiste Poline, Erika Proal, Bertrand Thirion, David C. {Van Essen}, Tonya White, and B. T. Thomas Yeo.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_COBIDAS2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_COBIDAS2016" class="abstract">
			Neuroimaging enables rich noninvasive measurements of human brain activity, but translating such data into neuroscientific insights and clinical applications requires complex analyses and collaboration among a diverse array of researchers. The open science movement is reshaping scientific culture and addressing the challenges of transparency and reproducibility of research. To advance open science in neuroimaging the Organization for Human Brain Mapping created the Committee on Best Practice in Data Analysis and Sharing (COBIDAS), charged with creating a report that collects best practice recommendations from experts and the entire brain imaging community. The purpose of this work is to elaborate the principles of open and reproducible research for neuroimaging using Magnetic Resonance Imaging (MRI), and then distill these principles to specific research practices. Many elements of a study are so varied that practice cannot be prescribed, but for these areas we detail the information that must be reported to fully understand and potentially replicate a study. For other elements of a study, like statistical modelling where specific poor practices can be identified, and the emerging areas of data sharing and reproducibility, we detail both good practice and reporting standards. For each of seven areas of a study we provide tabular listing of over 100 items to help plan, execute, report and share research in the most transparent fashion. Whether for individual scientists, or for editors and reviewers, we hope these guidelines serve as a benchmark, to raise the standards of practice and reporting in neuroimaging using MRI.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160009 -->
			<span class="pubtitle">
				Power and sample size calculations for fMRI studies based on the prevalence of active peaks ..
			</span><br />
			<span class="authors">
				Joke Durnez, Jasper Degryse, Beatrijs Moerkerke, Ruth Seurinck, Vanessa Sochat, Russell A Poldrack, and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160009 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://biorxiv.org/lookup/doi/10.1101/048249">Minimal Data Needed for Valid \& Accurate Image-Based fMRI Meta-Analysis</a>.
			</span><br />
			<span class="authors">
				Camille Maumet and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Novel genetic loci associated with hippocampal volume.
			</span><br />
			<span class="authors">
				Mohammad Arfan Ikram, Derrek Hibar, Hieab Adams, Neda Jahanshad, Ganesh Chauhan, Jason Stein, Edith Hofer, Miguel Renter{\'{i}}a, Joshua Bis, Alejandro Arias-Vasquez, M Kamran Ikram, Sylvane Desrivieres, Meike Vernooij, Lucija Abramovic, Saud Alhusaini, Najaf Amin, Micael Andersson, Konstantinos Arfanakis, Benjamin Aribisala, Nicola Armstrong, Lavinia Athanasiu, Tomas Axelsson, Ashley Beecham, Alexa Beiser, Manon Bernard, Susan Blanton, Marc Bohlken, Marco Boks, Janita Bralten, Adam Brickman, Owen Carmichael, Mallar Chakravarty, Qiang Chen, Christopher Ching, Vincent Chouraki, Fabrice Crivello, Gabriel Cuellar-Partida, Anouk Braber, Nhat Trung Doan, Stefan Ehrlich, Sudheer Giddaluru, Aaron Goldman, Rebecca Gottesman, Oliver Grimm, Michael Griswold, Tulio Guadalupe, Boris Gutman, Johanna Hass, Unn Haukvik, David Hoehn, Avram Holmes, Martine Hoogman, Deborah Janowitz, Tianye Jia, Kjetil J{\o}rgensen, Nazanin Mirza-Schreiber, Dalia Kasperaviciute, Sungeun Kim, Marieke Klein, Bernd Kr{\"{a}}mer, Phil Lee, David Liewald, Lorna Lopez, Michelle Luciano, Christine Macare, Andre Marquand, Mar Matarin, Karen Mather, Manuel Mattheisen, David McKay, Yuri Milaneschi, Susana {Mu{\~{n}}oz Maniega}, Kwangsik Nho, Allison Nugent, Paul Nyquist, Loes Olde Loohuis, Jaap Oosterlaan, Martina Papmeyer, Lukas Pirpamer, Benno P{\"{u}}tz, Adaikalavan Ramasamy, Jennifer Richards, Shannon Risacher, Roberto Roiz-Santia{\~{n}}ez, Nanda Rommelse, Stefan Ropele, Emma Rose, Natalie Royle, Tatjana Rundek, Philipp S{\"{a}}mann, Claudia Satizabal, Lianne Schmaal, Andrew Schork, Li Shen, Jean Shin, Elena Shumskaya, Albert Smith, Emma Sprooten, Lachlan Strike, Alexander Teumer, Diana Tordesillas-Gutierrez, Roberto Toro, Daniah Trabzuni, Stella Trompet, Dhananjay Vaidya, Jeroen Van Grond, Sven Lee, Dennis Van Meer, Marjolein Donkelaar, Kristel Van Eijk, Theo Erp, Daan Van Rooij, Esther Walton, Lars Tjelta Westlye, Christopher Whelan, Beverly Windham, Anderson M. Winkler, Katharina Wittfeld, Girma Woldehawariat, Christiane Wolf, Thomas Wolfers, Lisa Yanek, Jingyun Yang, Alex Zijdenbos, Marcel Zwiers, Ingrid Agartz, Laura Almasy, David Ames, Philippe Amouyel, Ole Andreassen, Sampath Arepalli, Amelia Assareh, Sandra Barral, Mark Bastin, Diane Becker, James Becker, David Bennett, John Blangero, Hans Bokhoven, Dorret Boomsma, Henry Brodaty, Rachel Brouwer, Han Brunner, Randy Buckner, Jan Buitelaar, Kazima Bulayeva, Wiepke Cahn, Vince Calhoun, Dara Cannon, Gianpiero Cavalleri, Ching-Yu Cheng, Sven Cichon, Mark Cookson, Aiden Corvin, Benedicto Crespo-Facorro, Joanne Curran, Michael Czisch, Anders Dale, Gareth Davies, Anton De Craen, Philip De Jager, Eco Geus, Greig De Zubicaray, Ian Deary, St{\'{e}}phanie Debette, Charles DeCarli, Norman Delanty, Chantal Depondt, Anita DeStefano, Allissa Dillman, Gary Donohoe, Wayne Drevets, Srdjan Djurovic, Ravi Duggirala, Thomas Dyer, Christian Enzinger, Susanne Erk, Thomas Espeseth, Iryna Fedko, Guill{\'{e}}n Fern{\'{a}}ndez, Luigi Ferrucci, Simon E. Fisher, Debra Fleischman, Ian Ford, Myriam Fornage, Tatiana Foroud, Peter Fox, Clyde Francks, Masaki Fukunaga, J Gibbs, David Glahn, Randy Gollub, Harald G{\"{o}}ring, Robert Green, Oliver Gruber, Vilmundur Gudnason, Manuel Guelfi, Narelle Hansell, John Hardy, Catharina Hartman, Ryota Hashimoto, Katrin Hegenscheid, Andreas Heinz, Stephanie Hellard, Dena Hernandez, Dirk Heslenfeld, Beng-Choon Ho, Pieter Hoekstra, Wolfgang Hoffmann, Albert Hofman, Florian Holsboer, Georg Homuth, Norbert Hosten, Jouke Hottenga, Hilleke Hulshoff Pol, Masashi Ikeda, Clifford Jack, Mark Jenkinson, Robert Johnson, Erik Jonsson, J Wouter Jukema, Ryota Kanai, Iwona Kloszewska, Rene Kahn, David Knopman, Peter Kochunov, John Kwok, Stephen Lawrie, Herv{\'{e}} Lema{\^{i}}tre, Xinmin Liu, Dan Longo, Oscar Lopez, Simon Lovestone, Oliver Martinez, Jean-Luc Martinot, Venkata Mattay, Colm McDonald, Andrew Mcintosh, Francis McMahon, Katie McMahon, Patrizia Mecocci, Ingrid Melle, Andreas Meyer-Lindenberg, Sebastian Mohnke, Grant Montgomery, Derek Morris, Thomas Mosley, Thomas M{\"{u}}hleisen, Bertram M{\"{u}}ller-Myhsok, Michael Nalls, Matthias Nauck, Thomas Nichols, Wiro Niessen, Markus N{\"{o}}then, Lars Nyberg, Kazutaka Ohi, Rene Olvera, Roel Ophoff, Massimo Pandolfo, Tomas Paus, Zdenka Pausova, Brenda Penninx, G Bruce Pike, Steven Potkin, Bruce Psaty, Simone Reppermund, Marcella Rietschel, Joshua Roffman, Nina Romanczuk-Seiferth, Jerome Rotter, Mina Ryten, Ralph Sacco, Perminder Sachdev, Andrew Saykin, Reinhold Schmidt, Helena Schmidt, Peter Schofield, Sigurdur Sigursson, Andrew Simmons, Andrew Singleton, Sanjay Sisodiya, Colin Smith, Jordan Smoller, Hilkka Soininen, Vidar Steen, David Stott, Jess Sussmann, Anbupalam Thalamuthu, Arthur Toga, Bryan Traynor, Juan Troncoso, Magda Tsolaki, Christophe Tzourio, Andr{\'{e}} Uitterlinden, Maria C. {Vald{\'{e}}s Hern{\'{a}}ndez}, Dennis 't Ent, Marcel Van Brug, Aad Lugt, Nic Wee, Neeltje Haren, Marie-Jose Van Tol, Badri Vardarajan, Bruno Vellas, Dick Veltman, Henry V{\"{o}}lzke, Henrik Walter, Joanna Wardlaw, Thomas Wassink, Mike Weale, Daniel Weinberger, Michael Weiner, Wei Wen, Eric Westman, Tonya White, Tien Wong, Clinton Wright, Ronald Zielke, Alan Zonderman, Nicholas Martin, Cornelia Duijn, Margaret J. Wright, WT Longstreth Jr, Gunter Schumann, Hans Grabe, Barbara Franke, Lenore Launer, Sarah Medland, Sudha Seshadri, Paul Thompson, Arvin Saremi, Asta Haberg, and Matthew Huentelman.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature Communications</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/27922621 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC5139675">Sharing brain mapping statistical results with the neuroimaging data model.</a>.
			</span><br />
			<span class="authors">
				Camille Maumet, Tibor Auer, Alexander Bowring, Gang Chen, Samir Das, Guillaume Flandin, Satrajit Ghosh, Tristan Glatard, Krzysztof J Gorgolewski, Karl G Helmer, Mark Jenkinson, David B Keator, B Nolan Nichols, Jean-baptiste Poline, Richard Reynolds, Vanessa Sochat, Jessica Turner, and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Scientific data</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Maumet2016a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Maumet2016a" class="abstract">
			Only a tiny fraction of the data and metadata produced by an fMRI study is finally conveyed to the community. This lack of transparency not only hinders the reproducibility of neuroimaging results but also impairs future meta-analyses. In this work we introduce NIDM-Results, a format specification providing a machine-readable description of neuroimaging statistical results along with key image data summarising the experiment. NIDM-Results provides a unified representation of mass univariate analyses including a level of detail consistent with available best practices. This standardized representation allows authors to relay methods and results in a platform-independent regularized format that is not tied to a particular neuroimaging software package. Tools are available to export NIDM-Result graphs and associated files from the widely used SPM and FSL software packages, and the NeuroVault repository can import NIDM-Results archives. The specification is publically available at: http://nidm.nidash.org/specs/nidm-results.html.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.nature.com/doifinder/10.1038/nn.4398">Novel genetic loci underlying human intracranial volume identified through genome-wide association</a>.
			</span><br />
			<span class="authors">
				Hieab H H Adams, Derrek P Hibar, Vincent Chouraki, Jason L Stein, Paul A Nyquist, Miguel E Renter{\'{i}}a, Stella Trompet, Alejandro Arias-Vasquez, Sudha Seshadri, Sylvane Desrivi{\`{e}}res, Ashley H Beecham, Neda Jahanshad, Katharina Wittfeld, Sven J {Van der Lee}, Lucija Abramovic, Saud Alhusaini, Najaf Amin, Micael Andersson, Konstantinos Arfanakis, Benjamin S Aribisala, Nicola J Armstrong, Lavinia Athanasiu, Tomas Axelsson, Alexa Beiser, Manon Bernard, Joshua C Bis, Laura M E Blanken, Susan H Blanton, Marc M Bohlken, Marco P Boks, Janita Bralten, Adam M Brickman, Owen Carmichael, M Mallar Chakravarty, Ganesh Chauhan, Qiang Chen, Christopher R K Ching, Gabriel Cuellar-Partida, Anouk Den Braber, Nhat Trung Doan, Stefan Ehrlich, Irina Filippi, Tian Ge, Sudheer Giddaluru, Aaron L Goldman, Rebecca F Gottesman, Corina U Greven, Oliver Grimm, Michael E Griswold, Tulio Guadalupe, Johanna Hass, Unn K Haukvik, Saima Hilal, Edith Hofer, David Hoehn, Avram J Holmes, Martine Hoogman, Deborah Janowitz, Tianye Jia, Dalia Kasperaviciute, Sungeun Kim, Marieke Klein, Bernd Kraemer, Phil H Lee, Jiemin Liao, David C M Liewald, Lorna M Lopez, Michelle Luciano, Christine Macare, Andre Marquand, Mar Matarin, Karen A Mather, Manuel Mattheisen, Bernard Mazoyer, David R McKay, Rebekah McWhirter, Yuri Milaneschi, Nazanin Mirza-Schreiber, Ryan L Muetzel, Susana Mu{\~{n}}oz Maniega, Kwangsik Nho, Allison C Nugent, Loes M Olde Loohuis, Jaap Oosterlaan, Martina Papmeyer, Irene Pappa, Lukas Pirpamer, Sara Pudas, Benno P{\"{u}}tz, Kumar B Rajan, Adaikalavan Ramasamy, Jennifer S Richards, Shannon L Risacher, Roberto Roiz-Santia{\~{n}}ez, Nanda Rommelse, Emma J Rose, Natalie A Royle, Tatjana Rundek, Philipp G S{\"{a}}mann, Claudia L Satizabal, Lianne Schmaal, Andrew J Schork, Li Shen, Jean Shin, Elena Shumskaya, Albert V Smith, Emma Sprooten, Lachlan T Strike, Alexander Teumer, Russell Thomson, Diana Tordesillas-Gutierrez, Roberto Toro, Daniah Trabzuni, Dhananjay Vaidya, Jeroen {Van der Grond}, Dennis {Van der Meer}, Marjolein M J {Van Donkelaar}, Kristel R {Van Eijk}, Theo G M {Van Erp}, Daan {Van Rooij}, Esther Walton, Lars T Westlye, Christopher D Whelan, Beverly G Windham, Anderson M. Winkler, Girma Woldehawariat, Christiane Wolf, Thomas Wolfers, Bing Xu, Lisa R Yanek, Jingyun Yang, Alex Zijdenbos, Marcel P Zwiers, Ingrid Agartz, Neelum T Aggarwal, Laura Almasy, David Ames, Philippe Amouyel, Ole A Andreassen, Sampath Arepalli, Amelia A Assareh, Sandra Barral, Mark E Bastin, Diane M Becker, James T Becker, David A Bennett, John Blangero, Hans Bokhoven, Dorret I Boomsma, Henry Brodaty, Rachel M Brouwer, Han G Brunner, Randy L. Buckner, Jan K Buitelaar, Kazima B Bulayeva, Wiepke Cahn, Vince D Calhoun, Dara M Cannon, Gianpiero L Cavalleri, Christopher Chen, Ching-Yu Cheng, Sven Cichon, Mark R Cookson, Aiden Corvin, Benedicto Crespo-Facorro, Joanne E Curran, Michael Czisch, Anders M Dale, Gareth E Davies, Eco J C {De Geus}, Philip L {De Jager}, Greig I Zubicaray, Norman Delanty, Chantal Depondt, Anita L DeStefano, Allissa Dillman, Srdjan Djurovic, Gary Donohoe, Wayne C Drevets, Ravi Duggirala, Thomas D Dyer, Susanne Erk, Thomas Espeseth, Denis A Evans, Iryna O Fedko, Guill{\'{e}}n Fern{\'{a}}ndez, Luigi Ferrucci, Simon E Fisher, Debra A Fleischman, Ian Ford, Tatiana M Foroud, Peter T Fox, Clyde Francks, Masaki Fukunaga, J Raphael Gibbs, David C Glahn, Randy L Gollub, Harald H H G{\"{o}}ring, Hans J Grabe, Robert C Green, Oliver Gruber, Vilmundur Gudnason, Sebastian Guelfi, Narelle K Hansell, John Hardy, Catharina A Hartman, Ryota Hashimoto, Katrin Hegenscheid, Andreas Heinz, Stephanie {Le Hellard}, Dena G Hernandez, Dirk J Heslenfeld, Beng-Choon Ho, Pieter J Hoekstra, Wolfgang Hoffmann, Albert Hofman, Florian Holsboer, Georg Homuth, Norbert Hosten, Jouke-Jan Hottenga, Hilleke E Hulshoff Pol, Masashi Ikeda, M Kamran Arfan Ikram, Clifford R Jack Jr, Mark Jenkinson, Robert Johnson, Erik G J{\"{o}}nsson, J Wouter Jukema, Ren{\'{e}} S Kahn, Ryota Kanai, Iwona Kloszewska, David S Knopman, Peter Kochunov, John B Kwok, Stephen M Lawrie, Herv{\'{e}} Lema{\^{i}}tre, Xinmin Liu, Dan L Longo, W T Longstreth Jr, Oscar L Lopez, Simon Lovestone, Oliver Martinez, Jean-Luc Martinot, Venkata S Mattay, Colm McDonald, Andrew M McIntosh, Katie L McMahon, Francis J McMahon, Patrizia Mecocci, Ingrid Melle, Andreas Meyer-Lindenberg, Sebastian Mohnke, Grant W Montgomery, Derek W Morris, Thomas H Mosley, Thomas W M{\"{u}}hleisen, Bertram M{\"{u}}ller-Myhsok, Michael A Nalls, Matthias Nauck, Thomas E Nichols, Wiro J Niessen, Markus M N{\"{o}}then, Lars Nyberg, Kazutaka Ohi, Rene L Olvera, Roel A Ophoff, Massimo Pandolfo, Tomas Paus, Zdenka Pausova, Brenda W J H Penninx, G Bruce Pike, Steven G Potkin, Bruce M Psaty, Simone Reppermund, Marcella Rietschel, Joshua L Roffman, Nina Romanczuk-Seiferth, Jerome I Rotter, Mina Ryten, Ralph L Sacco, Perminder S Sachdev, Andrew J Saykin, Reinhold Schmidt, Peter R Schofield, Sigurdur Sigurdsson, Andy Simmons, Andrew Singleton, Sanjay M Sisodiya, Colin Smith, Jordan W Smoller, Hilkka Soininen, Velandai Srikanth, Vidar M Steen, David J Stott, Jessika E Sussmann, Anbupalam Thalamuthu, Henning Tiemeier, Arthur W Toga, Bryan J Traynor, Juan Troncoso, Jessica A Turner, Christophe Tzourio, Andre G Uitterlinden, Maria C Vald{\'{e}}s Hern{\'{a}}ndez, Marcel {Van der Brug}, Aad {Van der Lugt}, Nic J A {Van der Wee}, Cornelia M {Van Duijn}, Neeltje E M {Van Haren}, Dennis {Van 't Ent}, Marie-Jose {Van Tol}, Badri N Vardarajan, Dick J Veltman, Meike W Vernooij, Henry V{\"{o}}lzke, Henrik Walter, Joanna M Wardlaw, Thomas H Wassink, Michael E Weale, Daniel R Weinberger, Michael W Weiner, Wei Wen, Eric Westman, Tonya White, Tien Y Wong, Clinton B Wright, H Ronald Zielke, Alan B Zonderman, Ian J Deary, Charles DeCarli, Helena Schmidt, Nicholas G Martin, Anton J M {De Craen}, Margaret J. Wright, Lenore J Launer, Gunter Schumann, Myriam Fornage, Barbara Franke, St{\'{e}}phanie Debette, Sarah E Medland, and Paul M Thompson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature Neuroscience</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1016/j.neuroimage.2016.05.068">Faster permutation inference in brain imaging</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Gerard R. Ridgway, Gwena??lle Douaud, Thomas E. Nichols, and Stephen M. Smith.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Winkler2016b').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Winkler2016b" class="abstract">
			Permutation tests are increasingly being used as a reliable method for inference in neuroimaging analysis. However, they are computationally intensive. For small, non-imaging datasets, recomputing a model thousands of times is seldom a problem, but for large, complex models this can be prohibitively slow, even with the availability of inexpensive computing power. Here we exploit properties of statistics used with the general linear model (GLM) and their distributions to obtain accelerations irrespective of generic software or hardware improvements. We compare the following approaches: (i) performing a small number of permutations; (ii) estimating the p-value as a parameter of a negative binomial distribution; (iii) fitting a generalised Pareto distribution to the tail of the permutation distribution; (iv) computing p-values based on the expected moments of the permutation distribution, approximated from a gamma distribution; (v) direct fitting of a gamma distribution to the empirical permutation distribution; and (vi) permuting a reduced number of voxels, with completion of the remainder using low rank matrix theory. Using synthetic data we assessed the different methods in terms of their error rates, power, agreement with a reference result, and the risk of taking a different decision regarding the rejection of the null hypotheses (known as the resampling risk). We also conducted a re-analysis of a voxel-based morphometry study as a real-data example. All methods yielded exact error rates. Likewise, power was similar across methods. Resampling risk was higher for methods (i), (iii) and (v). For comparable resampling risks, the method in which no permutations are done (iv) was the absolute fastest. All methods produced visually similar maps for the real data, with stronger effects being detected in the family-wise error rate corrected maps by (iii) and (v), and generally similar to the results seen in the reference set. Overall, for uncorrected p-values, method (iv) was found the best as long as symmetric errors can be assumed. In all other settings, including for familywise error corrected p-values, we recommend the tail approximation (iii). The methods considered are freely available in the tool PALM ??? Permutation Analysis of Linear Models.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				Seeking optimal region-of-interest (ROI) single-value summary measures for fMRI studies in imaging genetics.
			</span><br />
			<span class="authors">
				Yunxia Tong, Qiang Chen, Thomas E. Nichols, Roberta Rasetti, Joseph H. Callicott, Karen F. Berman, Daniel R. Weinberger, and Venkata S. Mattay.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS ONE 11(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Tong2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Tong2016" class="abstract">
			A data-driven hypothesis-free genome-wide association (GWA) approach in imaging genetics studies allows screening the entire genome to discover novel genes that modulate brain structure, chemistry, and function. However, a whole brain voxel-wise analysis approach in such genome-wide based imaging genetic studies can be computationally intense and also likely has low statistical power since a stringent multiple comparisons correction is needed for searching over the entire genome and brain. In imaging genetics with functional magnetic resonance imaging (fMRI) phenotypes, since many experimental paradigms activate focal regions that can be pre-specified based on a priori knowledge, reducing the voxel-wise search to single-value summary measures within a priori ROIs could prove efficient and promising. The goal of this investigation is to evaluate the sensitivity and reliability of different single-value ROI summary measures and provide guidance in future work. Four different fMRI databases were tested and comparisons across different groups (patients with schizophrenia, their siblings, vs. normal control subjects; across genotype groups) were conducted. Our results show that four of these measures, particularly those that represent values from the top most-activated voxels within an ROI are more powerful at reliably detecting group differences and generating greater effect sizes than the others.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/80355/ http://www.ncbi.nlm.nih.gov/pubmed/27326542 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4978148">The brain imaging data structure, a format for organizing and describing outputs of neuroimaging experiments.</a>.
			</span><br />
			<span class="authors">
				Krzysztof J Gorgolewski, Tibor Auer, Vince D Calhoun, R Cameron Craddock, Samir Das, Eugene P Duff, Guillaume Flandin, Satrajit S Ghosh, Tristan Glatard, Yaroslav O Halchenko, Daniel A Handwerker, Michael Hanke, David Keator, Xiangrui Li, Zachary Michael, Camille Maumet, B Nolan Nichols, Thomas E Nichols, John Pellman, Jean-Baptiste Poline, Ariel Rokem, Gunnar Schaefer, Vanessa Sochat, William Triplett, Jessica A Turner, Ga{\"{e}}l Varoquaux, and Russell A Poldrack.
			</span><br />
			<span class="venuetype"></span><span class="venue">Scientific data</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_GorgolewskiKrzysztofJ.AuerTiborCalhounVinceD.CraddockR.CameronDasSamirDuffEugeneP.FlandinGuillaumeGhoshSatrajitS.GlatardTristanHalchenkoYaroslavOHandwerkerDanielA.HankeMichaelKeatorDavidLiXiangru2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_GorgolewskiKrzysztofJ.AuerTiborCalhounVinceD.CraddockR.CameronDasSamirDuffEugeneP.FlandinGuillaumeGhoshSatrajitS.GlatardTristanHalchenkoYaroslavOHandwerkerDanielA.HankeMichaelKeatorDavidLiXiangru2016" class="abstract">
			The development of magnetic resonance imaging (MRI) techniques has defined modern neuroimaging. Since its inception, tens of thousands of studies using techniques such as functional MRI and diffusion weighted imaging have allowed for the non-invasive study of the brain. Despite the fact that MRI is routinely used to obtain data for neuroscience research, there has been no widely adopted standard for organizing and describing the data collected in an imaging experiment. This renders sharing and reusing data (within or between labs) difficult if not impossible and unnecessarily complicates the application of automatic pipelines and quality assurance protocols. To solve this problem, we have developed the Brain Imaging Data Structure (BIDS), a standard for organizing and describing MRI datasets. The BIDS standard uses file formats compatible with existing software, unifies the majority of practices already common in the field, and captures the metadata necessary for most common data processing operations.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/27503879">Correction for Eklund et al., Cluster failure: Why fMRI inferences for spatial extent have inflated false-positive rates.</a>.
			</span><br />
			<span class="authors">
				Anders Eklund, Thomas E. Nichols, and Hans Knutsson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings of the National Academy of Sciences of the United States of America</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://journal.frontiersin.org/article/10.3389/fninf.2016.00024">Exploring fMRI Results Space: 31 Variants of an fMRI Analysis in AFNI, FSL, and SPM</a>.
			</span><br />
			<span class="authors">
				Ruth Pauli, Alexander Bowring, Richard Reynolds, Gang Chen, Thomas E. Nichols, and Camille Maumet.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in Neuroinformatics 10(July)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1602413113 http://www.ncbi.nlm.nih.gov/pubmed/27357684 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4948312">Cluster failure: Why fMRI inferences for spatial extent have inflated false-positive rates.</a>.
			</span><br />
			<span class="authors">
				Anders Eklund, Thomas E Nichols, and Hans Knutsson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings of the National Academy of Sciences of the United States of America 113(28)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Eklund2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Eklund2016" class="abstract">
			The most widely used task functional magnetic resonance imaging (fMRI) analyses use parametric statistical methods that depend on a variety of assumptions. In this work, we use real resting-state data and a total of 3 million random task group analyses to compute empirical familywise error rates for the fMRI software packages SPM, FSL, and AFNI, as well as a nonparametric permutation method. For a nominal familywise error rate of 5{\%}, the parametric statistical methods are shown to be conservative for voxelwise inference and invalid for clusterwise inference. Our results suggest that the principal cause of the invalid cluster inferences is spatial autocorrelation functions that do not follow the assumed Gaussian shape. By comparison, the nonparametric permutation test is found to produce nominal results for voxelwise as well as clusterwise inference. These findings speak to the need of validating the statistical methods being used in the field of neuroimaging.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1038/nn.4228 10.1038/nn.4228 http://www.nature.com/neuro/journal/vaop/ncurrent/abs/nn.4228.html{\#}supplementary-information http://www.nature.com/doifinder/10.1038/nn.4228">Genetic influences on schizophrenia and subcortical brain volumes: large-scale proof of concept</a>.
			</span><br />
			<span class="authors">
				Barbara Franke, Jason L Stein, Stephan Ripke, Verneri Anttila, Derrek P Hibar, Kimm J E Hulzen, Alejandro Arias-Vasquez, Jordan W Smoller, Thomas E Nichols, Michael C Neale, Andrew M McIntosh, Phil Lee, Francis J McMahon, Andreas Meyer-Lindenberg, Manuel Mattheisen, Ole A Andreassen, Oliver Gruber, Perminder S Sachdev, Roberto Roiz-Santia{\~{n}}ez, Andrew J Saykin, Stefan Ehrlich, Karen A Mather, Jessica A Turner, Emanuel Schwarz, Anbupalam Thalamuthu, Yin Yao, Yvonne Y W Ho, Nicholas G Martin, Margaret J. Wright, Michael C O'Donovan, Paul M Thompson, Benjamin M Neale, Sarah E Medland, Patrick F Sullivan, Aiden Corvin, James T R Walters, Kai-How Farh, Peter A Holmans, Brendan Bulik-Sullivan, David A Collier, Hailiang Huang, Tune H Pers, Ingrid Agartz, Esben Agerbo, Margot Albus, Madeline Alexander, Farooq Amin, Silviu A Bacanu, Martin Begemann, Richard A Belliveau, Judit Bene, Sarah E Bergen, Elizabeth Bevilacqua, Tim B Bigdeli, Donald W Black, Richard Bruggeman, Nancy G Buccola, Randy L. Buckner, William F Byerley, Wiepke Cahn, Guiqing Cai, Murray J Cairns, Dominique Campion, Rita M Cantor, Vaughan J Carr, Noa Carrera, Stanley V Catts, Kimberley D Chambert, Raymond C K Chan, Eric Y H Chen, Ronald Y L Chen, Wei Cheng, Eric F C Cheung, Siow Ann Chong, C Robert Cloninger, David Cohen, Nadine Cohen, Paul Cormican, Nick Craddock, Benedicto Crespo-Facorro, James J Crowley, David Curtis, Michael Davidson, Kenneth L Davis, Franziska Degenhardt, Jurgen {Del Favero}, Lynn E DeLisi, Ditte Demontis, Dimitris Dikeos, Timothy Dinan, Srdjan Djurovic, Gary Donohoe, Elodie Drapeau, Jubao Duan, Frank Dudbridge, Peter Eichhammer, Johan Eriksson, Valentina Escott-Price, Laurent Essioux, Ayman H Fanous, Martilias S Farrell, Josef Frank, Lude Franke, Robert Freedman, Nelson B Freimer, Joseph I Friedman, Menachem Fromer, Giulio Genovese, Lyudmila Georgieva, Elliot S Gershon, Ina Giegling, Paola Giusti-Rodr{\'{i}}guez, Stephanie Godard, Jacqueline I Goldstein, Srihari Gopal, Jacob Gratten, Lieuwe Haan, Christian Hammer, Marian L Hamshere, Mark Hansen, Thomas Hansen, Vahram Haroutunian, Annette M Hartmann, Frans A Henskens, Stefan L Herms, Joel N Hirschhorn, Per Hoffmann, Andrea Hofman, Mads V Hollegaard, David M Hougaard, Masashi Ikeda, Inge Joa, Antonio Juli{\`{a}}, Anna K K{\"{a}}hler, Ren{\'{e}} S Kahn, Luba Kalaydjieva, Sena Karachanak-Yankova, Juha Karjalainen, David Kavanagh, Matthew C Keller, Brian J Kelly, James L Kennedy, Andrey Khrunin, Yunjung Kim, Janis Klovins, James A Knowles, Bettina Konte, Vaidutis Kucinskas, Zita Ausrele Kucinskiene, Hana Kuzelova-Ptackova, Claudine Laurent, S Hong Lee, Jimmy Lee Chee Keong, Sophie E Legge, Bernard Lerer, Miaoxin Li, Tao Li, Kung-Yee Liang, Jeffrey Lieberman, Svetlana Limborska, Jouko L{\"{o}}nnqvist, Carmel M Loughland, Jan Lubinski, Milan Macek, Patrik K E Magnusson, Brion S Maher, Wolfgang Maier, Jacques Mallet, Sara Marsal, Morten Mattingsdal, Robert W McCarley, Colm McDonald, Sandra Meier, Carin J Meijer, Bela Melegh, Ingrid Melle, Raquelle I Mesholam-Gately, Andres Metspalu, Patricia T Michie, Lili Milani, Vihra Milanova, Younes Mokrab, Derek W Morris, Ole Mors, Bertram M{\"{u}}ller-Myhsok, Kieran C Murphy, Robin M Murray, Inez Myin-Germeys, Mari Nelis, Igor Nenadic, Deborah A Nertney, Gerald Nestadt, Kristin K Nicodemus, Liene Nikitina-Zake, Laura Nisenbaum, Annelie Nordin, Eadbhard O'Callaghan, Colm O'Dushlaine, F Anthony O'Neill, Sang-Yun Oh, Ann Olincy, Line Olsen, Jim {Van Os}, Christos Pantelis, George N Papadimitriou, Sergi Papiol, Elena Parkhomenko, Michele T Pato, Tiina Paunio, Diana O Perkins, Olli Pietil{\"{a}}inen, Jonathan Pimm, Andrew J Pocklington, John Powell, Alkes Price, Ann E Pulver, Shaun M Purcell, Digby Quested, Henrik B Rasmussen, Abraham Reichenberg, Mark A Reimers, Alexander L Richards, Joshua L Roffman, Panos Roussos, Douglas M Ruderfer, Veikko Salomaa, Alan R Sanders, Ulrich Schall, Christian R Schubert, Thomas G Schulze, Sibylle G Schwab, Edward M Scolnick, Rodney J Scott, Larry J Seidman, Jianxin Shi, Jeremy M Silverman, Kang Sim, Petr Slominsky, Hon-Cheong So, Erik S{\"{o}}derman, Chris C A Spencer, Eli A Stahl, Elisabeth Stogmann, Richard E Straub, Eric Strengman, Jana Strohmaier, T Scott Stroup, Mythily Subramaniam, Jaana Suvisaari, Dragan M Svrakic, Jin P Szatkiewicz, Srinivas Thirumalai, Draga Toncheva, Paul A Tooney, Juha Veijola, John Waddington, Dermot Walsh, Dai Wang, Qiang Wang, Bradley T Webb, Mark Weiser, Dieter B Wildenauer, Nigel M Williams, Stephanie Williams, Stephanie H Witt, Aaron R Wolen, Emily H M Wong, Brandon K Wormley, Jing Qin Wu, Hualin Simon Xi, Clement C Zai, Xuebin Zheng, Fritz Zimprich, Naomi R Wray, Peter M Visscher, Rolf Adolfsson, Douglas H R Blackwood, Anders D B{\o}rglum, Elvira Bramon, Joseph D Buxbaum, Sven Cichon, Ariel Darvasi, Enrico Domenici, Hannelore Ehrenreich, T{\~{o}}nu Esko, Pablo V Gejman, Michael Gill, Hugh Gurling, Christina M Hultman, Nakao Iwata, Assen V Jablensky, Erik G J{\"{o}}nsson, Kenneth S Kendler, George Kirov, Jo Knight, Todd Lencz, Douglas F Levinson, Qingqin S Li, Jianjun Liu, Anil K Malhotra, Steven A McCarroll, Andrew McQuillin, Jennifer L Moran, Preben B Mortensen, Bryan J Mowry, Markus M N{\"{o}}then, Roel A Ophoff, Michael J Owen, Aarno Palotie, Carlos N Pato, Tracey L Petryshen, Danielle Posthuma, Marcella Rietschel, Brien P Riley, Dan Rujescu, Pak C Sham, Pamela Sklar, David {St Clair}, Daniel R Weinberger, Jens R Wendland, Thomas Werge, Mark J Daly, Miguel E Renteria, Sylvane Desrivi{\`{e}}res, Neda Jahanshad, Roberto Toro, Katharina Wittfeld, Lucija Abramovic, Micael Andersson, Benjamin S Aribisala, Nicola J Armstrong, Manon Bernard, Marc M Bohlken, Marco P Boks, Janita Bralten, Andrew A Brown, M Mallar Chakravarty, Qiang Chen, Christopher R K Ching, Gabriel Cuellar-Partida, Anouk Braber, Sudheer Giddaluru, Aaron L Goldman, Oliver Grimm, Tulio Guadalupe, Johanna Hass, Girma Woldehawariat, Avram J Holmes, Martine Hoogman, Deborah Janowitz, Tianye Jia, Sungeun Kim, Marieke Klein, Bernd Kraemer, Loes M Olde Loohuis, Michelle Luciano, Christine Macare, Yuri Milaneschi, Kwangsik Nho, Martina Papmeyer, Adaikalavan Ramasamy, Shannon L Risacher, Emma J Rose, Alireza Salami, Philipp G S{\"{a}}mann, Lianne Schmaal, Andrew J Schork, Jean Shin, Lachlan T Strike, Alexander Teumer, Marjolein M J Donkelaar, Kristel R Eijk, Raymond K Walters, Lars T Westlye, Christopher D Whelan, Anderson M. Winkler, Marcel P Zwiers, Saud Alhusaini, Lavinia Athanasiu, Marina M H Hakobjan, Cecilie B Hartberg, Unn Haukvik, Angelien J G A M Heister, David H{\"{o}}hn, Dalia Kasperaviciute, David C M Liewald, Lorna M Lopez, Remco R R Makkinje, Mar Matarin, Marlies A M Naber, David R McKay, Margaret Needham, Allison C Nugent, Benno P{\"{u}}tz, Natalie A Royle, Li Shen, Emma Sprooten, Daniah Trabzuni, Saskia S L Marel, Esther Walton, Christiane Wolf, Laura Almasy, David Ames, Sampath Arepalli, Amelia A Assareh, Mark E Bastin, Henry Brodaty, Kazima B Bulayeva, Melanie A Carless, Joanne E Curran, Michael Czisch, Greig I Zubicaray, Allissa Dillman, Ravi Duggirala, Thomas D Dyer, Susanne Erk, Iryna O Fedko, Luigi Ferrucci, Tatiana M Foroud, Peter T Fox, Masaki Fukunaga, Raphael Gibbs, Harald H H G{\"{o}}ring, Robert C Green, Sebastian Guelfi, Narelle K Hansell, Catharina A Hartman, Katrin Hegenscheid, Andreas Heinz, Dena G Hernandez, Dirk J Heslenfeld, Pieter J Hoekstra, Florian Holsboer, Georg Homuth, Jouke-Jan Hottenga, Clifford R Jack, Mark Jenkinson, Robert Johnson, Ryota Kanai, Maria Keil, Jack W Kent, Peter Kochunov, John B Kwok, Stephen M Lawrie, Xinmin Liu, Dan L Longo, Katie L McMahon, Eva Meisenzahl, Sebastian Mohnke, Grant W Montgomery, Jeanette C Mostert, Thomas W M{\"{u}}hleisen, Michael A Nalls, Lars G Nilsson, Kazutaka Ohi, Rene L Olvera, Rocio Perez-Iglesias, G Bruce Pike, Steven G Potkin, Ivar Reinvang, Simone Reppermund, Nina Romanczuk-Seiferth, Glenn D Rosen, Knut Schnell, Peter R Schofield, Colin Smith, Vidar M Steen, Jessika E Sussmann, Arthur W Toga, Bryan Traynor, Juan Troncoso, Maria C Vald{\'{e}}s Hern{\'{a}}ndez, Dennis van 't Ent, Marcel Brug, Nic J A Wee, Marie-Jose Tol, Dick J Veltman, Thomas H Wassink, Eric Westman, Ronald H Zielke, Alan Zonderman, David G Ashbrook, Reinmar Hager, Lu Lu, Robert W Williams, Han G Brunner, Jan K Buitelaar, Vince D Calhoun, Gianpiero L Cavalleri, Anders M Dale, Gareth E Davies, Norman Delanty, Chantal Depondt, Wayne C Drevets, Thomas Espeseth, Randy L Gollub, Beng-Choon Ho, Wolfgang Hoffmann, Norbert Hosten, Stephanie LeHellard, Matthias Nauck, Lars Nyberg, Massimo Pandolfo, Brenda W J H Penninx, Sanjay M Sisodiya, Hans Bokhoven, Neeltje E M Haren, Henry V{\"{o}}lzke, Henrik Walter, Michael W Weiner, Wei Wen, Tonya White, John Blangero, Dorret I Boomsma, Rachel M Brouwer, Dara M Cannon, Mark R Cookson, Eco J C Geus, Ian J Deary, Guill{\'{e}}n Fern{\'{a}}ndez, Simon E Fisher, Clyde Francks, David C Glahn, Hans J Grabe, John Hardy, Ryota Hashimoto, Hilleke E {Hulshoff Pol}, Iwona Kloszewska, Simon Lovestone, Venkata S Mattay, Patrizia Mecocci, Tomas Paus, Zdenka Pausova, Mina Ryten, Andy Simmons, Andrew Singleton, Hilkka Soininen, Joanna M Wardlaw, Michael E Weale, Hieab H H Adams, Lenore J Launer, Stephan Seiler, Reinhold Schmidt, Ganesh Chauhan, Claudia L Satizabal, James T Becker, Lisa Yanek, Sven J Lee, Maritza Ebling, Bruce Fischl, W T Longstreth, Douglas Greve, Helena Schmidt, Paul Nyquist, Louis N Vinke, Cornelia M Duijn, Xue Luting, Bernard Mazoyer, Joshua C Bis, Vilmundur Gudnason, Sudha Seshadri, M Arfan Ikram, Gunter Schumann, Roberto Roiz-Santianez, Schizophrenia Working Group of the Psychiatric Genomics Consortium, and Enigma Consortium.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature neuroscience</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Franke2016a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Franke2016a" class="abstract">
			Schizophrenia is a devastating psychiatric illness with high heritability. Brain structure and function differ, on average, between people with schizophrenia and healthy individuals. As common genetic associations are emerging for both schizophrenia and brain imaging phenotypes, we can now use genome-wide data to investigate genetic overlap. Here we integrated results from common variant studies of schizophrenia (33,636 cases, 43,008 controls) and volumes of several (mainly subcortical) brain structures (11,840 subjects). We did not find evidence of genetic overlap between schizophrenia risk and subcortical volume measures either at the level of common variant genetic architecture or for single genetic markers. These results provide a proof of concept (albeit based on a limited set of structural brain measures) and define a roadmap for future studies investigating the genetic covariance between structural or functional brain phenotypes and risk for psychiatric disorders.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://doi.wiley.com/10.1002/hbm.23115 http://www.ncbi.nlm.nih.gov/pubmed/26848101">Non-parametric combination and related permutation tests for neuroimaging.</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Matthew A. Webster, Jonathan C. Brooks, Irene Tracey, Stephen M. Smith, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Human brain mapping</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Winkler2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Winkler2016" class="abstract">
			In this work, we show how permutation methods can be applied to combination analyses such as those that include multiple imaging modalities, multiple data acquisitions of the same modality, or simply multiple hypotheses on the same data. Using the well-known definition of union-intersection tests and closed testing procedures, we use synchronized permutations to correct for such multiplicity of tests, allowing flexibility to integrate imaging data with different spatial resolutions, surface and/or volume-based representations of the brain, including non-imaging data. For the problem of joint inference, we propose and evaluate a modification of the recently introduced non-parametric combination (NPC) methodology, such that instead of a two-phase algorithm and large data storage requirements, the inference can be performed in a single phase, with reasonable computational demands. The method compares favorably to classical multivariate tests (such as MANCOVA), even when the latter is assessed using permutations. We also evaluate, in the context of permutation tests, various combining methods that have been proposed in the past decades, and identify those that provide the best control over error rate and power across a range of situations. We show that one of these, the method of Tippett, provides a link between correction for the multiplicity of tests and their combination. Finally, we discuss how the correction can solve certain problems of multiple comparisons in one-way ANOVA designs, and how the combination is distinguished from conjunctions, even though both can be assessed using permutation tests. We also provide a common algorithm that accommodates combination and correction. Hum Brain Mapp, 2016.{\textcopyright} 2016 The Authors Human Brain Mapping Published by Wiley Periodicals, Inc.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.sciencedirect.com/science/article/pii/S1053811915010101 http://linkinghub.elsevier.com/retrieve/pii/S1053811915010101 http://www.ncbi.nlm.nih.gov/pubmed/26574120">Sharing the wealth: Neuroimaging data repositories.</a>.
			</span><br />
			<span class="authors">
				Simon B. Eickhoff, Thomas E. Nichols, John D. {Van Horn}, and Jessica A. Turner.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 124(Pt B)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.sciencedirect.com/science/article/pii/S1053811915003067 http://linkinghub.elsevier.com/retrieve/pii/S1053811915003067 http://www.ncbi.nlm.nih.gov/pubmed/25869863">NeuroVault.org: A repository for sharing unthresholded statistical maps, parcellations, and atlases of the human brain.</a>.
			</span><br />
			<span class="authors">
				Krzysztof J. Gorgolewski, Gael Varoquaux, Gabriel Rivera, Yannick Schwartz, Vanessa V. Sochat, Satrajit S. Ghosh, Camille Maumet, Thomas E. Nichols, Jean-Baptiste Poline, Tal Yarkoni, Daniel S. Margulies, and Russell a. Poldrack.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 124(Pt B)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Gorgolewski2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Gorgolewski2015" class="abstract">
			NeuroVault.org is dedicated to storing outputs of analyses in the form of statistical maps, parcellations and atlases, a unique strategy that contrasts with most neuroimaging repositories that store raw acquisition data or stereotaxic coordinates. Such maps are indispensable for performing meta-analyses, validating novel methodology, and deciding on precise outlines for regions of interest (ROIs). NeuroVault is open to maps derived from both healthy and clinical populations, as well as from various imaging modalities (sMRI, fMRI, EEG, MEG, PET, etc.). The repository uses modern web technologies such as interactive web-based visualization, cognitive decoding, and comparison with other maps to provide researchers with efficient, intuitive tools to improve the understanding of their results. Each dataset and map is assigned a permanent Universal Resource Locator (URL), and all of the data is accessible through a REST Application Programming Interface (API). Additionally, the repository supports the NIDM-Results standard and has the ability to parse outputs from popular FSL and SPM software packages to automatically extract relevant metadata. This ease of use, modern web-integration, and pioneering functionality holds promise to improve the workflow for making inferences about and sharing whole-brain statistical maps.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20160007 --><big><strong>Books & Chapters</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://link.springer.com/10.1007/978-1-4939-5611-1{\_}7">Statistical Analysis of fMRI Data</a>.
			</span><br />
			<span class="authors">
				Mark W. Woolrich, Christian F. Beckmann, Thomas E. Nichols, and Stephen M. Smith.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=incollection)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2015</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915004255 http://www.ncbi.nlm.nih.gov/pubmed/26025292 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4554832 http://www.bios.unc.edu/research/bias/documents/FastGWAS{\_}11{\_}20{\_}2014.pdf">FVGWAS : Fast Voxelwise Genome Wide Association Analysis of Large-scale Imaging Genetic Data</a>.
			</span><br />
			<span class="authors">
				Meiyan Huang, Thomas Nichols, Chao Huang, Yu Yang, Zhaohua Lu, Qianjing Feng, Rebecca C. Knickmeyer, Hongtu Zhu, Yang Yu, Zhaohua Lu, Rebecca C. Knickmeyer, Qianjing Feng, Hongtu Zhu, and {Alzheimer's Disease Neuroimaging Initiative}.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Huang2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Huang2015" class="abstract">
			More and more large-scale imaging genetic studies are being widely conducted to collect a rich set of imaging, genetic, and clinical data to detect putative genes for complexly inherited neuropsychiatric and neurodegenerative disorders. Several major big-data challenges arise from testing genome-wide (NC{\textgreater}12 million known variants) associations with signals at millions of locations (NV{\~{}}10(6)) in the brain from thousands of subjects (n{\~{}}10(3)). The aim of this paper is to develop a Fast Voxelwise Genome Wide Association analysiS (FVGWAS) framework to efficiently carry out whole-genome analyses of whole-brain data. FVGWAS consists of three components including a heteroscedastic linear model, a global sure independence screening (GSIS) procedure, and a detection procedure based on wild bootstrap methods. Specifically, for standard linear association, the computational complexity is O (nNVNC) for voxelwise genome wide association analysis (VGWAS) method compared with O ((NC+NV)n(2)) for FVGWAS. Simulation studies show that FVGWAS is an efficient method of searching sparse signals in an extremely large search space, while controlling for the family-wise error rate. Finally, we have successfully applied FVGWAS to a large-scale imaging genetic data analysis of ADNI data with 708 subjects, 193,275voxels in RAVENS maps, and 501,584 SNPs, and the total processing time was 203,645s for a single CPU. Our FVGWAS may be a valuable statistical toolbox for large-scale imaging genetic analysis as the field is rapidly advancing with ultra-high-resolution imaging and whole-genome sequencing.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://journal.frontiersin.org/Article/10.3389/fphys.2015.00351/abstract">An Ultra-High Field Magnetic Resonance Spectroscopy Study of Post Exercise Lactate, Glutamate and Glutamine Change in the Human Brain</a>.
			</span><br />
			<span class="authors">
				Andrea Dennis, Adam G. Thomas, Nancy B. Rawlings, Jamie Near, Thomas E. Nichols, Stuart Clare, Heidi Johansen-Berg, and Charlotte J. Stagg.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in Physiology 6(December)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/26413211 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4578221">Medial Demons Registration Localizes The Degree of Genetic Influence Over Subcortical Shape Variability: An N= 1480 Meta-Analysis.</a>.
			</span><br />
			<span class="authors">
				Boris A Gutman, Neda Jahanshad, Christopher R K Ching, Yalin Wang, Peter V Kochunov, Thomas E Nichols, and Paul M Thompson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings / IEEE International Symposium on Biomedical Imaging: from nano to macro. IEEE International Symposium on Biomedical Imaging</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Gutman2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Gutman2015" class="abstract">
			We present a multi-cohort shape heritability study, extending the fast spherical demons registration to subcortical shapes via medial modeling. A multi-channel demons registration based on vector spherical harmonics is applied to medial and curvature features, while controlling for metric distortion. We registered and compared seven subcortical structures of 1480 twins and siblings from the Queensland Twin Imaging Study and Human Connectome Project: Thalamus, Caudate, Putamen, Pallidum, Hippocampus, Amygdala, and Nucleus Accumbens. Radial distance and tensor-based morphometry (TBM) features were found to be highly heritable throughout the entire basal ganglia and limbic system. Surface maps reveal subtle variation in heritability across functionally distinct parts of each structure. Medial Demons reveals more significantly heritable regions than two previously described surface registration methods. This approach may help to prioritize features and measures for genome-wide association studies.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S2213158215001758">Classifying individuals at high-risk for psychosis based on functional brain activity during working memory processing</a>.
			</span><br />
			<span class="authors">
				Kerstin Bendfeldt, Renata Smieskova, Nikolaos Koutsouleris, Stefan Kl{\"{o}}ppel, Andr{\'{e}} Schmidt, Anna Walter, Fabienne Harrisberger, Johannes Wrege, Andor Simon, Bernd Taschler, Thomas Nichols, Anita Riecher-R{\"{o}}ssler, Undine E. Lang, Ernst-Wilhelm Radue, and Stefan Borgwardt.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage: Clinical</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915009611 http://www.ncbi.nlm.nih.gov/pubmed/26499807">The common genetic influence over processing speed and white matter microstructure: Evidence from the Old Order Amish and Human Connectome Projects.</a>.
			</span><br />
			<span class="authors">
				Peter Kochunov, Paul M. Thompson, Anderson M. Winkler, Mary Morrissey, Mao Fu, Thomas R. Coyle, Xiaoming Du, Florian Muellerklein, Anya Savransky, Christopher Gaudiot, Hemalatha Sampath, George Eskandar, Neda Jahanshad, Binish Patel, Laura Rowland, Thomas E. Nichols, Jeffrey R. O'Connell, Alan R. Shuldiner, Braxton D. Mitchell, and L. Elliot Hong.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Kochunov2016').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Kochunov2016" class="abstract">
			Speed with which brain performs information processing influences overall cognition and is dependent on the white matter fibers. To understand genetic influences on processing speed and white matter FA, we assessed processing speed and diffusion imaging fractional anisotropy (FA) in related individuals from two populations. Discovery analyses were performed in 146 individuals from large Old Order Amish (OOA) families and findings were replicated in 485 twins and siblings of the Human Connectome Project (HCP). The heritability of processing speed was h(2)=43{\%} and 49{\%} (both p{\textless}0.005), while the heritability of whole brain FA was h(2)=87{\%} and 88{\%} (both p{\textless}0.001), in the OOA and HCP, respectively. Whole brain FA was significantly correlated with processing speed in the two cohorts. Quantitative genetic analysis demonstrated a significant degree to which common genes influenced joint variation in FA and brain processing speed. These estimates suggested common sets of genes influencing variation in both phenotypes, consistent with the idea that common genetic variations contributing to white matter may also support their associated cognitive behavior.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://bayesian.org/BA/12638 http://projecteuclid.org/euclid.ba/1422884981">Searching Multiregression Dynamic Models of Resting-State fMRI Networks Using Integer Programming</a>.
			</span><br />
			<span class="authors">
				Lilia Costa, Jim Smith, Thomas Nichols, James Cussens, Eugene P. Duff, Tamar R. Makin, and Eugene P. Duff.
			</span><br />
			<span class="venuetype"></span><span class="venue">Bayesian Analysis 10(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Costa2013').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Costa2013" class="abstract">
			A Multiregression Dynamic Model (MDM) is a class of multivariate time series that represents various dynamic causal processes in a graphical way. One of the advantages of this class is that, in contrast to many other Dynamic Bayesian Networks, the hypothesised relationships accommodate conditional con- jugate inference. We demonstrate for the first time how straightforward it is to search over all possible connectivity networks with dynamically changing inten- sity of transmission to find the Maximum a Posteriori Probability (MAP) model within this class. This search method is made feasible by using a novel applica- tion of an Integer Programming algorithm. The efficacy of applying this particular class of dynamic models to this domain is shown and more specifically the com- putational efficiency of a corresponding search of 11-node Directed Acyclic Graph (DAG) model space. We proceed to show how diagnostic methods, analogous to those defined for static Bayesian Networks, can be used to suggest embellishment of the model class to extend the process of model selection. All methods are il- lustrated using simulated and real resting-state functional Magnetic Resonance Imaging (fMRI) data.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915010812 http://www.ncbi.nlm.nih.gov/pubmed/26658930">ENIGMA and the Individual: Predicting Factors that Affect the Brain in 35 Countries Worldwide.</a>.
			</span><br />
			<span class="authors">
				Paul M. Thompson, Ole A. Andreassen, Alejandro Arias-Vasquez, Carrie E. Bearden, Premika S. Boedhoe, Rachel M. Brouwer, Randy L. Buckner, Jan K. Buitelaar, Kazima B. Bulaeva, Dara M. Cannon, Ronald A. Cohen, Patricia J. Conrod, Anders M. Dale, Ian J. Deary, Emily L. Dennis, Marcel A. Reus, Sylvane Desrivieres, Danai Dima, Gary Donohoe, Simon E. Fisher, Jean-Paul Fouche, Clyde Francks, Sophia Frangou, Barbara Franke, Habib Ganjgahi, Hugh Garavan, David C. Glahn, Hans J. Grabe, Tulio Guadalupe, Boris A. Gutman, Ryota Hashimoto, Derrek P. Hibar, Dominic Holland, Martine Hoogman, Hilleke E. Hulshoff Pol, Norbert Hosten, Neda Jahanshad, Sinead Kelly, Peter Kochunov, William S. Kremen, Phil H. Lee, Scott Mackey, Nicholas G. Martin, Bernard Mazoyer, Colm McDonald, Sarah E. Medland, Rajendra A. Morey, Thomas E. Nichols, Tomas Paus, Zdenka Pausova, Lianne Schmaal, Gunter Schumann, Li Shen, Sanjay M. Sisodiya, Dirk J A Smit, Jordan W. Smoller, Dan J Stein, Jason L. Stein, Roberto Toro, Jessica A. Turner, Martijn P Heuvel, Odile A. Heuvel, Theo G M Erp, Daan Rooij, Dick J. Veltman, Henrik Walter, Yalin Wang, Joanna M. Wardlaw, Christopher D. Whelan, Margaret J. Wright, Jieping Ye, and {ENIGMA Consortium}.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Thompson2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Thompson2015" class="abstract">
			In this review, we discuss recent work by the ENIGMA Consortium (http://enigma.ini.usc.edu) - a global alliance of over 500 scientists spread across 200 institutions in 35 countries collectively analyzing brain imaging, clinical, and genetic data. Initially formed to detect genetic influences on brain measures, ENIGMA has grown to over 30 working groups studying 12 major brain diseases by pooling and comparing brain data. In some of the largest neuroimaging studies to date - of schizophrenia and major depression - ENIGMA has found replicable disease effects on the brain that are consistent worldwide, as well as factors that modulate disease effects. In partnership with other consortia including ADNI, CHARGE, IMAGEN and others, ENIGMA&#39;s genomic screens - now numbering over 30,000 MRI scans - have revealed at least 8 genetic loci that affect brain volumes. Downstream of gene findings, ENIGMA has revealed how these individual variants - and genetic variants in general - may affect both the brain and risk for a range of diseases. The ENIGMA consortium is discovering factors that consistently affect brain structure and function that will serve as future predictors linking individual brain scans and genomic data. It is generating vast pools of normative data on brain measures - from tens of thousands of people - that may help detect deviations from normal development or aging in specific groups of subjects. We discuss challenges and opportunities in applying these predictors to individual subjects and new cohorts, as well as lessons we have learned in ENIGMA&#39;s efforts so far.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915010721 http://www.ncbi.nlm.nih.gov/pubmed/26654786">Multi-modal characterization of rapid anterior hippocampal volume increase associated with aerobic exercise.</a>.
			</span><br />
			<span class="authors">
				Adam G. Thomas, Andrea Dennis, Nancy B. Rawlings, Charlotte J. Stagg, Lucy Matthews, Martyn Morris, Shannon H. Kolind, Sean Foxley, Mark Jenkinson, Thomas E. Nichols, Helen Dawes, Peter A. Bandettini, and Heidi Johansen-Berg.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Thomas2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Thomas2015" class="abstract">
			The hippocampus has been shown to demonstrate a remarkable degree of plasticity in response to a variety of tasks and experiences. For example, the size of the human hippocampus has been shown to increase in response to aerobic exercise. However, it is currently unknown what underlies these changes. Here we scanned sedentary, young to middle-aged human adults before and after a six-week exercise intervention using nine different neuroimaging measures of brain structure, vasculature, and diffusion. We then tested two different hypotheses regarding the nature of the underlying changes in the tissue. Surprisingly, we found no evidence of a vascular change as has been previously reported. Rather, the pattern of changes is better explained by an increase in myelination. Finally, we show that hippocampal volume increase is temporary, returning to baseline after an additional six weeks without aerobic exercise. This is the first demonstration of a change in hippocampal volume in early to middle adulthood suggesting that hippocampal volume is modulated by aerobic exercise throughout the lifespan rather than only in the presence of age related atrophy. It is also the first demonstration of hippocampal volume change over a period of only six weeks, suggesting that gross morphometric hippocampal plasticity occurs faster than previously thought.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1038/nn.4125 http://www.ncbi.nlm.nih.gov/pubmed/26414616 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4625579">A positive-negative mode of population covariation links brain connectivity, demographics and behavior.</a>.
			</span><br />
			<span class="authors">
				Stephen M Smith, Thomas E Nichols, Diego Vidaurre, Anderson M. Winkler, Timothy E J Behrens, Matthew F Glasser, Kamil Ugurbil, Deanna M Barch, David C. {Van Essen}, and Karla L Miller.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature neuroscience 18(11)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Smith2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Smith2015" class="abstract">
			We investigated the relationship between individual subjects&#39; functional connectomes and 280 behavioral and demographic measures in a single holistic multivariate analysis relating imaging to non-imaging data from 461 subjects in the Human Connectome Project. We identified one strong mode of population co-variation: subjects were predominantly spread along a single &#39;positive-negative&#39; axis linking lifestyle, demographic and psychometric measures to each other and to a specific pattern of brain connectivity.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S105381191500508X http://www.ncbi.nlm.nih.gov/pubmed/26074200 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4644991">Multi-level block permutation.</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Matthew a. Webster, Diego Vidaurre, Thomas E. Nichols, and Stephen M. Smith.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Winkler2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Winkler2015" class="abstract">
			Under weak and reasonable assumptions, mainly that data are exchangeable under the null hypothesis, permutation tests can provide exact control of false positives and allow the use of various non-standard statistics. There are, however, various common examples in which global exchangeability can be violated, including paired tests, tests that involve repeated measurements, tests in which subjects are relatives (members of pedigrees) - any dataset with known dependence among observations. In these cases, some permutations, if performed, would create data that would not possess the original dependence structure, and thus, should not be used to construct the reference (null) distribution. To allow permutation inference in such cases, we test the null hypothesis using only a subset of all otherwise possible permutations, i.e., using only the rearrangements of the data that respect exchangeability, thus retaining the original joint distribution unaltered. In a previous study, we defined exchangeability for blocks of data, as opposed to each datum individually, then allowing permutations to happen within block, or the blocks as a whole to be permuted. Here we extend that notion to allow blocks to be nested, in a hierarchical, multi-level definition. We do not explicitly model the degree of dependence between observations, only the lack of independence; the dependence is implicitly accounted for by the hierarchy and by the permutation scheme. The strategy is compatible with heteroscedasticity and variance groups, and can be used with permutations, sign flippings, or both combined. We evaluate the method for various dependence structures, apply it to real data from the Human Connectome Project (HCP) as an example application, show that false positives can be avoided in such cases, and provide a software implementation of the proposed approach.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1016/j.neuroimage.2015.03.065">Age differences in the brain mechanisms of good taste</a>.
			</span><br />
			<span class="authors">
				Edmund T Rolls, Michele B Kellerhals, and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915005194 http://www.ncbi.nlm.nih.gov/pubmed/26067343">A voxelation-corrected non-stationary 3D cluster-size test based on random field theory.</a>.
			</span><br />
			<span class="authors">
				Huanjie Li, Lisa D. Nickerson, Xuna Zhao, Thomas E. Nichols, and Jia-Hong Gao.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Li2015a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Li2015a" class="abstract">
			Cluster-size tests (CSTs) based on random field theory (RFT) are commonly adopted to identify significant differences in brain images. However, the use of RFT in CSTs rests on the assumption of uniform smoothness (stationarity). When images are non-stationary, CSTs based on RFT will likely lead to increased false positives in smooth regions and reduced power in rough regions. An adjustment to the cluster size according to the local smoothness at each voxel has been proposed for the standard test based on RFT to address non-stationarity, however, this technique requires images with a large degree of spatial smoothing, large degrees of freedom and high intensity thresholding. Recently, we proposed a voxelation-corrected 3D CST based on Gaussian random field theory that does not place constraints on the degree of spatial smoothness. However, this approach is only applicable to stationary images, requiring further modification to enable use for non-stationary images. In this study, we present modifications of this method to develop a voxelation-corrected non-stationary 3D CST based on RFT. Both simulated and real data were used to compare the voxelation-corrected non-stationary CST to the standard cluster-size adjusted non-stationary CST based on RFT and the voxelation-corrected stationary CST. We found that voxelation-corrected stationary CST is liberal for non-stationary images and the voxelation-corrected non-stationary CST performs better than cluster-size adjusted non-stationary CST based on RFT under low smoothness, low intensity threshold and low degrees of freedom.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915001512 http://www.ncbi.nlm.nih.gov/pubmed/25747917 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4387079">Heritability of fractional anisotropy in human white matter: a comparison of Human Connectome Project and ENIGMA-DTI data.</a>.
			</span><br />
			<span class="authors">
				Peter Kochunov, Neda Jahanshad, Daniel Marcus, Anderson M. Winkler, Emma Sprooten, Thomas E. Nichols, Susan N. Wright, L. Elliot Hong, Binish Patel, Timothy Behrens, Saad Jbabdi, Jesper Andersson, Christophe Lenglet, Essa Yacoub, Steen Moeller, Eddie Auerbach, Kamil Ugurbil, Stamatios N. Sotiropoulos, Rachel M. Brouwer, Bennett Landman, Herv{\'{e}} Lemaitre, Anouk Braber, Marcel P. Zwiers, Stuart Ritchie, Kimm Hulzen, Laura Almasy, Joanne Curran, Greig I. DeZubicaray, Ravi Duggirala, Peter Fox, Nicholas G. Martin, Katie L. McMahon, Braxton Mitchell, Rene L. Olvera, Charles Peterson, John Starr, Jessika Sussmann, Joanna Wardlaw, Margie Wright, Dorret I. Boomsma, Rene Kahn, Eco J.C. C Geus, Douglas E. Williamson, Ahmad Hariri, Dennis Ent, Mark E. Bastin, Andrew McIntosh, Ian J. Deary, Hilleke E. {Hulshoff Pol}, John Blangero, Paul M. Thompson, David C. Glahn, David C. {Van Essen}, and Dennis {van 't Ent}.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Kochunov2015b').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Kochunov2015b" class="abstract">
			The degree to which genetic factors influence brain connectivity is beginning to be understood. Large-scale efforts are underway to map the profile of genetic effects in various brain regions. The NIH-funded Human Connectome Project (HCP) is providing data valuable for analyzing the degree of genetic influence underlying brain connectivity revealed by state-of-the-art neuroimaging methods. We calculated the heritability of the fractional anisotropy (FA) measure derived from diffusion tensor imaging (DTI) reconstruction in 481 HCP subjects (194/287 M/F) consisting of 57/60 pairs of mono- and dizygotic twins, and 246 siblings. FA measurements were derived using (Enhancing NeuroImaging Genetics through Meta-Analysis) ENIGMA DTI protocols and heritability estimates were calculated using the SOLAR-Eclipse imaging genetic analysis package. We compared heritability estimates derived from HCP data to those publicly available through the ENIGMA-DTI consortium, which were pooled together from five-family based studies across the US, Europe, and Australia. FA measurements from the HCP cohort for eleven major white matter tracts were highly heritable (h(2)=0.53-0.90, p{\textless}10(-5)), and were significantly correlated with the joint-analytical estimates from the ENIGMA cohort on the tract and voxel-wise levels. The similarity in regional heritability suggests that the additive genetic contribution to white matter microstructure is consistent across populations and imaging acquisition parameters. It also suggests that the overarching genetic influence provides an opportunity to define a common genetic search space for future gene-discovery studies. Uniquely, the measurements of additive genetic contribution performed in this study can be repeated using online genetic analysis tools provided by the HCP ConnectomeDB web application.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811915001792 http://dx.doi.org/10.1016/j.neuroimage.2015.03.005 http://www.ncbi.nlm.nih.gov/pubmed/25812717 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4463976">Fast and powerful heritability inference for family-based neuroimaging studies.</a>.
			</span><br />
			<span class="authors">
				Habib Ganjgahi, Anderson M. Winkler, David C. Glahn, John Blangero, Peter Kochunov, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Ganjgahi2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Ganjgahi2015" class="abstract">
			Heritability estimation has become an important tool for imaging genetics studies. The large number of voxel- and vertex-wise measurements in imaging genetics studies presents a challenge both in terms of computational intensity and the need to account for elevated false positive risk because of the multiple testing problem. There is a gap in existing tools, as standard neuroimaging software cannot estimate heritability, and yet standard quantitative genetics tools cannot provide essential neuroimaging inferences, like family-wise error corrected voxel-wise or cluster-wise P-values. Moreover, available heritability tools rely on P-values that can be inaccurate with usual parametric inference methods. In this work we develop fast estimation and inference procedures for voxel-wise heritability, drawing on recent methodological results that simplify heritability likelihood computations (Blangero et al., 2013). We review the family of score and Wald tests and propose novel inference methods based on explained sum of squares of an auxiliary linear model. To address problems with inaccuracies with the standard results used to find P-values, we propose four different permutation schemes to allow semi-parametric inference (parametric likelihood-based estimation, non-parametric sampling distribution). In total, we evaluate 5 different significance tests for heritability, with either asymptotic parametric or permutation-based P-value computations. We identify a number of tests that are both computationally efficient and powerful, making them ideal candidates for heritability studies in the massive data setting. We illustrate our method on fractional anisotropy measures in 859 subjects from the Genetics of Brain Structure study.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.plos.org/10.1371/journal.pcbi.1004066">A Bayesian Model of Category-Specific Emotional Brain Responses</a>.
			</span><br />
			<span class="authors">
				Tor D. Wager, Jian Kang, Timothy D. Johnson, Thomas E. Nichols, Ajay B. Satpute, and Lisa Feldman Barrett.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLOS Computational Biology</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://journal.frontiersin.org/article/10.3389/fninf.2015.00008 http://www.ncbi.nlm.nih.gov/pubmed/25914639 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4392315">NeuroVault.org: a web-based repository for collecting and sharing unthresholded statistical maps of the human brain.</a>.
			</span><br />
			<span class="authors">
				Krzysztof J. Gorgolewski, Gael Varoquaux, Gabriel Rivera, Yannick Schwarz, Satrajit S. Ghosh, Camille Maumet, Vanessa V. Sochat, Thomas E. Nichols, Russell a. Poldrack, Jean-Baptiste Poline, Tal Yarkoni, and Daniel S. Margulies.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in neuroinformatics 9(April)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Gorgolewski2015a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Gorgolewski2015a" class="abstract">
			Here we present NeuroVault-a web based repository that allows researchers to store, share, visualize, and decode statistical maps of the human brain. NeuroVault is easy to use and employs modern web technologies to provide informative visualization of data without the need to install additional software. In addition, it leverages the power of the Neurosynth database to provide cognitive decoding of deposited maps. The data are exposed through a public REST API enabling other services and tools to take advantage of it. NeuroVault is a new resource for researchers interested in conducting meta- and coactivation analyses.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pnas.org/lookup/doi/10.1073/pnas.1415603112 http://www.ncbi.nlm.nih.gov/pubmed/25675487 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC4345618">Massively expedited genome-wide heritability analysis (MEGHA).</a>.
			</span><br />
			<span class="authors">
				Tian Ge, Thomas E. Nichols, Phil H. Lee, Avram J. Holmes, Joshua L. Roffman, Randy L. Buckner, Mert R. Sabuncu, and Jordan W. Smoller.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings of the National Academy of Sciences of the United States of America 112(8)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Ge2015b').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Ge2015b" class="abstract">
			The discovery and prioritization of heritable phenotypes is a computational challenge in a variety of settings, including neuroimaging genetics and analyses of the vast phenotypic repositories in electronic health record systems and population-based biobanks. Classical estimates of heritability require twin or pedigree data, which can be costly and difficult to acquire. Genome-wide complex trait analysis is an alternative tool to compute heritability estimates from unrelated individuals, using genome-wide data that are increasingly ubiquitous, but is computationally demanding and becomes difficult to apply in evaluating very large numbers of phenotypes. Here we present a fast and accurate statistical method for high-dimensional heritability analysis using genome-wide SNP data from unrelated individuals, termed massively expedited genome-wide heritability analysis (MEGHA) and accompanying nonparametric sampling techniques that enable flexible inferences for arbitrary statistics of interest. MEGHA produces estimates and significance measures of heritability with several orders of magnitude less computational time than existing methods, making heritability-based prioritization of millions of phenotypes based on data from unrelated individuals tractable for the first time to our knowledge. As a demonstration of application, we conducted heritability analyses on global and local morphometric measurements derived from brain structural MRI scans, using genome-wide SNP data from 1,320 unrelated young healthy adults of non-Hispanic European ancestry. We also computed surface maps of heritability for cortical thickness measures and empirically localized cortical regions where thickness measures were significantly heritable. Our analyses demonstrate the unique capability of MEGHA for large-scale heritability-based screening and high-dimensional heritability profile construction.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/25600633">A kernel machine method for detecting effects of interaction between multidimensional variable sets: An imaging genetics application.</a>.
			</span><br />
			<span class="authors">
				Tian Ge, Thomas E Nichols, Debashis Ghosh, Elizabeth C Mormino, Jordan W Smoller, and Mert R Sabuncu.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Ge2015a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Ge2015a" class="abstract">
			Measurements derived from neuroimaging data can serve as markers of disease and/or healthy development, are largely heritable, and have been increasingly utilized as (intermediate) phenotypes in genetic association studies. To date, imaging genetic studies have mostly focused on discovering isolated genetic effects, typically ignoring potential interactions with non-genetic variables such as disease risk factors, environmental exposures, and epigenetic markers. However, identifying significant interaction effects is critical for revealing the true relationship between genetic and phenotypic variables, and shedding light on disease mechanisms. In this paper, we present a general kernel machine based method for detecting effects of the interaction between multidimensional variable sets. This method can model the joint and epistatic effect of a collection of single nucleotide polymorphisms (SNPs), accommodate multiple factors that potentially moderate genetic influences, and test for nonlinear interactions between sets of variables in a flexible framework. As a demonstration of application, we applied the method to the data from the Alzheimer&#39;s Disease Neuroimaging Initiative (ADNI) to detect the effects of the interactions between candidate Alzheimer&#39;s disease (AD) risk genes and a collection of cardiovascular disease (CVD) risk factors, on hippocampal volume measurements derived from structural brain magnetic resonance imaging (MRI) scans. Our method identified that two genes, CR1 and EPHA1, demonstrate significant interactions with CVD risk factors on hippocampal volume, suggesting that CR1 and EPHA1 may play a role in influencing AD-related neurodegeneration in the presence of CVD risks.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.nature.com/doifinder/10.1038/nature14101 http://dx.doi.org/10.1038/nature14101 10.1038/nature14101 http://www.nature.com/nature/journal/vaop/ncurrent/abs/nature14101.html{\#}supplementary-information http://www.ncbi.nlm.nih.gov/pubmed/25607358 htt">Common genetic variants influence human subcortical brain structures.</a>.
			</span><br />
			<span class="authors">
				Derrek P. Hibar, Jason L. Stein, Miguel E. Renteria, Alejandro Arias-Vasquez, Sylvane Desrivi{\`{e}}res, Neda Jahanshad, Roberto Toro, Katharina Wittfeld, Lucija Abramovic, Micael Andersson, Benjamin S. Aribisala, Nicola J. Armstrong, Manon Bernard, Marc M. Bohlken, Marco P. Boks, Janita Bralten, Andrew a. Brown, M Mallar Chakravarty, Qiang Chen, Christopher R. K. Ching, Gabriel Cuellar-Partida, Anouk Braber, Sudheer Giddaluru, Aaron L. Goldman, Oliver Grimm, Tulio Guadalupe, Johanna Hass, Girma Woldehawariat, Avram J. Holmes, Martine Hoogman, Deborah Janowitz, Tianye Jia, Sungeun Kim, Marieke Klein, Bernd Kraemer, Phil H. Lee, Loes M. {Olde Loohuis}, Michelle Luciano, Christine Macare, Karen a. Mather, Manuel Mattheisen, Yuri Milaneschi, Kwangsik Nho, Martina Papmeyer, Adaikalavan Ramasamy, Shannon L. Risacher, Roberto Roiz-Santia{\~{n}}ez, Emma J. Rose, Alireza Salami, Philipp G. S{\"{a}}mann, Lianne Schmaal, Andrew J. Schork, Jean Shin, Lachlan T. Strike, Alexander Teumer, Marjolein M. J. Donkelaar, Kristel R. Eijk, Raymond K. Walters, Lars T. Westlye, Christopher D. Whelan, Anderson M. Winkler, Marcel P. Zwiers, Saud Alhusaini, Lavinia Athanasiu, Stefan Ehrlich, Marina M. H. Hakobjan, Cecilie B. Hartberg, Unn K. Haukvik, Angelien J. G. a. M. Heister, David Hoehn, Dalia Kasperaviciute, David C. M. Liewald, Lorna M. Lopez, Remco R. R. Makkinje, Mar Matarin, Marlies a. M. Naber, D Reese McKay, Margaret Needham, Allison C. Nugent, Benno P{\"{u}}tz, Natalie a. Royle, Li Shen, Emma Sprooten, Daniah Trabzuni, Saskia S. L. Marel, Kimm J. E. Hulzen, Esther Walton, Christiane Wolf, Laura Almasy, David Ames, Sampath Arepalli, Amelia a. Assareh, Mark E. Bastin, Henry Brodaty, Kazima B. Bulayeva, Melanie a. Carless, Sven Cichon, Aiden Corvin, Joanne E. Curran, Michael Czisch, Greig I. Zubicaray, Allissa Dillman, Ravi Duggirala, Thomas D. Dyer, Susanne Erk, Iryna O. Fedko, Luigi Ferrucci, Tatiana M. Foroud, Peter T. Fox, Masaki Fukunaga, J Raphael Gibbs, Harald H. H. G{\"{o}}ring, Robert C. Green, Sebastian Guelfi, Narelle K. Hansell, Catharina a. Hartman, Katrin Hegenscheid, Andreas Heinz, Dena G. Hernandez, Dirk J. Heslenfeld, Pieter J. Hoekstra, Florian Holsboer, Georg Homuth, Jouke-Jan Hottenga, Masashi Ikeda, Clifford R. Jack, Mark Jenkinson, Robert Johnson, Ryota Kanai, Maria Keil, Jack W. Kent, Peter Kochunov, John B. Kwok, Stephen M. Lawrie, Xinmin Liu, Dan L. Longo, Katie L McMahon, Eva Meisenzahl, Ingrid Melle, Sebastian Mohnke, Grant W. Montgomery, Jeanette C. Mostert, Thomas W. M{\"{u}}hleisen, Michael a. Nalls, Thomas E. Nichols, Lars G. Nilsson, Markus M. N{\"{o}}then, Kazutaka Ohi, Rene L. Olvera, Rocio Perez-Iglesias, G Bruce Pike, Steven G. Potkin, Ivar Reinvang, Simone Reppermund, Marcella Rietschel, Nina Romanczuk-Seiferth, Glenn D. Rosen, Dan Rujescu, Knut Schnell, Peter R. Schofield, Colin Smith, Vidar M. Steen, Jessika E. Sussmann, Anbupalam Thalamuthu, Arthur W. Toga, Bryan J. Traynor, Juan Troncoso, Jessica a. Turner, Maria C. {Vald{\'{e}}s Hern{\'{a}}ndez}, Dennis {van 't Ent}, Marcel Brug, Nic J. a. Wee, Marie-Jose Tol, Dick J. Veltman, Thomas H. Wassink, Eric Westman, Ronald H. Zielke, Alan B. Zonderman, David G. Ashbrook, Reinmar Hager, Lu Lu, Francis J. McMahon, Derek W. Morris, Robert W. Williams, Han G. Brunner, Randy L. Buckner, Jan K. Buitelaar, Wiepke Cahn, Vince D. Calhoun, Gianpiero L. Cavalleri, Benedicto Crespo-Facorro, Anders M. Dale, Gareth E. Davies, Norman Delanty, Chantal Depondt, Srdjan Djurovic, Wayne C. Drevets, Thomas Espeseth, Randy L. Gollub, Beng-Choon Ho, Wolfgang Hoffmann, Norbert Hosten, Ren{\'{e}} S Kahn, Stephanie {Le Hellard}, Andreas Meyer-Lindenberg, Bertram M{\"{u}}ller-Myhsok, Matthias Nauck, Lars Nyberg, Massimo Pandolfo, Brenda W. J. H. Penninx, Joshua L. Roffman, Sanjay M. Sisodiya, Jordan W. Smoller, Hans Bokhoven, Neeltje E. M. Haren, Henry V{\"{o}}lzke, Henrik Walter, Michael W. Weiner, Wei Wen, Tonya White, Ingrid Agartz, Ole a. Andreassen, John Blangero, Dorret I. Boomsma, Rachel M. Brouwer, Dara M. Cannon, Mark R. Cookson, Eco J. C. Geus, Ian J. Deary, Gary Donohoe, Guill{\'{e}}n Fern{\'{a}}ndez, Simon E. Fisher, Clyde Francks, David C. Glahn, Hans J. Grabe, Oliver Gruber, John Hardy, Ryota Hashimoto, Hilleke E. {Hulshoff Pol}, Erik G. J{\"{o}}nsson, Iwona Kloszewska, Simon Lovestone, Venkata S. Mattay, Patrizia Mecocci, Colm McDonald, Andrew M. McIntosh, Roel a. Ophoff, Tomas Paus, Zdenka Pausova, Mina Ryten, Perminder S. Sachdev, Andrew J. Saykin, Andy Simmons, Andrew Singleton, Hilkka Soininen, Joanna M. Wardlaw, Michael E. Weale, Daniel R. Weinberger, Hieab H. H. Adams, Lenore J. Launer, Stephan Seiler, Reinhold Schmidt, Ganesh Chauhan, Claudia L. Satizabal, James T. Becker, Lisa Yanek, Sven J. Lee, Maritza Ebling, Bruce Fischl, W. T. Longstreth, Douglas Greve, Helena Schmidt, Paul Nyquist, Louis N. Vinke, Cornelia M. Duijn, Luting Xue, Bernard Mazoyer, Joshua C. Bis, Vilmundur Gudnason, Sudha Seshadri, M Arfan Ikram, {Alzheimer's Disease Neuroimaging Initiative}, {CHARGE Consortium}, EPIGEN, IMAGEN, SYS, Nicholas G. Martin, Margaret J. Wright, Gunter Schumann, Barbara Franke, Paul M. Thompson, and Sarah E. Medland.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature 520(7546)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Hibar2015').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Hibar2015" class="abstract">
			The highly complex structure of the human brain is strongly shaped by genetic influences. Subcortical brain regions form circuits with cortical areas to coordinate movement, learning, memory and motivation, and altered circuits can lead to abnormal behaviour and disease. To investigate how common genetic variants affect the structure of these brain regions, here we conduct genome-wide association studies of the volumes of seven subcortical regions and the intracranial volume derived from magnetic resonance images of 30,717 individuals from 50 cohorts. We identify five novel genetic variants influencing the volumes of the putamen and caudate nucleus. We also find stronger evidence for three loci with previously established influences on hippocampal volume and intracranial volume. These variants show specific volumetric effects on brain structures rather than global effects across structures. The strongest effects were found for the putamen, where a novel intergenic locus with replicable influence on volume (rs945270; P = 1.08 x 10(-33); 0.52{\%} variance explained) showed evidence of altering the expression of the KTN1 gene in both brain and blood tissue. Variants influencing putamen volume clustered near developmental genes that regulate apoptosis, axon guidance and vesicle transport. Identification of these genetic variants provides insight into the causes of variability in human brain development, and may help to determine mechanisms of neuropsychiatric dysfunction.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20150005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://ieeexplore.ieee.org/lpdocs/epic03/wrapper.htm?arnumber=7164132">Empirically investigating the statistical validity of SPM, FSL and AFNI for single subject fMRI analysis</a>.
			</span><br />
			<span class="authors">
				Anders Eklund, Thomas Nichols, Mats Andersson, and Hans Knutsson.
			</span><br />
			<span class="venuetype"></span><span class="venue">2015 IEEE 12th International Symposium on Biomedical Imaging (ISBI)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2014</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20140009 --><big><strong>Preprints</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www2.warwick.ac.uk/fac/sci/statistics/crism/research/paper14-25">Modelling the Penumbra in Computed Tomography</a>.
			</span><br />
			<span class="authors">
				Audrey Kueh, Jason W Warnett, Gregory Gibbons, Julia Brettschneider, Thomas E Nichols, Mark A WIlliams, and Wilfrid Stephen Kendall.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Impact of family structure and common environment on heritability estimation for neuroimaging genetics studies using Sequential Oligogenic Linkage Analysis Routines.
			</span><br />
			<span class="authors">
				Mary Ellen Koran, Tricia a Thornton-Wells, Neda Jahanshad, David C Glahn, Paul M Thompson, John Blangero, Thomas E Nichols, Peter Kochunov, and Bennett a Landman.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of medical imaging (Bellingham, Wash.) 1(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Koran2014a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Koran2014a" class="abstract">
			Imaging genetics is an emerging methodological field that combines genetic information with medical imaging-derived metrics to understand how genetic factors impact observable phenotypes. In order for a trait to be a reasonable phenotype in an imaging genetics study, it must be heritable: at least some proportion of its variance must be due to genetic influences. The Sequential Oligogenic Linkage Analysis Routines (SOLAR) imaging genetics software can estimate the heritability of a trait in complex pedigrees. We investigate the ability of SOLAR to accurately estimate heritability and common environmental effects on simulated imaging phenotypes in various family structures. We found that heritability is reliably estimated with small family-based studies of 40 to 80 individuals, though subtle differences remain between the family structures. In an imaging application analysis, we found that with 80 subjects in any of the family structures, estimated heritability of white matter fractional anisotropy was biased by {\textless}10{\%} for every region of interest. Results from these studies can be used when investigators are evaluating power in planning genetic analyzes.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/25173418">Imaging proteomics for diagnosis, monitoring and prediction of Alzheimer's disease.</a>.
			</span><br />
			<span class="authors">
				Arash Nazeri, Habib Ganjgahi, Tina Roostaei, Thomas Nichols, and Mojtaba Zarei.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Nazeri2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Nazeri2014" class="abstract">
			Proteomic and imaging markers have been widely studied as potential biomarkers for diagnosis, monitoring and prognosis of Alzheimer&#39;s disease. In this study, we used Alzheimer Disease Neuroimaging Initiative dataset and performed parallel independent component analysis on cross sectional and longitudinal proteomic and imaging data in order to identify the best proteomic model for diagnosis, monitoring and prediction of Alzheimer disease (AD). We used plasma proteins measurement and imaging data from AD and healthy controls (HC) at the baseline and 1 year follow-up. Group comparisons at baseline and changes over 1 year were calculated for proteomic and imaging data. The results were fed into parallel independent component analysis in order to identify proteins that were associated with structural brain changes cross sectionally and longitudinally. Regression model was used to find the best model that can discriminate AD from HC, monitor AD and to predict MCI converters from non-converters. We showed that five proteins are associated with structural brain changes in the brain. These proteins could discriminate AD from HC with 57{\%} specificity and 89{\%} sensitivity. Four proteins whose change over 1 year were associated with brain structural changes could discriminate AD from HC with sensitivity of 93{\%}, and specificity of 92{\%}. This model predicted MCI conversion to AD in 2 years with 94{\%} accuracy. This model has the highest accuracy in prediction of MCI conversion to AD within the ADNI-1 dataset. This study shows that combination of selected plasma protein levels and MR imaging is a useful method in identifying potential biomarker.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=4192369{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Joint genetic analysis of hippocampal size in mouse and human identifies a novel gene linked to neurodegenerative disease.</a>.
			</span><br />
			<span class="authors">
				David G Ashbrook, Robert W Williams, Lu Lu, Jason L Stein, Derrek P Hibar, Thomas E Nichols, Sarah E Medland, Paul M Thompson, and Reinmar Hager.
			</span><br />
			<span class="venuetype"></span><span class="venue">BMC genomics 15(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Ashbrook2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Ashbrook2014" class="abstract">
			BACKGROUND: Variation in hippocampal volume has been linked to significant differences in memory, behavior, and cognition among individuals. To identify genetic variants underlying such differences and associated disease phenotypes, multinational consortia such as ENIGMA have used large magnetic resonance imaging (MRI) data sets in human GWAS studies. In addition, mapping studies in mouse model systems have identified genetic variants for brain structure variation with great power. A key challenge is to understand how genetically based differences in brain structure lead to the propensity to develop specific neurological disorders. RESULTS: We combine the largest human GWAS of brain structure with the largest mammalian model system, the BXD recombinant inbred mouse population, to identify novel genetic targets influencing brain structure variation that are linked to increased risk for neurological disorders. We first use a novel cross-species, comparative analysis using mouse and human genetic data to identify a candidate gene, MGST3, associated with adult hippocampus size in both systems. We then establish the coregulation and function of this gene in a comprehensive systems-analysis. CONCLUSIONS: We find that MGST3 is associated with hippocampus size and is linked to a group of neurodegenerative disorders, such as Alzheimer&#39;s.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/25380333">Toward a Multisubject Analysis of Neural Connectivity.</a>.
			</span><br />
			<span class="authors">
				C J Oates, L Costa, and T E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neural computation</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Oates2014a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Oates2014a" class="abstract">
			Directed acyclic graphs (DAGs) and associated probability models are widely used to model neural connectivity and communication channels. In many experiments, data are collected from multiple subjects whose connectivities may differ but are likely to share many features. In such circumstances, it is natural to leverage similarity among subjects to improve statistical efficiency. The first exact algorithm for estimation of multiple related DAGs was recently proposed by Oates, Smith, Mukherjee, and Cussens (2014). In this letter we present examples and discuss implications of the methodology as applied to the analysis of fMRI data from a multisubject experiment. Elicitation of tuning parameters requires care, and we illustrate how this may proceed retrospectively based on technical replicate data. In addition to joint learning of subject-specific connectivity, we allow for heterogeneous collections of subjects and simultaneously estimate relationships between the subjects themselves. This letter aims to highlight the potential for exact estimation in the multisubject setting.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811913012731"> Corrigendum to "Multi-site genetic analysis of diffusion images and voxelwise heritability analysis: A pilot project of the ENIGMA-DTI working group" [NeuroImage 81 (2013) 455-469]</a>.
			</span><br />
			<span class="authors">
				Neda Jahanshad, Peter V. Kochunov, Emma Sprooten, Ren{\'{e}} C. Mandl, Thomas E. Nichols, Laura Almasy, John Blangero, Rachel M. Brouwer, Joanne E. Curran, Greig I. Zubicaray, Ravi Duggirala, Peter T. Fox, L. Elliot Hong, Bennett a. Landman, Nicholas G. Martin, Katie L. McMahon, Sarah E. Medland, Braxton D. Mitchell, Rene L. Olvera, Charles P. Peterson, John M. Starr, Jessika E. Sussmann, Arthur W. Toga, Joanna M. Wardlaw, Margaret J. Wright, Hilleke E. Hulshoff Pol, Mark E. Bastin, Andrew M. McIntosh, Ian J. Deary, Paul M. Thompson, and David C. Glahn.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/24657781">Multi-site study of additive genetic effects on fractional anisotropy of cerebral white matter: Comparing meta and megaanalytical approaches for data pooling.</a>.
			</span><br />
			<span class="authors">
				Peter Kochunov, Neda Jahanshad, Emma Sprooten, Thomas E Nichols, Ren{\'{e}} C Mandl, Laura Almasy, Tom Booth, Rachel M Brouwer, Joanne E Curran, Greig I Zubicaray, Rali Dimitrova, Ravi Duggirala, Peter T Fox, L {Elliot Hong}, Bennett a Landman, Herv{\'{e}} Lemaitre, Lorna M Lopez, Nicholas G Martin, Katie L McMahon, Braxton D Mitchell, Rene L Olvera, Charles P. Peterson, John M Starr, Jessika E Sussmann, Arthur W Toga, Joanna M Wardlaw, Margaret J. Wright, Susan N Wright, Mark E Bastin, Andrew M McIntosh, Dorret I Boomsma, Ren{\'{e}} S Kahn, Anouk Braber, Eco J C Geus, Ian J Deary, Hilleke E {Hulshoff Pol}, Douglas E Williamson, John Blangero, Dennis {van 't Ent}, Paul M Thompson, and David C Glahn.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Kochunov2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Kochunov2014" class="abstract">
			Combining datasets across independent studies can boost statistical power by increasing the numbers of observations and can achieve more accurate estimates of effect sizes. This is especially important for genetic studies where a large number of observations are required to obtain sufficient power to detect and replicate genetic effects. There is a need to develop and evaluate methods for joint-analytical analyses of rich datasets collected in imaging genetics studies. The ENIGMA-DTI consortium is developing and evaluating approaches for obtaining pooled estimates of heritability through meta-and mega-genetic analytical approaches, to estimate the general additive genetic contributions to the intersubject variance in fractional anisotropy (FA) measured from diffusion tensor imaging (DTI). We used the ENIGMA-DTI data harmonization protocol for uniform processing of DTI data from multiple sites. We evaluated this protocol in five family-based cohorts providing data from a total of 2248 children and adults (ages: 9-85) collected with various imaging protocols. We used the imaging genetics analysis tool, SOLAR-Eclipse, to combine twin and family data from Dutch, Australian and Mexican-American cohorts into one large &#34;mega-family&#34;. We showed that heritability estimates may vary from one cohort to another. We used two meta-analytical (the sample-size and standard-error weighted) approaches and a mega-genetic analysis to calculate heritability estimates across-population. We performed leave-one-out analysis of the joint estimates of heritability, removing a different cohort each time to understand the estimate variability. Overall, meta- and mega-genetic analyses of heritability produced robust estimates of heritability.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=4133042{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract http://www.ncbi.nlm.nih.gov/pubmed/25122193">Global Genetic Variations Predict Brain Response to Faces.</a>.
			</span><br />
			<span class="authors">
				Erin W Dickie, Amir Tahmasebi, Leon French, Natasa Kovacevic, Tobias Banaschewski, Gareth J Barker, Arun Bokde, Christian B{\"{u}}chel, Patricia Conrod, Herta Flor, Hugh Garavan, Juergen Gallinat, Penny Gowland, Andreas Heinz, Bernd Ittermann, Claire Lawrence, Karl Mann, Jean-Luc Martinot, Frauke Nees, Thomas Nichols, Mark Lathrop, Eva Loth, Zdenka Pausova, Marcela Rietschel, Michal N Smolka, Andreas Str{\"{o}}hle, Roberto Toro, Gunter Schumann, and Tom{\'{a}}{\v{s}} Paus.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS genetics 10(8)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Dickie2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Dickie2014" class="abstract">
			Face expressions are a rich source of social signals. Here we estimated the proportion of phenotypic variance in the brain response to facial expressions explained by common genetic variance captured by ~ 500,000 single nucleotide polymorphisms. Using genomic-relationship-matrix restricted maximum likelihood (GREML), we related this global genetic variance to that in the brain response to facial expressions, as assessed with functional magnetic resonance imaging (fMRI) in a community-based sample of adolescents (n = 1,620). Brain response to facial expressions was measured in 25 regions constituting a face network, as defined previously. In 9 out of these 25 regions, common genetic variance explained a significant proportion of phenotypic variance (40-50{\%}) in their response to ambiguous facial expressions; this was not the case for angry facial expressions. Across the network, the strength of the genotype-phenotype relationship varied as a function of the inter-individual variability in the number of functional connections possessed by a given region (R(2) = 0.38, p{\textless}0.001). Furthermore, this variability showed an inverted U relationship with both the number of observed connections (R2 = 0.48, p{\textless}0.001) and the magnitude of brain response (R(2) = 0.32, p{\textless}0.001). Thus, a significant proportion of the brain response to facial expressions is predicted by common genetic variance in a subset of regions constituting the face network. These regions show the highest inter-individual variability in the number of connections with other network nodes, suggesting that the genetic model captures variations across the adolescent brains in co-opting these regions into the face network.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/25128255">Cluster-based computational methods for mass univariate analyses of event-related brain potentials/fields: A simulation study.</a>.
			</span><br />
			<span class="authors">
				C R Pernet, M Latinus, T E Nichols, and G a Rousselet.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of neuroscience methods</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Pernet2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Pernet2014" class="abstract">
			BACKGROUND: In recent years, analyses of event related potentials/fields have moved from the selection of a few components and peaks to a mass-univariate approach in which the whole data space is analyzed. Such extensive testing increases the number of false positives and correction for multiple comparisons is needed. METHOD: Here we review all cluster-based correction for multiple comparison methods (cluster-height, cluster-size, cluster-mass, and threshold free cluster enhancement - TFCE), in conjunction with two computational approaches (permutation and bootstrap). RESULTS: Data driven Monte-Carlo simulations comparing two conditions within subjects (two sample Student&#39;s t-test) showed that, on average, all cluster-based methods using permutation or bootstrap alike control well the family-wise error rate (FWER), with a few caveats. CONCLUSIONS: (i) A minimum of 800 iterations are necessary to obtain stable results; (ii) below 50 trials, bootstrap methods are too conservative; (iii) for low critical family-wise error rates (e.g. p=1{\%}), permutations can be too liberal; (iv) TFCE controls best the type 1 error rate with an attenuated extent parameter (i.e. power{\textless}1).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.plos.org/10.1371/journal.pone.0097584">Stochastic Blockmodeling of the Modules and Core of the Caenorhabditis elegans Connectome</a>.
			</span><br />
			<span class="authors">
				Dragana M. Pavlovic, Petra E. V{\'{e}}rtes, Edward T. Bullmore, William R. Schafer, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS ONE 9(7)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://biostatistics.oxfordjournals.org/cgi/doi/10.1093/biostatistics/kxu026 http://www.ncbi.nlm.nih.gov/pubmed/24963012">Multiple comparison procedures for neuroimaging genomewide association studies.</a>.
			</span><br />
			<span class="authors">
				Wen-Yu Hua, Thomas E. Nichols, and Debashis Ghosh.
			</span><br />
			<span class="venuetype"></span><span class="venue">Biostatistics (Oxford, England)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Hua2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Hua2014" class="abstract">
			Recent research in neuroimaging has focused on assessing associations between genetic variants that are measured on a genomewide scale and brain imaging phenotypes. A large number of works in the area apply massively univariate analyses on a genomewide basis to find single nucleotide polymorphisms that influence brain structure. In this paper, we propose using various dimensionality reduction methods on both brain structural MRI scans and genomic data, motivated by the Alzheimer&#39;s Disease Neuroimaging Initiative (ADNI) study. We also consider a new multiple testing adjustment method and compare it with two existing false discovery rate (FDR) adjustment methods. The simulation results suggest an increase in power for the proposed method. The real-data analysis suggests that the proposed procedure is able to find associations between genetic variants and brain volume differences that offer potentially new biological insights.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://proceedings.spiedigitallibrary.org/proceeding.aspx?doi=10.1117/12.2043565 http://dx.doi.org/10.1117/12.2043565">On study design in neuroimaging heritability analyses</a>.
			</span><br />
			<span class="authors">
				Mary Ellen Koran, Bo Li, Neda Jahanshad, Tricia a. Thornton-Wells, David C. Glahn, Paul M. Thompson, John Blangero, Thomas E. Nichols, Peter Kochunov, and Bennett a. Landman.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proc. SPIE 9034(615)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_doi:10.1117/12.2043565').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_doi:10.1117/12.2043565" class="abstract">
			Imaging genetics is an emerging methodology that combines genetic information with imaging-derived metrics to understand how genetic factors impact observable structural, functional, and quantitative phenotypes. Many of the most well-known genetic studies are based on Genome-Wide Association Studies (GWAS), which use large populations of related or unrelated individuals to associate traits and disorders with individual genetic factors. Merging imaging and genetics may potentially lead to improved power of association in GWAS because imaging traits may be more sensitive phenotypes, being closer to underlying genetic mechanisms, and their quantitative nature inherently increases power. We are developing SOLAR-ECLIPSE (SE) imaging genetics software which is capable of performing genetic analyses with both large-scale quantitative trait data and family structures of variable complexity. This program can estimate the contribution of genetic commonality among related subjects to a given phenotype, and essentially answer the question of whether or not the phenotype is heritable. This central factor of interest, heritability, offers bounds on the direct genetic influence over observed phenotypes. In order for a trait to be a good phenotype for GWAS, it must be heritable: at least some proportion of its variance must be due to genetic influences. A variety of family structures are commonly used for estimating heritability, yet the variability and biases for each as a function of the sample size are unknown. Herein, we investigate the ability of SOLAR to accurately estimate heritability models based on imaging data simulated using Monte Carlo methods implemented in R. We characterize the bias and the variability of heritability estimates from SOLAR as a function of sample size and pedigree structure (including twins, nuclear families, and nuclear families with grandparents).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.sciencedirect.com/science/article/pii/S1053811914001761 http://www.ncbi.nlm.nih.gov/pubmed/24650594 http://linkinghub.elsevier.com/retrieve/pii/S1053811914001761">Fast and accurate modelling of longitudinal and repeated measures neuroimaging data.</a>.
			</span><br />
			<span class="authors">
				Bryan Guillaume, Xue Hua, Paul M. Thompson, Lourens Waldorp, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Guillaume2014b').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Guillaume2014b" class="abstract">
			Despite the growing importance of longitudinal data in neuroimaging, the standard analysis methods make restrictive or unrealistic assumptions (e.g., assumption of Compound Symmetry-the state of all equal variances and equal correlations-or spatially homogeneous longitudinal correlations). While some new methods have been proposed to more accurately account for such data, these methods are based on iterative algorithms that are slow and failure-prone. In this article, we propose the use of the Sandwich Estimator (SwE) method which first estimates the parameters of interest with a simple Ordinary Least Square model and second estimates variances/covariances with the &#34;so-called&#34; SwE which accounts for the within-subject correlation existing in longitudinal data. Here, we introduce the SwE method in its classic form, and we review and propose several adjustments to improve its behaviour, specifically in small samples. We use intensive Monte Carlo simulations to compare all considered adjustments and isolate the best combination for neuroimaging data. We also compare the SwE method to other popular methods and demonstrate its strengths and weaknesses. Finally, we analyse a highly unbalanced longitudinal dataset from the Alzheimer&#39;s Disease Neuroimaging Initiative and demonstrate the flexibility of the SwE method to fit within- and between-subject effects in a single model. Software implementing this SwE method has been made freely available at http://warwick.ac.uk/tenichols/SwE.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/24530839 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=4010955{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Permutation inference for the general linear model.</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Gerard R Ridgway, Matthew A Webster, Stephen M Smith, and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Winkler2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Winkler2014" class="abstract">
			Permutation methods can provide exact control of false positives and allow the use of non-standard statistics, making only weak assumptions about the data. With the availability of fast and inexpensive computing, their main limitation would be some lack of flexibility to work with arbitrary experimental designs. In this paper we report on results on approximate permutation methods that are more flexible with respect to the experimental design and nuisance variables, and conduct detailed simulations to identify the best method for settings that are typical for imaging research scenarios. We present a generic framework for permutation inference for complex general linear models (GLMS) when the errors are exchangeable and/or have a symmetric distribution, and show that, even in the presence of nuisance effects, these permutation inferences are powerful while providing excellent control of false positives in a wide range of common and relevant imaging research scenarios. We also demonstrate how the inference on GLM parameters, originally intended for independent data, can be used in certain special but useful cases in which independence is violated. Detailed examples of common neuroimaging applications are provided, as well as a complete algorithm - the &#34;randomise&#34; algorithm - for permutation inference with the GLM.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://projecteuclid.org/euclid.aoas/1404229527 http://www.ncbi.nlm.nih.gov/pubmed/25431633">Analysis Of Multiple Sclerosis Lesions Via Spatially Varying Coefficients</a>.
			</span><br />
			<span class="authors">
				Tian Ge, Nicole M{\"{u}}ller-Lenke, Kerstin Bendfeldt, Thomas E. Nichols, and Timothy D Johnson.
			</span><br />
			<span class="venuetype"></span><span class="venue">The annals of applied statistics 8(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_GeMuBeNiJo2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_GeMuBeNiJo2014" class="abstract">
			Magnetic resonance imaging (MRI) plays a vital role in the scientific investigation and clinical management of multiple sclerosis. Analyses of binary multiple sclerosis lesion maps are typically &#34;mass univariate&#34; and conducted with standard linear models that are ill suited to the binary nature of the data and ignore the spatial dependence between nearby voxels (volume elements). Smoothing the lesion maps does not entirely eliminate the non-Gaussian nature of the data and requires an arbitrary choice of the smoothing parameter. Here we present a Bayesian spatial model to accurately model binary lesion maps and to determine if there is spatial dependence between lesion location and subject specific covariates such as MS subtype, age, gender, disease duration and disease severity measures. We apply our model to binary lesion maps derived from T 2-weighted MRI images from 250 multiple sclerosis patients classified into five clinical subtypes, and demonstrate unique modeling and predictive capabilities over existing methods.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/24399358 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=4008818{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">The ENIGMA Consortium: large-scale collaborative analyses of neuroimaging and genetic data.</a>.
			</span><br />
			<span class="authors">
				Paul M Thompson, Jason L Stein, Sarah E Medland, Derrek P Hibar, Alejandro Arias Vasquez, Miguel E Renteria, Roberto Toro, Neda Jahanshad, Gunter Schumann, Barbara Franke, Margaret J. Wright, Nicholas G Martin, Ingrid Agartz, Martin Alda, Saud Alhusaini, Laura Almasy, Jorge Almeida, Kathryn Alpert, Nancy C Andreasen, Ole A Andreassen, Liana G Apostolova, Katja Appel, Nicola J Armstrong, Benjamin Aribisala, Mark E Bastin, Michael Bauer, Carrie E Bearden, Orjan Bergmann, Elisabeth B Binder, John Blangero, Henry J Bockholt, Erlend B{\o}en, Catherine Bois, Dorret I Boomsma, Tom Booth, Ian J Bowman, Janita Bralten, Rachel M Brouwer, Han G Brunner, David G Brohawn, Randy L. Buckner, Jan Buitelaar, Kazima Bulayeva, Juan R Bustillo, Vince D Calhoun, Dara M Cannon, Rita M Cantor, Melanie A Carless, Xavier Caseras, Gianpiero L Cavalleri, M Mallar Chakravarty, Kiki D Chang, Christopher R K Ching, Andrea Christoforou, Sven Cichon, Vincent P Clark, Patricia Conrod, Giovanni Coppola, Benedicto Crespo-Facorro, Joanne E Curran, Michael Czisch, Ian J Deary, Eco J C Geus, Anouk Braber, Giuseppe Delvecchio, Chantal Depondt, Lieuwe Haan, Greig I Zubicaray, Danai Dima, Rali Dimitrova, Srdjan Djurovic, Hongwei Dong, Gary Donohoe, Ravindranath Duggirala, Thomas D Dyer, Stefan Ehrlich, Carl Johan Ekman, Torbj{\o}rn Elvs{\aa}shagen, Louise Emsell, Susanne Erk, Thomas Espeseth, Jesen Fagerness, Scott Fears, Iryna Fedko, Guill{\'{e}}n Fern{\'{a}}ndez, Simon E Fisher, Tatiana Foroud, Peter T Fox, Clyde Francks, Sophia Frangou, Eva Maria Frey, Thomas Frodl, Vincent Frouin, Hugh Garavan, Sudheer Giddaluru, David C Glahn, Beata Godlewska, Rita Z Goldstein, Randy L Gollub, Hans J Grabe, Oliver Grimm, Oliver Gruber, Tulio Guadalupe, Raquel E Gur, Ruben C Gur, Harald H H G{\"{o}}ring, Saskia Hagenaars, Tomas Hajek, Geoffrey B Hall, Jeremy Hall, John Hardy, Catharina A Hartman, Johanna Hass, Sean N Hatton, Unn K Haukvik, Katrin Hegenscheid, Andreas Heinz, Ian B Hickie, Beng-Choon Ho, David Hoehn, Pieter J Hoekstra, Marisa Hollinshead, Avram J Holmes, Georg Homuth, Martine Hoogman, L Elliot Hong, Norbert Hosten, Jouke-Jan Hottenga, Hilleke E {Hulshoff Pol}, Kristy S Hwang, Clifford R Jack, Mark Jenkinson, Caroline Johnston, Erik G J{\"{o}}nsson, Ren{\'{e}} S Kahn, Dalia Kasperaviciute, Sinead Kelly, Sungeun Kim, Peter Kochunov, Laura Koenders, Bernd Kr{\"{a}}mer, John B J Kwok, Jim Lagopoulos, Gonzalo Laje, Mikael Landen, Bennett A Landman, John Lauriello, Stephen M Lawrie, Phil H Lee, Stephanie {Le Hellard}, Herve Lema{\^{i}}tre, Cassandra D Leonardo, Chiang-Shan Li, Benny Liberg, David C Liewald, Xinmin Liu, Lorna M Lopez, Eva Loth, Anbarasu Lourdusamy, Michelle Luciano, Fabio Macciardi, Marise W J Machielsen, Glenda M Macqueen, Ulrik F Malt, Ren{\'{e}} Mandl, Dara S Manoach, Jean-Luc Martinot, Mar Matarin, Karen A Mather, Manuel Mattheisen, Morten Mattingsdal, Andreas Meyer-Lindenberg, Colm McDonald, Andrew M McIntosh, Francis J McMahon, Katie L McMahon, Eva Meisenzahl, Ingrid Melle, Yuri Milaneschi, Sebastian Mohnke, Grant W Montgomery, Derek W Morris, Eric K Moses, Bryon A Mueller, Susana {Mu{\~{n}}oz Maniega}, Thomas W M{\"{u}}hleisen, Bertram M{\"{u}}ller-Myhsok, Benson Mwangi, Matthias Nauck, Kwangsik Nho, Thomas E Nichols, Lars-G{\"{o}}ran Nilsson, Allison C Nugent, Lars Nyberg, Rene L Olvera, Jaap Oosterlaan, Roel A Ophoff, Massimo Pandolfo, Melina Papalampropoulou-Tsiridou, Martina Papmeyer, Tomas Paus, Zdenka Pausova, Godfrey D Pearlson, Brenda W Penninx, Charles P. Peterson, Andrea Pfennig, Mary Phillips, G Bruce Pike, Jean-Baptiste Poline, Steven G. Potkin, Benno P{\"{u}}tz, Adaikalavan Ramasamy, Jerod Rasmussen, Marcella Rietschel, Mark Rijpkema, Shannon L Risacher, Joshua L Roffman, Roberto Roiz-Santia{\~{n}}ez, Nina Romanczuk-Seiferth, Emma J Rose, Natalie A Royle, Dan Rujescu, Mina Ryten, Perminder S Sachdev, Alireza Salami, Theodore D Satterthwaite, Jonathan Savitz, Andrew J Saykin, Cathy Scanlon, Lianne Schmaal, Hugo G Schnack, Andrew J Schork, S Charles Schulz, Remmelt Sch{\"{u}}r, Larry Seidman, Li Shen, Jody M Shoemaker, Andrew Simmons, Sanjay M Sisodiya, Colin Smith, Jordan W Smoller, Jair C Soares, Scott R Sponheim, Emma Sprooten, John M Starr, Vidar M Steen, Stephen Strakowski, Lachlan Strike, Jessika Sussmann, Philipp G S{\"{a}}mann, Alexander Teumer, Arthur W Toga, Diana Tordesillas-Gutierrez, Daniah Trabzuni, Sarah Trost, Jessica Turner, Martijn P Heuvel, Nic J Wee, Kristel Eijk, Theo G M Erp, Neeltje E M Haren, Dennis {van 't Ent}, Marie-Jose Tol, Maria C {Vald{\'{e}}s Hern{\'{a}}ndez}, Dick J Veltman, Amelia Versace, Henry V{\"{o}}lzke, Robert Walker, Henrik Walter, Lei Wang, Joanna M Wardlaw, Michael E Weale, Michael W Weiner, Wei Wen, Lars T Westlye, Heather C Whalley, Christopher D Whelan, Tonya White, Anderson M. Winkler, Katharina Wittfeld, Girma Woldehawariat, Christiane Wolf, David Zilles, Marcel P Zwiers, Anbupalam Thalamuthu, Peter R Schofield, Nelson B Freimer, Natalia S Lawrence, and Wayne Drevets.
			</span><br />
			<span class="venuetype"></span><span class="venue">Brain imaging and behavior 8(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Thompson2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Thompson2014" class="abstract">
			The Enhancing NeuroImaging Genetics through Meta-Analysis (ENIGMA) Consortium is a collaborative network of researchers working together on a range of large-scale studies that integrate data from 70 institutions worldwide. Organized into Working Groups that tackle questions in neuroscience, genetics, and medicine, ENIGMA studies have analyzed neuroimaging data from over 12,826 subjects. In addition, data from 12,171 individuals were provided by the CHARGE consortium for replication of findings, in a total of 24,997 subjects. By meta-analyzing results from many sites, ENIGMA has detected factors that affect the brain that no individual site could detect on its own, and that require larger numbers of subjects than any individual neuroimaging study has currently collected. ENIGMA&#39;s first project was a genome-wide association study identifying common variants in the genome associated with hippocampal volume or intracranial volume. Continuing work is exploring genetic associations with subcortical volumes (ENIGMA2) and white matter microstructure (ENIGMA-DTI). Working groups also focus on understanding how schizophrenia, bipolar illness, major depression and attention deficit/hyperactivity disorder (ADHD) affect the brain. We review the current progress of the ENIGMA Consortium, along with challenges and unexpected discoveries made on the way.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/23927901">Post-hoc power estimation for topological inference in fMRI.</a>.
			</span><br />
			<span class="authors">
				Joke Durnez, Beatrijs Moerkerke, and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Durnez2014a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Durnez2014a" class="abstract">
			When analyzing functional MRI data, several thresholding procedures are available to account for the huge number of volume units or features that are tested simultaneously. The main focus of these methods is to prevent an inflation of false positives. However, this comes with a serious decrease in power and leads to a problematic imbalance between type I and type II errors. In this paper, we show how estimating the number of activated peaks or clusters enables one to estimate post-hoc how powerful the selection procedure performs. This procedure can be used in real studies as a diagnostics tool, and raises awareness on how much activation is potentially missed. The method is evaluated and illustrated using simulations and a real data example. Our real data example illustrates the lack of power in current fMRI research.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/23804554">Progression in disability and regional grey matter atrophy in relapsing-remitting multiple sclerosis.</a>.
			</span><br />
			<span class="authors">
				Louis Hofstetter, Yvonne Naegelin, Lukas Filli, Pascal Kuster, Stefan Traud, Renata Smieskova, Nicole Mueller-Lenke, Ludwig Kappos, Achim Gass, Till Sprenger, Iris-Katharina Penner, Thomas E Nichols, Hugo Vrenken, Frederik Barkhof, Chris H Polman, Ernst-Wilhelm Radue, Stefan J Borgwardt, and Kerstin Bendfeldt.
			</span><br />
			<span class="venuetype"></span><span class="venue">Multiple sclerosis (Houndmills, Basingstoke, England) 20(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Hofstetter2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Hofstetter2014" class="abstract">
			BACKGROUND: In multiple sclerosis (MS) regional grey matter (GM) atrophy has been associated with disability progression. OBJECTIVE: The aim of this study was to compare regional GM volume changes in relapsing-remitting MS (RRMS) patients with progressive and stable disability, using voxel-based morphometry (VBM). METHODS: We acquired baseline and 1-year follow-up 3-dimensional (3D) T1-weighted magnetic resonance imaging (MRI) data of RRMS patients, using two 1.5-Tesla scanners. Patients were matched pair-wise with respect to age, gender, disease duration, medication, scanner and baseline Expanded Disability Status Scale (EDSS) into 13 pairs, with either progressive EDSS (&gt;= 1 point change y(-1)) or stable EDSS, as well as into 29 pairs with either progressive Multiple Sclerosis Functional Composite (MSFC) at &gt;= 0.25{\%} decrease in y(-1) in any component, or stable MSFC. We analysed longitudinal regional differences in GM volumes in the progressive and stable EDSS and MSFC groups, respectively, using VBM. RESULTS: Significant GM volume reductions occurred in the right precuneus, in the progressive EDSS group. Differential between-group effects occurred in the right precuneus and in the postcentral gyrus. Further longitudinal GM volume reductions occurred in the right orbicular gyrus, in the progressive MSFC group, but no between-group differences were observed (non-stationary cluster-wise inference, all P(corrected) {\textless} 0.05). CONCLUSION: These results suggested a direct association of disability progression and regional GM atrophy in RRMS.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140007 --><big><strong>Books & Chapters</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1007/978-3-319-10470-6{\_}99">Spatial Modeling of Multiple Sclerosis for Disease Subtype Prediction</a>.
			</span><br />
			<span class="authors">
				Bernd Taschler, Tian Ge, Kerstin Bendfeldt, Nicole M{\"{u}}ller-Lenke, Timothy D Johnson, and Thomas E Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=incollection)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140007 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1007/978-3-319-02475-2{\_}21">Power Estimates for Voxel-Based Genetic Association Studies Using Diffusion Imaging</a>.
			</span><br />
			<span class="authors">
				Neda Jahanshad, Peter Kochunov, David C. Glahn, John Blangero, Thomas E. Nichols, Katie L. McMahon, Greig I. Zubicaray, Nicholas G. Martin, Margaret J. Wright, Clifford R. {Jack, Jr.}, Matt A. Bernstein, Michael W. Weiner, Arthur W. Toga, and Paul M. Thompson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=incollection)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140006 --><big><strong>Miscealeneous </strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S0006322314001589">Development of Impulse Control Circuitry in Children of Alcoholics</a>.
			</span><br />
			<span class="authors">
				Jillian E. Hardee, Barbara J. Weiland, Thomas E. Nichols, Robert C. Welsh, Mary E. Soules, Davia B. Steinberg, Jon Kar Zubieta, Robert A. Zucker, and Mary M. Heitzeg.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=misc)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Hardee2014').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Hardee2014" class="abstract">
			Background: Difficulty with impulse control is heightened in children with a family history of alcohol use disorders and is a risk factor for later substance problems. Cross-sectional functional magnetic resonance imaging studies have shown altered impulse control processing in adolescents with a positive family history, yet developmental trajectories have yet to be examined. Methods: Longitudinal functional magnetic resonance imaging was conducted in children of alcoholic families (family history positive [FH+]; n = 43) and children of control families (family history negative [FH-]; n = 30) starting at ages 7-12 years. Participants performed a go/no-go task during functional magnetic resonance imaging at intervals of 1-2 years, with two to four scans performed per subject. We implemented a repeated-measures linear model fit across all subjects to conduct a whole-brain search for developmental differences between groups. Results: Performance improved with age in both groups, and there were no performance differences between groups. Significant between-group differences in linear age-related activation changes were found in the right caudate, middle cingulate, and middle frontal gyrus. Post hoc analyses revealed significant activation decreases with age in the caudate and middle frontal gyrus for FH- subjects and a significant increase with age in middle cingulate activation for FH+ subjects. Group differences were evident at age 7-12 years, even in alcohol- and drug-na{\&#34;{i}}ve participants, with FH+ subjects showing significantly blunted activation at baseline compared with FH- subjects. Conclusions: Differences in response inhibition circuitry are visible in FH+ individuals during childhood; these differences continue into adolescence, displaying trajectories that are inconsistent with development of normal response inhibition. These patterns precede problem drinking and may be a contributing factor for subsequent substance use problems. {\textcopyright} 2014 Society of Biological Psychiatry.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20140004 --><big><strong>techreport</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.amstat.org/misc/StatisticsBRAIN{\_}April2014.pdf">STATISTICAL RESEARCH AND TRAINING UNDER THE BRAIN INITIATIVE</a>.
			</span><br />
			<span class="authors">
				Robert Kass, Genevera Allen, Brian Caffo, John Cunningham, Uri Eden, Timothy D Johnson, Martin A Lindquist, Thomas E Nichols, Hernando Ombao, Liam Paninski, Russell T Shinohara, and Bin Yu.
			</span><br />
			<span class="venuetype">Technical Report </span><span class="venue"></span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2013</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/24238796">Functional connectomics from resting-state fMRI.</a>.
			</span><br />
			<span class="authors">
				Stephen M Smith, Diego Vidaurre, Christian F Beckmann, Matthew F Glasser, Mark Jenkinson, Karla L Miller, Thomas E Nichols, Emma C Robinson, Gholamreza Salimi-Khorshidi, Mark W Woolrich, Deanna M Barch, Kamil Ugurbil, and David C {Van Essen}.
			</span><br />
			<span class="venuetype"></span><span class="venue">Trends in cognitive sciences 17(12)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Smith2013a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Smith2013a" class="abstract">
			Spontaneous fluctuations in activity in different parts of the brain can be used to study functional brain networks. We review the use of resting-state functional MRI (rfMRI) for the purpose of mapping the macroscopic functional connectome. After describing MRI acquisition and image-processing methods commonly used to generate data in a form amenable to connectomics network analysis, we discuss different approaches for estimating network structure from that data. Finally, we describe new possibilities resulting from the high-quality rfMRI data being generated by the Human Connectome Project and highlight some upcoming challenges in functional connectomics.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://projecteuclid.org/euclid.aoas/1414091235">A Bayesian Hierarchical Spatial Point Process Model for Multi-Type Neuroimaging Meta-Analysis</a>.
			</span><br />
			<span class="authors">
				Jian Kang, Thomas E. Nichols, Tor D. Wager, and Timothy D. Johnson.
			</span><br />
			<span class="venuetype"></span><span class="venue">The Annals of Applied Statistics 8(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://link.springer.com/10.3758/s13415-013-0185-3 http://www.ncbi.nlm.nih.gov/pubmed/23868644">Alternative-based thresholding with application to presurgical fMRI.</a>.
			</span><br />
			<span class="authors">
				Joke Durnez, Beatrijs Moerkerke, Andreas Bartsch, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Cognitive, affective {\&} behavioral neuroscience 13(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Durnez2013').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Durnez2013" class="abstract">
			Functional magnetic reasonance imaging (fMRI) plays an important role in pre-surgical planning for patients with resectable brain lesions such as tumors. With appropriately designed tasks, the results of fMRI studies can guide resection, thereby preserving vital brain tissue. The mass univariate approach to fMRI data analysis consists of performing a statistical test in each voxel, which is used to classify voxels as either active or inactive-that is, related, or not, to the task of interest. In cognitive neuroscience, the focus is on controlling the rate of false positives while accounting for the severe multiple testing problem of searching the brain for activations. However, stringent control of false positives is accompanied by a risk of false negatives, which can be detrimental, particularly in clinical settings where false negatives may lead to surgical resection of vital brain tissue. Consequently, for clinical applications, we argue for a testing procedure with a stronger focus on preventing false negatives. We present a thresholding procedure that incorporates information on false positives and false negatives. We combine two measures of significance for each voxel: a classical p-value, which reflects evidence against the null hypothesis of no activation, and an alternative p-value, which reflects evidence against activation of a prespecified size. This results in a layered statistical map for the brain. One layer marks voxels exhibiting strong evidence against the traditional null hypothesis, while a second layer marks voxels where activation cannot be confidently excluded. The third layer marks voxels where the presence of activation can be rejected.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://projecteuclid.org/euclid.aoas/1372338475">Dynamic filtering of static dipoles in magnetoencephalography</a>.
			</span><br />
			<span class="authors">
				Alberto Sorrentino, Adam M. Johansen, John a. D. Aston, Thomas E. Nichols, and Wilfrid S. Kendall.
			</span><br />
			<span class="venuetype"></span><span class="venue">The Annals of Applied Statistics 7(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811913005077 http://www.ncbi.nlm.nih.gov/pubmed/23707675">Genetics of the connectome.</a>.
			</span><br />
			<span class="authors">
				Paul M. Thompson, Tian Ge, David C. Glahn, Neda Jahanshad, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Thompson2013').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Thompson2013" class="abstract">
			Connectome genetics attempts to discover how genetic factors affect brain connectivity. Here we review a variety of genetic analysis methods--such as genome-wide association studies (GWAS), linkage and candidate gene studies--that have been fruitfully adapted to imaging data to implicate specific variants in the genome for brain-related traits. Studies that emphasized the genetic influences on brain connectivity. Some of these analyses of brain integrity and connectivity using diffusion MRI, and others have mapped genetic effects on functional networks using resting state functional MRI. Connectome-wide genome-wide scans have also been conducted, and we review the multivariate methods required to handle the extremely high dimension of the genomic and network data. We also review some consortium efforts, such as ENIGMA, that offer the power to detect robust common genetic associations using phenotypic harmonization procedures and meta-analysis. Current work on connectome genetics is advancing on many fronts and promises to shed light on how disease risk genes affect the brain. It is already discovering new genetic loci and even entire genetic networks that affect brain organization and connectivity.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/23707588">Transcriptomics of cortical gray matter thickness decline during normal aging.</a>.
			</span><br />
			<span class="authors">
				P Kochunov, J Charlesworth, Anderson M. Winkler, L E Hong, T Nichols, J E Curran, E Sprooten, N Jahanshad, Paul M. Thompson, M P Johnson, J W Kent, B a Landman, B Mitchell, S a Cole, T D Dyer, E K Moses, H H H Goring, L Almasy, R Duggirala, R L Olvera, David C. Glahn, and John Blangero.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Kochunov2013').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Kochunov2013" class="abstract">
			INTRODUCTION: We performed a whole-transcriptome correlation analysis, followed by the pathway enrichment and testing of innate immune response pathway analyses to evaluate the hypothesis that transcriptional activity can predict cortical gray matter thickness (GMT) variability during normal cerebral aging. METHODS: Transcriptome and GMT data were available for 379 individuals (age range=28-85) community-dwelling members of large extended Mexican American families. Collection of transcriptome data preceded that of neuroimaging data by 17years. Genome-wide gene transcriptome data consisted of 20,413 heritable lymphocytes-based transcripts. GMT measurements were performed from high-resolution (isotropic 800$\mu$m) T1-weighted MRI. Transcriptome-wide and pathway enrichment analysis was used to classify genes correlated with GMT. Transcripts for sixty genes from seven innate immune pathways were tested as specific predictors of GMT variability. RESULTS: Transcripts for eight genes (IGFBP3, LRRN3, CRIP2, SCD, IDS, TCF4, GATA3, and HN1) passed the transcriptome-wide significance threshold. Four orthogonal factors extracted from this set predicted 31.9{\%} of the variability in the whole-brain and between 23.4 and 35{\%} of regional GMT measurements. Pathway enrichment analysis identified six functional categories including cellular proliferation, aggregation, differentiation, viral infection, and metabolism. The integrin signaling pathway was significantly (p{\textless}10(-6)) enriched with GMT. Finally, three innate immune pathways (complement signaling, toll-receptors and scavenger and immunoglobulins) were significantly associated with GMT. CONCLUSION: Expression activity for the genes that regulate cellular proliferation, adhesion, differentiation and inflammation can explain a significant proportion of individual variability in cortical GMT. Our findings suggest that normal cerebral aging is the product of a progressive decline in regenerative capacity and increased neuroinflammation.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/23629049 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3729717{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Multi-site genetic analysis of diffusion images and voxelwise heritability analysis: a pilot project of the ENIGMA-DTI working group.</a>.
			</span><br />
			<span class="authors">
				Neda Jahanshad, Peter V Kochunov, Emma Sprooten, Ren{\'{e}} C Mandl, Thomas E Nichols, Laura Almasy, John Blangero, Rachel M Brouwer, Joanne E Curran, Greig I Zubicaray, Ravi Duggirala, Peter T Fox, L Elliot Hong, Bennett a Landman, Nicholas G Martin, Katie L McMahon, Sarah E Medland, Braxton D Mitchell, Rene L Olvera, Charles P. Peterson, John M Starr, Jessika E Sussmann, Arthur W. Toga, Joanna M Wardlaw, Margaret J. Wright, Hilleke E {Hulshoff Pol}, Mark E. Bastin, Andrew M McIntosh, Ian J Deary, Paul M. Thompson, and David C. Glahn.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Jahanshad2013').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Jahanshad2013" class="abstract">
			The ENIGMA (Enhancing NeuroImaging Genetics through Meta-Analysis) Consortium was set up to analyze brain measures and genotypes from multiple sites across the world to improve the power to detect genetic variants that influence the brain. Diffusion tensor imaging (DTI) yields quantitative measures sensitive to brain development and degeneration, and some common genetic variants may be associated with white matter integrity or connectivity. DTI measures, such as the fractional anisotropy (FA) of water diffusion, may be useful for identifying genetic variants that influence brain microstructure. However, genome-wide association studies (GWAS) require large populations to obtain sufficient power to detect and replicate significant effects, motivating a multi-site consortium effort. As part of an ENIGMA-DTI working group, we analyzed high-resolution FA images from multiple imaging sites across North America, Australia, and Europe, to address the challenge of harmonizing imaging data collected at multiple sites. Four hundred images of healthy adults aged 18-85 from four sites were used to create a template and corresponding skeletonized FA image as a common reference space. Using twin and pedigree samples of different ethnicities, we used our common template to evaluate the heritability of tract-derived FA measures. We show that our template is reliable for integrating multiple datasets by combining results through meta-analysis and unifying the data through exploratory mega-analyses. Our results may help prioritize regions of the FA map that are consistently influenced by additive genetic factors for future genetic discovery studies. Protocols and templates are publicly available at (http://enigma.loni.ucla.edu/ongoing/dti-working-group/).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pnas.org/cgi/doi/10.1073/pnas.1301816110 http://www.ncbi.nlm.nih.gov/pubmed/23690582">Preventing Alzheimer's disease-related gray matter atrophy by B-vitamin treatment.</a>.
			</span><br />
			<span class="authors">
				Gwena{\"{e}}lle Douaud, Helga Refsum, Celeste a. Jager, Robin Jacoby, Thomas E Nichols, Stephen M Smith, and A. David Smith.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings of the National Academy of Sciences of the United States of America 110(23)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Douaud2013b').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Douaud2013b" class="abstract">
			Is it possible to prevent atrophy of key brain regions related to cognitive decline and Alzheimer&#39;s disease (AD)? One approach is to modify nongenetic risk factors, for instance by lowering elevated plasma homocysteine using B vitamins. In an initial, randomized controlled study on elderly subjects with increased dementia risk (mild cognitive impairment according to 2004 Petersen criteria), we showed that high-dose B-vitamin treatment (folic acid 0.8 mg, vitamin B6 20 mg, vitamin B12 0.5 mg) slowed shrinkage of the whole brain volume over 2 y. Here, we go further by demonstrating that B-vitamin treatment reduces, by as much as seven fold, the cerebral atrophy in those gray matter (GM) regions specifically vulnerable to the AD process, including the medial temporal lobe. In the placebo group, higher homocysteine levels at baseline are associated with faster GM atrophy, but this deleterious effect is largely prevented by B-vitamin treatment. We additionally show that the beneficial effect of B vitamins is confined to participants with high homocysteine (above the median, 11 \mu mol/L) and that, in these participants, a causal Bayesian network analysis indicates the following chain of events: B vitamins lower homocysteine, which directly leads to a decrease in GM atrophy, thereby slowing cognitive decline. Our results show that B-vitamin supplementation can slow the atrophy of specific brain regions that are a key component of the AD process and that are associated with cognitive decline. Further B-vitamin supplementation trials focusing on elderly subjets with high homocysteine levels are warranted to see if progression to dementia can be prevented.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20130008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/22627277">A Bayesian non-parametric Potts model with application to pre-surgical FMRI data.</a>.
			</span><br />
			<span class="authors">
				Timothy D Johnson, Zhuqing Liu, Andreas J Bartsch, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Statistical methods in medical research 22(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Johnson2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Johnson2012" class="abstract">
			The Potts model has enjoyed much success as a prior model for image segmentation. Given the individual classes in the model, the data are typically modeled as Gaussian random variates or as random variates from some other parametric distribution. In this article, we present a non-parametric Potts model and apply it to a functional magnetic resonance imaging study for the pre-surgical assessment of peritumoral brain activation. In our model, we assume that the Z-score image from a patient can be segmented into activated, deactivated, and null classes, or states. Conditional on the class, or state, the Z-scores are assumed to come from some generic distribution which we model non-parametrically using a mixture of Dirichlet process priors within the Bayesian framework. The posterior distribution of the model parameters is estimated with a Markov chain Monte Carlo algorithm, and Bayesian decision theory is used to make the final classifications. Our Potts prior model includes two parameters, the standard spatial regularization parameter and a parameter that can be interpreted as the a priori probability that each voxel belongs to the null, or background state, conditional on the lack of spatial regularization. We assume that both of these parameters are unknown, and jointly estimate them along with other model parameters. We show through simulation studies that our model performs on par, in terms of posterior expected loss, with parametric Potts models when the parametric model is correctly specified and outperforms parametric models when the parametric model in misspecified.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2012</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3641659{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Measuring and comparing brain cortical surface area and other areal quantities.</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Mert R Sabuncu, B T Thomas Yeo, Bruce Fischl, Douglas N Greve, Peter Kochunov, Thomas E Nichols, John Blangero, and David C Glahn.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 61(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Winkler2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Winkler2012" class="abstract">
			Structural analysis of MRI data on the cortical surface usually focuses on cortical thickness. Cortical surface area, when considered, has been measured only over gross regions or approached indirectly via comparisons with a standard brain. Here we demonstrate that direct measurement and comparison of the surface area of the cerebral cortex at a fine scale is possible using mass conservative interpolation methods. We present a framework for analyses of the cortical surface area, as well as for any other measurement distributed across the cortex that is areal by nature. The method consists of the construction of a mesh representation of the cortex, registration to a common coordinate system and, crucially, interpolation using a pycnophylactic method. Statistical analysis of surface area is done with power-transformed data to address lognormality, and inference is done with permutation methods. We introduce the concept of facewise analysis, discuss its interpretation and potential applications.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3635491{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Identification of common variants associated with human hippocampal and intracranial volumes.</a>.
			</span><br />
			<span class="authors">
				Jason L Stein, Sarah E Medland, Alejandro Arias Vasquez, Derrek P Hibar, Rudy E Senstad, Anderson M. Winkler, Roberto Toro, Katja Appel, Richard Bartecek, {\O}rjan Bergmann, Manon Bernard, Andrew a Brown, Dara M Cannon, M Mallar Chakravarty, Andrea Christoforou, Martin Domin, Oliver Grimm, Marisa Hollinshead, Avram J Holmes, Georg Homuth, Jouke-Jan Hottenga, Camilla Langan, Lorna M Lopez, Narelle K Hansell, Kristy S Hwang, Sungeun Kim, Gonzalo Laje, Phil H Lee, Xinmin Liu, Eva Loth, Anbarasu Lourdusamy, Morten Mattingsdal, Sebastian Mohnke, Susana Mu{\~{n}}oz Maniega, Kwangsik Nho, Allison C Nugent, Carol O'Brien, Martina Papmeyer, Benno P{\"{u}}tz, Adaikalavan Ramasamy, Jerod Rasmussen, Mark Rijpkema, Shannon L Risacher, J Cooper Roddey, Emma J Rose, Mina Ryten, Li Shen, Emma Sprooten, Eric Strengman, Alexander Teumer, Daniah Trabzuni, Jessica Turner, Kristel Eijk, Theo G M Erp, Marie-Jose Tol, Katharina Wittfeld, Christiane Wolf, Saskia Woudstra, Andre Aleman, Saud Alhusaini, Laura Almasy, Elisabeth B Binder, David G Brohawn, Rita M Cantor, Melanie a Carless, Aiden Corvin, Michael Czisch, Joanne E Curran, Gail Davies, Marcio a a Almeida, Norman Delanty, Chantal Depondt, Ravi Duggirala, Thomas D Dyer, Susanne Erk, Jesen Fagerness, Peter T Fox, Nelson B Freimer, Michael Gill, Harald H H G{\"{o}}ring, Donald J Hagler, David Hoehn, Florian Holsboer, Martine Hoogman, Norbert Hosten, Neda Jahanshad, Matthew P Johnson, Dalia Kasperaviciute, Jack W Kent, Peter Kochunov, Jack L Lancaster, Stephen M Lawrie, David C Liewald, Ren{\'{e}} Mandl, Mar Matarin, Manuel Mattheisen, Eva Meisenzahl, Ingrid Melle, Eric K Moses, Thomas W M{\"{u}}hleisen, Matthias Nauck, Markus M N{\"{o}}then, Rene L Olvera, Massimo Pandolfo, G Bruce Pike, Ralf Puls, Ivar Reinvang, Miguel E Renter{\'{i}}a, Marcella Rietschel, Joshua L Roffman, Natalie a Royle, Dan Rujescu, Jonathan Savitz, Hugo G Schnack, Knut Schnell, Nina Seiferth, Colin Smith, Vidar M Steen, Maria C {Vald{\'{e}}s Hern{\'{a}}ndez}, Martijn P Heuvel, Nic J Wee, Neeltje E M {Van Haren}, Joris a Veltman, Henry V{\"{o}}lzke, Robert Walker, Lars T Westlye, Christopher D Whelan, Ingrid Agartz, Dorret I Boomsma, Gianpiero L Cavalleri, Anders M Dale, Srdjan Djurovic, Wayne C Drevets, Peter Hagoort, Jeremy Hall, Andreas Heinz, Clifford R Jack, Tatiana M Foroud, Stephanie {Le Hellard}, Fabio Macciardi, Grant W Montgomery, Jean-Baptiste Poline, David J Porteous, Sanjay M Sisodiya, John M Starr, Jessika Sussmann, Arthur W Toga, Dick J Veltman, Henrik Walter, Michael W Weiner, Joshua C Bis, M Arfan Ikram, Albert V Smith, Vilmundur Gudnason, Christophe Tzourio, Meike W Vernooij, Lenore J Launer, Charles DeCarli, Sudha Seshadri, Ole a Andreassen, Liana G Apostolova, Mark E Bastin, John Blangero, Han G Brunner, Randy L. Buckner, Sven Cichon, Giovanni Coppola, Greig I Zubicaray, Ian J Deary, Gary Donohoe, Eco J C Geus, Thomas Espeseth, Guill{\'{e}}n Fern{\'{a}}ndez, David C Glahn, Hans J Grabe, John Hardy, Hilleke E {Hulshoff Pol}, Mark Jenkinson, Ren{\'{e}} S Kahn, Colm McDonald, Andrew M McIntosh, Francis J McMahon, Katie L McMahon, Andreas Meyer-Lindenberg, Derek W Morris, Bertram M{\"{u}}ller-Myhsok, Thomas E Nichols, Roel a Ophoff, Tomas Paus, Zdenka Pausova, Brenda W Penninx, Steven G. Potkin, Philipp G S{\"{a}}mann, Andrew J Saykin, Gunter Schumann, Jordan W Smoller, Joanna M Wardlaw, Michael E Weale, Nicholas G Martin, Barbara Franke, Margaret J. Wright, and Paul M Thompson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature genetics 44(5)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Stein2012a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Stein2012a" class="abstract">
			Identifying genetic variants influencing human brain structures may reveal new biological mechanisms underlying cognition and neuropsychiatric illness. The volume of the hippocampus is a biomarker of incipient Alzheimer&#39;s disease and is reduced in schizophrenia, major depression and mesial temporal lobe epilepsy. Whereas many brain imaging phenotypes are highly heritable, identifying and replicating genetic influences has been difficult, as small effects and the high costs of magnetic resonance imaging (MRI) have led to underpowered studies. Here we report genome-wide association meta-analyses and replication for mean bilateral hippocampal, total brain and intracranial volumes from a large multinational consortium. The intergenic variant rs7294919 was associated with hippocampal volume (12q24.22; N = 21,151; P = 6.70 x 10(-16)) and the expression levels of the positional candidate gene TESC in brain tissue. Additionally, rs10784502, located within HMGA2, was associated with intracranial volume (12q14.3; N = 15,782; P = 1.12 x 10(-12)). We also identified a suggestive association with total brain volume at rs10494373 within DDR2 (1q23.3; N = 6,500; P = 5.81 x 10(-7)).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/22495945">Spatiotemporal distribution of white matter lesions in relapsing-remitting and secondary progressive multiple sclerosis.</a>.
			</span><br />
			<span class="authors">
				Lukas Filli, Louis Hofstetter, Pascal Kuster, Stefan Traud, Nicole Mueller-Lenke, Yvonne Naegelin, Ludwig Kappos, Achim Gass, Till Sprenger, Thomas E Nichols, Hugo Vrenken, Frederik Barkhof, Chris H Polman, Ernst-Wilhelm Radue, Stefan J Borgwardt, and Kerstin Bendfeldt.
			</span><br />
			<span class="venuetype"></span><span class="venue">Multiple sclerosis (Houndmills, Basingstoke, England) 18(11)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Filli2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Filli2012" class="abstract">
			Multiple sclerosis (MS) is a chronic inflammatory disease of the central nervous system. MS lesions show a typical distribution pattern and primarily affect the white matter (WM) in the periventricular zone and in the centrum semiovale.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/22800732">Increasing power for voxel-wise genome-wide association studies: the random field theory, least square kernel machines and fast permutation procedures.</a>.
			</span><br />
			<span class="authors">
				Tian Ge, Jianfeng Feng, Derrek P Hibar, Paul M. Thompson, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 63(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Ge2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Ge2012" class="abstract">
			Imaging traits are thought to have more direct links to genetic variation than diagnostic measures based on cognitive or clinical assessments and provide a powerful substrate to examine the influence of genetics on human brains. Although imaging genetics has attracted growing attention and interest, most brain-wide genome-wide association studies focus on voxel-wise single-locus approaches, without taking advantage of the spatial information in images or combining the effect of multiple genetic variants. In this paper we present a fast implementation of voxel- and cluster-wise inferences based on the random field theory to fully use the spatial information in images. The approach is combined with a multi-locus model based on least square kernel machines to associate the joint effect of several single nucleotide polymorphisms (SNP) with imaging traits. A fast permutation procedure is also proposed which significantly reduces the number of permutations needed relative to the standard empirical method and provides accurate small p-value estimates based on parametric tail approximation. We explored the relation between 448,294 single nucleotide polymorphisms and 18,043 genes in 31,662 voxels of the entire brain across 740 elderly subjects from the Alzheimer&#39;s disease neuroimaging initiative (ADNI). Structural MRI scans were analyzed using tensor-based morphometry (TBM) to compute 3D maps of regional brain volume differences compared to an average template image based on healthy elderly subjects. We find method to be more sensitive compared with voxel-wise single-locus approaches. A number of genes were identified as having significant associations with volumetric changes. The most associated gene was GRIN2B, which encodes the N-methyl-d-aspartate (NMDA) glutamate receptor NR2B subunit and affects both the parietal and temporal lobes in human brains. Its role in Alzheimer&#39;s disease has been widely acknowledged and studied, suggesting the validity of the approach. The various advantages over existing approaches indicate a great potential offered by this novel framework to detect genetic influences on human brains.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.nature.com/doifinder/10.1038/ng.2250 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=3635491{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Identification of common variants associated with human hippocampal and intracranial volumes.</a>.
			</span><br />
			<span class="authors">
				Jason L Stein, Sarah E Medland, Alejandro Arias Vasquez, Derrek P Hibar, Rudy E Senstad, Anderson M. Winkler, Roberto Toro, Katja Appel, Richard Bartecek, {\O}rjan Bergmann, Manon Bernard, Andrew a Brown, Dara M Cannon, M Mallar Chakravarty, Andrea Christoforou, Martin Domin, Oliver Grimm, Marisa Hollinshead, Avram J Holmes, Georg Homuth, Jouke-Jan Hottenga, Camilla Langan, Lorna M Lopez, Narelle K Hansell, Kristy S Hwang, Sungeun Kim, Gonzalo Laje, Phil H Lee, Xinmin Liu, Eva Loth, Anbarasu Lourdusamy, Morten Mattingsdal, Sebastian Mohnke, Susana Mu{\~{n}}oz Maniega, Kwangsik Nho, Allison C Nugent, Carol O'Brien, Martina Papmeyer, Benno P{\"{u}}tz, Adaikalavan Ramasamy, Jerod Rasmussen, Mark Rijpkema, Shannon L Risacher, J Cooper Roddey, Emma J Rose, Mina Ryten, Li Shen, Emma Sprooten, Eric Strengman, Alexander Teumer, Daniah Trabzuni, Jessica Turner, Kristel Eijk, Theo G M Erp, Marie-Jose Tol, Katharina Wittfeld, Christiane Wolf, Saskia Woudstra, Andre Aleman, Saud Alhusaini, Laura Almasy, Elisabeth B Binder, David G Brohawn, Rita M Cantor, Melanie a Carless, Aiden Corvin, Michael Czisch, Joanne E Curran, Gail Davies, Marcio a a Almeida, Norman Delanty, Chantal Depondt, Ravi Duggirala, Thomas D Dyer, Susanne Erk, Jesen Fagerness, Peter T. Fox, Nelson B Freimer, Michael Gill, Harald H H G{\"{o}}ring, Donald J Hagler, David Hoehn, Florian Holsboer, Martine Hoogman, Norbert Hosten, Neda Jahanshad, Matthew P Johnson, Dalia Kasperaviciute, Jack W Kent, Peter Kochunov, Jack L. Lancaster, Stephen M Lawrie, David C Liewald, Ren{\'{e}} Mandl, Mar Matarin, Manuel Mattheisen, Eva Meisenzahl, Ingrid Melle, Eric K Moses, Thomas W M{\"{u}}hleisen, Matthias Nauck, Markus M N{\"{o}}then, Rene L Olvera, Massimo Pandolfo, G Bruce Pike, Ralf Puls, Ivar Reinvang, Miguel E Renter{\'{i}}a, Marcella Rietschel, Joshua L Roffman, Natalie a Royle, Dan Rujescu, Jonathan Savitz, Hugo G Schnack, Knut Schnell, Nina Seiferth, Colin Smith, Vidar M Steen, Maria C {Vald{\'{e}}s Hern{\'{a}}ndez}, Martijn P Heuvel, Nic J Wee, Neeltje E M {Van Haren}, Joris A Veltman, Henry V{\"{o}}lzke, Robert Walker, Lars T Westlye, Christopher D Whelan, Ingrid Agartz, Dorret I Boomsma, Gianpiero L Cavalleri, Anders M. Dale, Srdjan Djurovic, Wayne C Drevets, Peter Hagoort, Jeremy Hall, Andreas Heinz, Clifford R Jack, Tatiana M Foroud, Stephanie {Le Hellard}, Fabio Macciardi, Grant W Montgomery, Jean-Baptiste Poline, David J Porteous, Sanjay M Sisodiya, John M Starr, Jessika Sussmann, Arthur W. Toga, Dick J Veltman, Henrik Walter, Michael W Weiner, Joshua C Bis, M Arfan Ikram, Albert V Smith, Vilmundur Gudnason, Christophe Tzourio, Meike W Vernooij, Lenore J Launer, Charles DeCarli, Sudha Seshadri, Ole a Andreassen, Liana G Apostolova, Mark E. Bastin, John Blangero, Han G Brunner, Randy L. Buckner, Sven Cichon, Giovanni Coppola, Greig I Zubicaray, Ian J Deary, Gary Donohoe, Eco J C Geus, Thomas Espeseth, Guill{\'{e}}n Fern{\'{a}}ndez, David C. Glahn, Hans J Grabe, John Hardy, Hilleke E {Hulshoff Pol}, Mark Jenkinson, Ren{\'{e}} S Kahn, Colm McDonald, Andrew M McIntosh, Francis J McMahon, Katie L McMahon, Andreas Meyer-Lindenberg, Derek W Morris, Bertram M{\"{u}}ller-Myhsok, Thomas E. Nichols, Roel a Ophoff, Tomas Paus, Zdenka Pausova, Brenda W Penninx, Steven G. Potkin, Philipp G S{\"{a}}mann, Andrew J Saykin, Gunter Schumann, Jordan W Smoller, Joanna M Wardlaw, Michael E Weale, Nicholas G Martin, Barbara Franke, Margaret J. Wright, and Paul M. Thompson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature genetics 44(5)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Stein2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Stein2012" class="abstract">
			Identifying genetic variants influencing human brain structures may reveal new biological mechanisms underlying cognition and neuropsychiatric illness. The volume of the hippocampus is a biomarker of incipient Alzheimer&#39;s disease and is reduced in schizophrenia, major depression and mesial temporal lobe epilepsy. Whereas many brain imaging phenotypes are highly heritable, identifying and replicating genetic influences has been difficult, as small effects and the high costs of magnetic resonance imaging (MRI) have led to underpowered studies. Here we report genome-wide association meta-analyses and replication for mean bilateral hippocampal, total brain and intracranial volumes from a large multinational consortium. The intergenic variant rs7294919 was associated with hippocampal volume (12q24.22; N = 21,151; P = 6.70 x 10(-16)) and the expression levels of the positional candidate gene TESC in brain tissue. Additionally, rs10784502, located within HMGA2, was associated with intracranial volume (12q14.3; N = 15,782; P = 1.12 x 10(-12)). We also identified a suggestive association with total brain volume at rs10494373 within DDR2 (1q23.3; N = 6,500; P = 5.81 x 10(-7)).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/21867760">The danger of systematic bias in group-level FMRI-lag-based causality estimation.</a>.
			</span><br />
			<span class="authors">
				Stephen M. Smith, Peter A. Bandettini, K L Miller, Timothy E.J. Behrens, Karl J. Friston, O David, Thomas T. Liu, Mark W. Woolrich, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 59(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Smith2012a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Smith2012a" class="abstract">
			Schippers, Renken and Keysers (NeuroImage, 2011) present a simulation of multi-subject lag-based causality estimation. We fully agree that single-subject evaluations (e.g., Smith et al., 2011) need to be revisited in the context of multi-subject studies, and Schippers&#39; paper is a good example, including detailed multi-level simulation and cross-subject statistical modelling. The authors conclude that &#34;the average chance to find a significant Granger causality effect when no actual influence is present in the data stays well below the p-level imposed on the second level statistics&#34; and that &#34;when the analyses reveal a significant directed influence, this direction was accurate in the vast majority of the cases&#34;. Unfortunately, we believe that the general meaning that may be taken from these statements is not supported by the paper&#39;s results, as there may in reality be a systematic (group-average) difference in haemodynamic delay between two brain areas. While many statements in the paper (e.g., the final two sentences) do refer to this problem, we fear that the overriding message that many readers may take from the paper could cause misunderstanding.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/22245259">Multivariate pattern classification of gray matter pathology in multiple sclerosis.</a>.
			</span><br />
			<span class="authors">
				Kerstin Bendfeldt, Stefan Kl{\"{o}}ppel, Thomas E. Nichols, Renata Smieskova, Pascal Kuster, Stefan Traud, Nicole Mueller-Lenke, Yvonne Naegelin, Ludwig Kappos, Ernst-Wilhelm Radue, and Stefan J Borgwardt.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 60(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Bendfeldt2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Bendfeldt2012" class="abstract">
			Univariate analyses have identified gray matter (GM) alterations in different groups of MS patients. While these methods detect differences on the basis of the single voxel or cluster, multivariate methods like support vector machines (SVM) identify the complex neuroanatomical patterns of GM differences. Using multivariate linear SVM analysis and leave-one-out cross-validation, we aimed at identifying neuroanatomical GM patterns relevant for individual classification of MS patients. We used SVM to separate GM segmentations of T1-weighted three-dimensional magnetic resonance (MR) imaging scans within different age- and sex-matched groups of MS patients with either early (n=17) or late MS (n=17) (contrast I), low (n=20) or high (n=20) white matter lesion load (contrast II), and benign MS (BMS, n=13) or non-benign MS (NBMS, n=13) (contrast III) scanned on a single 1.5T MR scanner. GM patterns most relevant for individual separation of MS patients comprised cortical areas of all the cerebral lobes as well as deep GM structures, including the thalamus and caudate. The patterns detected were sufficiently informative to separate individuals of the respective groups with high sensitivity and specificity in 85{\%} (contrast I), 83{\%} (contrast II) and 77{\%} (contrast III) of cases. The study demonstrates that neuroanatomical spatial patterns of GM segmentations contain information sufficient for correct classification of MS patients at the single case level, thus making multivariate SVM analysis a promising clinical application.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1016/j.neuroimage.2012.04.014 http://www.ncbi.nlm.nih.gov/pubmed/22521256">Multiple testing corrections, nonparametric methods, and random field theory.</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 62(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Nichols2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Nichols2012" class="abstract">
			I provide a selective review of the literature on the multiple testing problem in fMRI. By drawing connections with the older modalities, PET in particular, and how software implementations have tracked (or lagged behind) theoretical developments, my narrative aims to give the methodological researcher a historical perspective on this important aspect of fMRI data analysis.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20120008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/21091816">A Comparison of Gray Matter Density in Restless Legs Syndrome Patients and Matched Controls Using Voxel-Based Morphometry.</a>.
			</span><br />
			<span class="authors">
				Robert a Comley, Simon Cervenka, Sven E Palhagen, Georgios Panagiotidis, Julian C Matthews, Robert Y.K. Lai, Christer Halldin, Lars Farde, Thomas E. Nichols, and Brandon J Whitcher.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of neuroimaging : official journal of the American Society of Neuroimaging 22(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Comley2012').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Comley2012" class="abstract">
			BACKGROUND: Restless legs syndrome (RLS) is a common neurological disorder the pathophysiology of which is incompletely understood. Four studies have examined structural differences between the brains of RLS patients and healthy controls, using voxel-based morphometry (VBM). All 4 studies have provided different results. METHODS: Optimized VBM was used to search for structural differences in gray matter density. Sixteen RLS patients na{\&#34;{i}}ve to dopaminergic drugs and 16 age- and sex-matched controls received structural T1-weighted MR scans. Structural data were analyzed using FSL-VBM. RESULTS: No difference in gray matter density was detected between the two groups (voxel-wise significance: no significant voxels at P= .89 (whole brain Family Wise Error (FWE) corrected); no significant voxels at P {\textless} .05 (whole brain False Discovery Rate (FDR) corrected; smallest achievable FDR threshold .99). CONCLUSION/DISCUSSION: The present study did not replicate (confirm) previous findings of structural brain changes in RLS, but instead supported the findings of a recent study showing a lack of gray matter alteration in an elderly RLS population. More specifically, the results do not support neuronal loss as an underlying disease mechanism in RLS. Potential limitations in the application of VBM are also discussed.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2011</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/22186675">Orbitofrontal Connectivity with Resting-State Networks Is Associated with Midbrain Dopamine D3 Receptor Availability.</a>.
			</span><br />
			<span class="authors">
				David M Cole, Christian F. Beckmann, Graham E Searle, Christophe Plisson, Andri C Tziortzi, Thomas E. Nichols, Roger N Gunn, Paul M Matthews, Eugenii A. Rabiner, and John D Beaver.
			</span><br />
			<span class="venuetype"></span><span class="venue">Cerebral cortex</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Cole2011a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Cole2011a" class="abstract">
			Animal research and human postmortem evidence highlight the importance of brain dopamine D3 receptor (D3R) function in multiple neuropsychiatric disorders, including addiction. Separate anatomical and functional neuroimaging findings implicate disrupted frontal cortical connectivity with distributed brain networks in processes relevant for these diseases. This potential conjunction between molecular and functional markers has not, however, been tested directly. Here, we used a novel combination of [(11)C]-(+)-PHNO positron emission tomography and resting-state functional magnetic resonance imaging in the same healthy individuals to investigate whether differences in midbrain D3R availability are associated with functional interactions between large-scale networks and regions involved in reward processing and cognition. High midbrain D3R availability was associated with reduced functional connectivity between orbitofrontal cortex (OFC) and networks implicated in cognitive control and salience processing. The opposite pattern was observed in subcortical reward circuitry and the &#34;default mode&#34; network, which showed greater connectivity with OFC in individuals with high D3R availability. These findings demonstrate that differential interactions between OFC and networks implicated in cognitive control and reward are associated with midbrain D3R availability, consistent with the hypothesis that dopamine D3R signaling is an important molecular pathway underlying goal-directed behavior.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20977527">Structural brain changes in patients with recurrent major depressive disorder presenting with anxiety symptoms.</a>.
			</span><br />
			<span class="authors">
				Becky Inkster, Anil W Rao, Khanum Ridler, Thomas E. Nichols, Philipp G. Saemann, Dorothee P. Auer, Florian Holsboer, Federica Tozzi, Pierandrea Muglia, Emilio Merlo-Pich, and Paul M Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of neuroimaging : official journal of the American Society of Neuroimaging 21(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Inkster2011').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Inkster2011" class="abstract">
			Major depressive disorder (MDD) presents with extensive clinical heterogeneity. In particular, overlap with anxiety symptoms is common during depressive episodes and as a comorbid disorder. The aim of this study was to test for morphological brain differences between patients having a history of recurrent MDD with, and without, anxiety symptoms (MDD+A and MDD-A).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/22170084">A hybrid procedure for detecting global treatment effects in multivariate clinical trials: theory and applications to fMRI studies.</a>.
			</span><br />
			<span class="authors">
				Giorgos Minas, Fabio Rigat, Thomas E. Nichols, John a D Aston, and Nigel Stallard.
			</span><br />
			<span class="venuetype"></span><span class="venue">Statistics in medicine 31(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Minas2011').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Minas2011" class="abstract">
			In multivariate clinical trials, a key research endpoint is ascertaining whether a candidate treatment is more efficacious than an established alternative. This global endpoint is clearly of high practical value for studies, such as those arising from neuroimaging, where the outcome dimensions are not only numerous but they are also highly correlated and the available sample sizes are typically small. In this paper, we develop a two-stage procedure testing the null hypothesis of global equivalence between treatments effects and demonstrate its application to analysing phase II neuroimaging trials. Prior information such as suitable statistics of historical data or suitably elicited expert clinical opinions are combined with data collected from the first stage of the trial to learn a set of optimal weights. We apply these weights to the outcome dimensions of the second-stage responses to form the linear combination z and t tests statistics while controlling the test&#39;s false positive rate. We show that the proposed tests hold desirable asymptotic properties and characterise their power functions under wide conditions. In particular, by comparing the power of the proposed tests with that of Hotelling&#39;s T(2) , we demonstrate their advantages when sample sizes are close to the dimension of the multivariate outcome. We apply our methods to fMRI studies, where we find that, for sufficiently precise first stage estimates of the treatment effect, standard single-stage testing procedures are outperformed. Copyright {\textcopyright} 2011 John Wiley {\&amp;} Sons, Ltd.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				Genetic analysis of cortical thickness and fractional anisotropy of water diffusion in the brain.
			</span><br />
			<span class="authors">
				Peter Kochunov, David C. Glahn, Thomas E. Nichols, Anderson M. Winkler, Elliot L. Hong, Henry H Holcomb, Jason L Stein, Paul M. Thompson, Joanne E Curran, Melanie A Carless, Rene L Olvera, Matthew P Johnson, Shelley A Cole, Valeria Kochunov, Jack Kent, and John Blangero.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in neuroscience 5(October)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.frontiersin.org/Journal/Abstract.aspx?s=752{\&amp;}name=neuroinformatics{\&amp;}ART{\_}DOI=10.3389/conf.fninf.2011.08.00058">Large-scale automated synthesis of human functional neuroimaging data</a>.
			</span><br />
			<span class="authors">
				Tal Yarkoni, Russell A. Poldrack, Thomas E. Nichols, David C. {Van Essen}, and Tor D. Wager.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature Methods 8(8)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Yarkoni2011').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Yarkoni2011" class="abstract">
			BACKGROUND: A central goal of human neuroimaging research is to map relationships between mind and brain, enabling decoding of cognitive states from brain activity (Norman, Polyn, Detre, {\&amp;} Haxby, 2006; Poldrack, Halchenko, {\&amp;} Hanson, 2009). Previous decoding approaches have focused on discriminating between narrow sets of alternative states, overlooking the vast range of cognitive states human beings can experience. Here we introduce a novel framework called NeuroSynth that uses text mining and meta-analysis techniques to automatically produce accurate mappings between brain activity and a large number of broad cognitive states, providing a valuable new resource for a broad range of neuroimaging applications. The NeuroSynth framework consists of several components. First, an automated parser was used to extract activation coordinates from published neuroimaging studies. The current database contains nearly 145,000 activation foci drawn from over 4,400 studies. Second, all articles were ?tagged? with words and phrases that occur at high frequency within the article text, enabling rapid search and filtering. Third, meta-analysis was used to automatically produce whole-brain meta-analysis maps for over 200 distinct psychological concepts, each based on several hundred studies. Fourth, a na?ve Bayes classifier was used to classify the probability of specific concepts or states being present given new patterns of activation as input, enabling open-ended decoding and classification of brain activity in both entire studies and individual subjects. Validation analyses demonstrated the capacity of the NeuroSynth framework to generate meta-analysis maps that closely replicated previous results for many concepts (e.g., pain, emotion, and working memory). Subsequently, we used NeuroSynth to address two long-standing problems in neuroimaging. First, we addressed the well-known problem of reverse inference (Poldrack, 2006) by quantifying the likelihood of specific psychological processes given observed activation patterns. The results identified numerous cases in which reverse inference analyses identified results discrepant with prior findings (e.g., the lack of selectivity of medial frontal function, and the preferential role of posterior rather than anterior insula in pain). Second, we used a na?ve Bayes classifier to successfully ?decode? which of several psychological states individual subjects were experiencing given only patterns of brain activity, representing an important step towards open-ended classification of cognitive states based solely on prior knowledge. Collectively, our results validate a powerful new approach to the large-scale synthesis of neuroimaging findings. The tools we have developed enable researchers to address long-standing problems of central importance in neuroimaging, and can be used for a broad range of decoding and classification applications. To further encourage the use and development of synthesis-oriented approaches to neuroimaging data, we are publicly releasing the NeuroSynth data and code on the web (www.neurosynth.org) as an open resource. 0
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.plos.org/10.1371/journal.pone.0021570">Brain Network Analysis: Separating Cost from Topology Using Cost-Integration</a>.
			</span><br />
			<span class="authors">
				Cedric E. Ginestet, Thomas E. Nichols, Edward T. Bullmore, and Andrew Simmons.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS ONE 6(7)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://pubs.amstat.org/doi/abs/10.1198/jasa.2011.ap09735">Meta Analysis of Functional Neuroimaging Data via Bayesian Spatial Point Processes</a>.
			</span><br />
			<span class="authors">
				Jian Kang, Timothy D Johnson, Thomas E. Nichols, and Tor D. Wager.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of the American Statistical Association 106(493)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/21382766">Using Gaussian-Process Regression for Meta-analytic Neuroimaging Inference Based on Sparse Observations.</a>.
			</span><br />
			<span class="authors">
				Gholamreza Salimi-Khorshidi, Thomas E. Nichols, Stephen M. Smith, and Mark W. Woolrich.
			</span><br />
			<span class="venuetype"></span><span class="venue">IEEE Transactions on Medical Imaging 30(7)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Salimi-Khorshidi2011').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Salimi-Khorshidi2011" class="abstract">
			The purpose of neuroimaging meta-analysis is to localize the brain regions that are activated consistently in response to a certain intervention. As a commonly used technique, current coordinate-based meta-analyses (CBMA) of neuroimaging studies utilize relatively sparse information from published studies, typically only using (x,y,z) coordinates of the activation peaks. Such CBMA methods have several limitations. First, there is no way to jointly incorporate deactivation information when available, which has been shown to result in an inaccurate statistic image when assessing a difference contrast. Second, the scale of a kernel reflecting spatial uncertainty must be set without taking the effect size (e.g., Z-stat) into account. To address these problems, we employ Gaussian-process regression (GPR), explicitly estimating the unobserved statistic image given the sparse peak activation coordinate and standardized effectsize estimate data. In particular, our model allows estimation of effect size at each voxel, something existing CBMA methods cannot produce. Our results show that GPR outperforms existing CBMA techniques and is capable of more accurately reproducing the (usually unavailable) full-image analysis results.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20849959 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3063336">False positives in neuroimaging genetics using voxel-based morphometry data.</a>.
			</span><br />
			<span class="authors">
				Matt Silver, Giovanni Montana, Thomas E. Nichols, and {Alzheimer's Disease Neuroimaging Initiative}.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 54(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_silv:mont:nich:2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_silv:mont:nich:2010" class="abstract">
			Voxel-wise statistical inference is commonly used to identify significant experimental effects or group differences in both functional and structural studies of the living brain. Tests based on the size of spatially extended clusters of contiguous suprathreshold voxels are also widely used due to their typically increased statistical power. In &#34;imaging genetics&#34;, such tests are used to identify regions of the brain that are associated with genetic variation. However, concerns have been raised about the adequate control of rejection rates in studies of this type. A previous study tested the effect of a set of &#39;null&#39; SNPs on brain structure and function, and found that false positive rates were well-controlled. However, no similar analysis of false positive rates in an imaging genetic study using cluster size inference has yet been undertaken. We measured false positive rates in an investigation of the effect of 700 pre-selected null SNPs on grey matter volume using voxel-based morphometry (VBM). As VBM data exhibit spatially-varying smoothness, we used both non-stationary and stationary cluster size tests in our analysis. Image and genotype data on 181 subjects with mild cognitive impairment were obtained from the Alzheimer&#39;s Disease Neuroimaging Initiative (ADNI). At a nominal significance level of 5{\%}, false positive rates were found to be well-controlled (3.9-5.6{\%}), using a relatively high cluster-forming threshold, $\alpha$(c)=0.001, on images smoothed with a 12 mm Gaussian kernel. Tests were however anticonservative at lower cluster-forming thresholds ($\alpha$(c)=0.01, 0.05), and for images smoothed using a 6mm Gaussian kernel. Here false positive rates ranged from 9.8 to 67.6{\%}. In a further analysis, false positive rates using simulated data were observed to be well-controlled across a wide range of conditions. While motivated by imaging genetics, our findings apply to any VBM study, and suggest that parametric cluster size inference should only be used with high cluster-forming thresholds and smoothness. We would advocate the use of nonparametric methods in other cases.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811910011602">Network Modelling Methods for FMRI</a>.
			</span><br />
			<span class="authors">
				Stephen M. Smith, Karla L. Miller, Gholamreza Salimi-Khorshidi, Matthew Webster, Christian F. Beckmann, Thomas E. Nichols, Joseph D. Ramsey, and Mark W. Woolrich.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 54(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110007 --><big><strong>Books & Chapters</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.booksonline.iospress.nl/Content/View.aspx?piid=20349">Validation and Pilot Application of [18F]FDG-PET in Evaluation of a Metabolic Therapy for Alzheimer's Disease</a>.
			</span><br />
			<span class="authors">
				Sofia Tzimopoulou, Vincent J. Cunningham, Thomas E. Nichols, Graham Searle, Nick P. Bird, Prafull Mistry, Ian J. Dixon, William A. Hallett, Brandon Whitcher, Andrew P. Brown, Marina Zvartau-Hind, Narinder Lotay, Robert Y.K. Lai, Mary Castiglia, Barbara Jeter, Julian C. Matthews, Kewei Chen, Dan Bandy, Eric M. Reiman, Michael Gold, Eugenii A. Rabiner, and Paul M. Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=incollection)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110007 --><big><strong>Books & Chapters</strong></big><br />
			
			<span class="pubtitle">
				Handbook of fMRI Data Analysis.
			</span><br />
			<span class="authors">
				Russell A. Poldrack, Jeanette A. Mumford, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=book)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20110005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.frontiersin.org/Journal/MySubmissionViewDetails.aspx?stage=100{\&amp;}articleid=17598{\&amp;}submissionid=17598">Statistical Parametric Networks (SPNs): A Principled Way of Constructing Functional and Anatomical Networks</a>.
			</span><br />
			<span class="authors">
				Cedric E. Ginestet, Andrew Simmons, Edward T. Bullmore, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Front. Neuroinform. Conference Abstract: 4th INCF Congress of Neuroinformatics</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Ginestet').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Ginestet" class="abstract">
			A statistically principled way of conducting weighted network analysis is still lacking. In this paper, we describe two recent methodological advances in this field, related to (i) the construction of &#39;summary&#39; networks representing variability in connectivity among several populations of networks (Ginestet and Simmons, 2011), and (ii) a mathematical framework for comparing the topology of weighted networks that vary in their number of edges. Firstly, we adopt a mass univariate approach to construct statistical parametric networks (SPNs) that reflect significant modifications in functional connectivity in a canonical working memory task: the N-back paradigm. Our proposed method allows the extraction of `lost&#39; and `gained&#39; functional networks, providing concise graphical summaries of whole-brain network topological changes. Robust estimates of functional networks are obtained by pooling information about edges and vertices over subjects. Therefore, graph thresholding is here supplanted by inference. The analysis proceeds by firstly considering changes in weighted cost (i.e.{\~{}} mean between-region correlation) over the different N-back conditions and secondly comparing small-world topological measures integrated over network cost, thereby controlling for differences in mean correlation between conditions. In particular, we show that weighted cost considerably decreased as working memory load increased. Moreover, subject-specific weighted costs was also found to significantly predict behavioral performances on the N-back task, and therefore conferred predictive validity to functional connectivity strength, as measured by weighted cost. Secondly, we tackle the issue of brain network topological comparison from a formal mathematical perspective. Comparison of different populations of weighted networks is hard because topology is inherently dependent on wiring cost, where cost is defined as the number of edges in an unweighted graph. We evaluate different approaches to the comparison of weighted networks that differ in mean association weight. In our key result, we prove that iStatistical Parametric Networks (SPNs): A Principled Way of Constructing Functional and Anatomical Networks.ntegrating over cost is equivalent to controlling for any monotonic transformationStatistical Parametric Networks (SPNs): A Principled Way of Constructing Functional and Anatomical Networks. of the weight set of a weighted graph. That is, when integrating over cost, we eliminate the differences in topology that may be due to a monotonic transformation of the weight set. This proposition holds for any unweighted topological measure. Cost-integration is therefore helpful in disentangling differences in cost from differences in topology. By contrast, we show that the use of the weighted version of a topological metric does not constitute a valid approach to this problem. Indeed, we prove that, under mild conditions, the use of the weighted version of global efficiency is equivalent to simply comparing weighted costs. Thus, we recommend the reporting of (i) differences in weighted costs and (ii) differences in cost-integrated topological measures. All of these methods have been made publicly available through an R package entitled NetworkAnalysis, and some of the computational problems associated with handling large brain networks will also be discussed. Ginestet, C.E. and Simmons, A. (2011). Statistical parametric network analysis of functional connectivity dynamics during a working memory task. NeuroImage, 5(2), 688-704.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2010</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2949251{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Everything you never wanted to know about circular analysis, but were afraid to ask.</a>.
			</span><br />
			<span class="authors">
				Nikolaus Kriegeskorte, Martin A. Lindquist, Thomas E. Nichols, Russell A. Poldrack, and Edward Vul.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Cerebral Blood Flow {\&} Metabolism 30(9)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Kriegeskorte2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Kriegeskorte2010" class="abstract">
			Over the past year, a heated discussion about &#39;circular&#39; or &#39;nonindependent&#39; analysis in brain imaging has emerged in the literature. An analysis is circular (or nonindependent) if it is based on data that were selected for showing the effect of interest or a related effect. The authors of this paper are researchers who have contributed to the discussion and span a range of viewpoints. To clarify points of agreement and disagreement in the community, we collaboratively assembled a series of questions on circularity herein, to which we provide our individual current answers in {\textless}or=100 words per question. Although divergent views remain on some of the questions, there is also a substantial convergence of opinion, which we have summarized in a consensus box. The box provides the best current answers that the five authors could agree upon.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20955803">Adjusting the effect of nonstationarity in cluster-based and TFCE inference.</a>.
			</span><br />
			<span class="authors">
				Gholamreza Salimi-Khorshidi, Stephen M. Smith, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 54(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Salimi-Khorshidi2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Salimi-Khorshidi2010" class="abstract">
			In nonstationary images, cluster inference depends on the local image smoothness, as clusters tend to be larger in smoother regions by chance alone. In order to correct the inference for such a nonstationary, cluster sizes can be adjusted according to a local smoothness estimate. In this study, adjusted cluster sizes are used in a permutation-testing framework for both cluster-based and threshold-free cluster enhancement (TFCE) inference and tested on both simulated and real data. We find TFCE inference is already fairly robust to nonstationarity in the data, while cluster-based inference requires an adjustment to ensure homogeneity. A group of possible multi-level adjustments are introduced and their results on simulated and real data are assessed using a new performance index. We also find that adjusting for local smoothness via a separate resampling procedure is more effective at removing nonstationarity than an adjustment via a random field theory based smoothness estimator.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20930300">A Multi-Center Randomized Proof-of-Concept Clinical Trial Applying [18F]FDG-PET for Evaluation of Metabolic Therapy with Rosiglitazone XR in Mild to Moderate Alzheimer's Disease - Supplementary data</a>.
			</span><br />
			<span class="authors">
				Sofia Tzimopoulou, Vincent J. Cunningham, Thomas E. Nichols, Graham Searle, Nick P. Bird, Prafull Mistry, Ian J Dixon, William A. Hallett, Brandon Whitcher, Andrew P. Brown, Marina Zvartau-Hind, Narinder Lotay, Robert Y.K. Lai, Mary Castiglia, Barbara Jeter, Julian C Matthews, Kewei Chen, Dan Bandy, Eric M. Reiman, Michael Gold, Eugenii A. Rabiner, and Paul M. Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Alzheimer's disease 22(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Tzimopoulou2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Tzimopoulou2010" class="abstract">
			Here we report the first multi-center clinical trial in Alzheimer&#39;s disease (AD) using fluorodeoxyglucose positron emission tomography ([18F]FDG-PET) measures of brain glucose metabolism as the primary outcome. We contrasted effects of 12 months treatment with the PPAR$\gamma$ agonist Rosiglitazone XR versus placebo in 80 mild to moderate AD patients. Secondary objectives included testing for reduction in the progression of brain atrophy and improvement in cognition. Active treatment was associated with a sustained but not statistically significant trend from the first month for higher mean values in Kiindex and CMRgluindex, novel quantitative indices related to the combined forward rate constant for [18F]FDG uptake and to the rate of cerebral glucose utilization, respectively. However, neither these nor another analytical approach recently validated using data from the Alzheimer&#39;s Disease Neuroimaging Initiative indicated that active treatment decreased the progression of decline in brain glucose metabolism. Rates of brain atrophy were similar between active and placebo groups and measures of cognition also did not suggest clear group differences. Our study demonstrates the feasibility of using [18F]FDG-PET as part of a multi-center therapeutics trial. It suggests that Rosiglitazone is associated with an early increase in whole brain glucose metabolism, but not with any biological or clinical evidence for slowing progression over a 1 year follow up in the symptomatic stages of AD.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20624472">Discovering genetic associations with high-dimensional neuroimaging phenotypes: A sparse reduced-rank regression approach.</a>.
			</span><br />
			<span class="authors">
				Maria Vounou, Thomas E. Nichols, and Giovanni Montana.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 53(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Vounou2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Vounou2010" class="abstract">
			There is growing interest in performing genome-wide searches for associations between genetic variants and brain imaging phenotypes. While much work has focused on single scalar valued summaries of brain phenotype, accounting for the richness of imaging data requires a brain-wide, genome-wide search. In particular, the standard approach based on mass-univariate linear modelling (MULM) does not account for the structured patterns of correlations present in each domain. In this work, we propose sparse reduced rank regression (sRRR), a strategy for multivariate modelling of high-dimensional imaging responses (measurements taken over regions of interest or individual voxels) and genetic covariates (single nucleotide polymorphisms or copy number variations), which enforces sparsity in the regression coefficients. Such sparsity constraints ensure that the model performs simultaneous genotype and phenotype selection. Using simulation procedures that accurately reflect realistic human genetic variation and imaging correlations, we present detailed evaluations of the sRRR method in comparison with the more traditional MULM approach. In all settings considered, sRRR has better power to detect deleterious genetic variants compared to MULM. Important issues concerning model selection and connections to existing latent variable models are also discussed. This work shows that sRRR offers a promising alternative for detecting brain-wide, genome-wide associations.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20108225">Spatiotemporal distribution pattern of white matter lesion volumes and their association with regional grey matter volume reductions in relapsing-remitting multiple sclerosis.</a>.
			</span><br />
			<span class="authors">
				Kerstin Bendfeldt, Jan Ole Blumhagen, Hanspeter Egger, Patrick Loetscher, Niklaus Denier, Pascal Kuster, Stefan Traud, Nicole Mueller-Lenke, Yvonne Naegelin, Achim Gass, Jochen Hirsch, Ludwig Kappos, Thomas E. Nichols, Ernst-Wilhelm Radue, and Stefan J Borgwardt.
			</span><br />
			<span class="venuetype"></span><span class="venue">Human brain mapping 000(October 2009)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Bendfeldt2010a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Bendfeldt2010a" class="abstract">
			The association of white matter (WM) lesions and grey matter (GM) atrophy is a feature in relapsing-remitting multiple sclerosis (RRMS). The spatiotemporal distribution pattern of WM lesions, their relations to regional GM changes and the underlying dynamics are unclear. Here we combined parametric and non-parametric voxel-based morphometry (VBM) to clarify these issues. MRI data from RRMS patients with progressive (PLV, n = 45) and non-progressive WM lesion volumes (NPLV, n = 44) followed up for 12 months were analysed. Cross-sectionally, the spatial WM lesion distribution was compared using lesion probability maps (LPMs). Longitudinally, WM lesions and GM volumes were studied using FSL-VBM and SPM5-VBM, respectively. WM lesions clustered around the lateral ventricles and in the centrum semiovale with a more widespread pattern in the PLV than in the NPLV group. The maximum local probabilities were similar in both groups and higher for T2 lesions (PLV: 27{\%}, NPLV: 25{\%}) than for T1 lesions (PLV: 15{\%}, NPLV 14{\%}). Significant WM lesion changes accompanied by cortical GM volume reductions occured in the corpus callosum and optic radiations (P = 0.01 corrected), and more liberally tested (uncorrected P {\textless} 0.01) in the inferior fronto-occipital and longitudinal fasciculi, and corona radiata in the PLV group. Not any WM or GM changes were found in the NPLV group. In the PLV group, WM lesion distribution and development in fibres, was associated with regional GM volume loss. The different spatiotemporal distribution patterns of patients with progressive compared to patients with non-progressive WM lesions suggest differences in the dynamics of pathogenesis. Hum Brain Mapp, 2010. (c) 2010 Wiley-Liss, Inc.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20167205">Effect of immunomodulatory medication on regional gray matter loss in relapsing-remitting multiple sclerosis--a longitudinal MRI study.</a>.
			</span><br />
			<span class="authors">
				Kerstin Bendfeldt, Hanspeter Egger, Thomas E. Nichols, Patrick Loetscher, Niklaus Denier, Pascal Kuster, Stefan Traud, Nicole Mueller-Lenke, Yvonne Naegelin, Achim Gass, Ludwig Kappos, Ernst-Wilhelm Radue, and Stefan J Borgwardt.
			</span><br />
			<span class="venuetype"></span><span class="venue">Brain research</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Bendfeldt2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Bendfeldt2010" class="abstract">
			Prevention of global gray matter (GM) volume changes in multiple sclerosis (MS) are an objective in clinical trials, but the effect of immunomodulatory medication on regional GM atrophy progression is unclear. MRIs from 86 patients with relapsing-remitting MS (RRMS) followed up for 24 months were analyzed using voxel-based morphometry. An analysis of covariance model (cluster threshold, corrected p{\textless}0.05) was used to compare GM volumes between baseline and follow-up while stratified by immunomodulatory medication (IM): Interferone INF-beta-1a (n=34), INF-beta-1b (n=16), glatiramer acetate (GA) (n=15), and no-immunomodulatory treatment (n=21). In the INF-beta-1a/1b group (n=50), significant GM volume reductions were observed during follow-up in fronto-temporal, cingulate and cerebellar cortical brain regions, without significant differences between the INF-beta-1a and INF-beta-1b patients. In the GA group and in unmedicated patients, no significant regional GM volume reductions were observed. In contrast to GA, INF-beta-1a/1b treatment was associated with GM volume reductions in hippocampal/parahippocampal and anterior cingulate cortex. This is the first longitudinal study investigating the effects of IMs on GM in RRMS. Results suggest differences in the dynamics of regional GM volume atrophy in differentially treated or untreated RRMS patients.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20100008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/20219685">Pathway-based approaches to imaging genetics association studies: Wnt signaling, GSK3beta substrates and major depression.</a>.
			</span><br />
			<span class="authors">
				Becky Inkster, Thomas E. Nichols, Philipp G. Saemann, Dorothee P. Auer, Florian Holsboer, Pierandrea Muglia, and Paul M. Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 53(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Inkster2010').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Inkster2010" class="abstract">
			Several lines of evidence implicate glycogen synthase kinase 3 beta (GSK3beta) in mood disorders. We recently reported associations between GSK3beta polymorphisms and brain structural changes in patients with recurrent major depressive disorder (MDD). Here we provide supporting observations by showing that polymorphisms in additional genes encoding proteins directly related to GSK3beta biological functions are associated with similar regional grey matter (GM) volume changes in MDD patients. We tested specifically for associations with genetic variation in canonical Wnt signaling pathway genes and in genes that encode substrate proteins of GSK3beta. We applied a general linear model with non-stationary cluster-based inference to examine associations between polymorphisms and regional voxel-based morphometry GM volume differences in recurrent MDD patients (n=134) and in age-, gender-, and ethnicity-matched healthy controls (n=144) to test for genotype-by-MDD interactions. We observed associations for polymorphisms in 8/13 canonical Wnt pathway genes and 5/10 GSK3beta substrate genes, predominantly in the temporolateral and medial prefrontal cortices. Similar associations were not found for 100 unrelated polymorphisms tested. This work suggests that identifying SNPs related to genes that encode functionally-interacting proteins that modulate common anatomical regions offers a useful approach to increasing confidence in outcomes from imaging genetics association studies. This is of particular interest when replication datasets are not available. Our observations lend support to the hypothesis that polymorphisms in GSK3beta play a role in MDD susceptibility or expression, in part, by acting via the canonical Wnt signaling pathway and related substrates.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2009</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				Adjusting the neuroimaging statistical inferences for nonstationarity.
			</span><br />
			<span class="authors">
				Gholamreza Salimi-Khorshidi, Stephen M. Smith, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Lecture Notes in Computer Science 5761 LNCS(PART 1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Salimi-Khorshidi2009_LNCS').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Salimi-Khorshidi2009_LNCS" class="abstract">
			In neuroimaging cluster-based inference has generally been found to be more powerful than voxel-wise inference. However standard cluster-based methods assume stationarity (constant smoothness), while under nonstationarity clusters are larger in smooth regions just by chance, making false positive risk spatially variant. Hayasaka et al. proposed a Random Field Theory (RFT) based nonstationarity adjustment for cluster inference and validated the method in terms of controlling the overall family-wise false positive rate. The RFT-based methods, however, have never been directly assessed in terms of homogeneity of local false positive risk. In this work we propose a new cluster size adjustment that accounts for local smoothness, based on local empirical cluster size distributions and a two-pass permutation method. We also propose a new approach to measure homogeneity of local false positive risk, and use this method to compare the RFT-based and our new empirical adjustment methods. We apply these techniques to both cluster-based and a related inference, threshold-free cluster enhancement (TFCE). Using simulated and real data we confirm the expected heterogeneity in false positive risk with unadjusted cluster inference but find that RFT-based adjustment does not fully eliminate heterogeneity; we also observe that our proposed empirical adjustment dramatically increases the homogeneity and TFCE inference is generally quite robust to nonstationarity.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/21254367">Association of GSK3 Polymorphisms With Brain Structural Changes in Major Depressive Disorder</a>.
			</span><br />
			<span class="authors">
				Becky Inkster, Thomas E. Nichols, Philipp G. Saemann, Dorothee P. Auer, Florian Holsboer, Pierandrea Muglia, and Paul M. Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">Archives of general psychiatry 66(7)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://pps.sagepub.com/lookup/doi/10.1111/j.1745-6924.2009.01126.x">Commentary on Vul et al.'s (2009) "Puzzlingly High Correlations in fMRI Studies of Emotion, Personality, and Social Cognition"</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and Jean-Baptiste Poline.
			</span><br />
			<span class="venuetype"></span><span class="venue">Perspectives on Psychological Science 4(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/19463958">Simple group fMRI modeling and inference.</a>.
			</span><br />
			<span class="authors">
				Jeanette A Mumford and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 47(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Mumford2009').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Mumford2009" class="abstract">
			While many advanced mixed-effects models have been proposed and are used in fMRI, the simplest, ordinary least squares (OLS), is still the one that is most widely used. A survey of 90 papers found that 92{\%} of group fMRI analyses used OLS. Despite the widespread use, this simple approach has never been thoroughly justified and evaluated; for example, the typical reference for the method is a conference abstract, (Holmes, A., Friston, K., 1998. Generalisability, random effects {\&amp;} population inference. NeuroImage 7 (4 (2/3)), S754, proceedings of Fourth International Conference on Functional Mapping of the Human Brain, June 7-12, 1998, Montreal, Canada.), which has been referenced over 400 times. In this work we fully derive the simplified method in a general setting and carefully identify the homogeneity assumptions it is based on. We examine the specificity (Type I error rate) of the OLS method under heterogeneity in the one-sample case and find that the OLS method is valid, with only slight conservativeness. Surprisingly, a Satterthwaite approximation for effective degrees of freedom only makes the method more conservative, instead of more accurate. While other authors have highlighted the inferior power of the OLS method relative to optimal mixed-effects methods under heterogeneity, we revisit these results and find the power differences very modest. While statistical methods that make the best use of the data are always to be preferred, software or other practical concerns may require the use of the simple OLS group modeling. In such cases, we find that group mean inferences will be valid under the null hypothesis and will have nearly optimal sensitivity under the alternative.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				Modeling Inter-Subject Variability in fMRI Activation Location: A Bayesian Hierarchical Spatial Model.
			</span><br />
			<span class="authors">
				Lei Xu, Timothy D Johnson, Thomas E. Nichols, and Derek E Nee.
			</span><br />
			<span class="venuetype"></span><span class="venue">Biometrics 65(4)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811908012111 http://www.ncbi.nlm.nih.gov/pubmed/19063980 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=PMC3318962">Evaluating the consistency and specificity of neuroimaging data using meta-analysis.</a>.
			</span><br />
			<span class="authors">
				Tor D. Wager, Martin A. Lindquist, Thomas E. Nichols, Hedy Kober, and Jared X {Van Snellenberg}.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 45(1 Suppl)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Wager2009').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Wager2009" class="abstract">
			Making sense of a neuroimaging literature that is growing in scope and complexity will require increasingly sophisticated tools for synthesizing findings across studies. Meta-analysis of neuroimaging studies fills a unique niche in this process: It can be used to evaluate the consistency of findings across different laboratories and task variants, and it can be used to evaluate the specificity of findings in brain regions or networks to particular task types. This review discusses examples, implementation, and considerations when choosing meta-analytic techniques. It focuses on the multilevel kernel density analysis (MKDA) framework, which has been used in recent studies to evaluate consistency and specificity of regional activation, identify distributed functional networks from patterns of co-activation, and test hypotheses about functional cortical-subcortical pathways in healthy individuals and patients with mental disorders. Several tests of consistency and specificity are described.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1016/j.neuroimage.2008.03.061 http://www.ncbi.nlm.nih.gov/pubmed/18501637">Threshold-free cluster enhancement: addressing problems of smoothing, threshold dependence and localisation in cluster inference</a>.
			</span><br />
			<span class="authors">
				Stephen M. Smith and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 44(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Smith2009').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Smith2009" class="abstract">
			Many image enhancement and thresholding techniques make use of spatial neighbourhood information to boost belief in extended areas of signal. The most common such approach in neuroimaging is cluster-based thresholding, which is often more sensitive than voxel-wise thresholding. However, a limitation is the need to define the initial cluster-forming threshold. This threshold is arbitrary, and yet its exact choice can have a large impact on the results, particularly at the lower (e.g., t, z {\textless} 4) cluster-forming thresholds frequently used. Furthermore, the amount of spatial pre-smoothing is also arbitrary (given that the expected signal extent is very rarely known in advance of the analysis). In the light of such problems, we propose a new method which attempts to keep the sensitivity benefits of cluster-based thresholding (and indeed the general concept of &#34;clusters&#34; of signal), while avoiding (or at least minimising) these problems. The method takes a raw statistic image and produces an output image in which the voxel-wise values represent the amount of cluster-like local spatial support. The method is thus referred to as &#34;threshold-free cluster enhancement&#34; (TFCE). We present the TFCE approach and discuss in detail ROC-based optimisation and comparisons with cluster-based and voxel-based thresholding. We find that TFCE gives generally better sensitivity than other methods over a wide range of test signal shapes and SNR values. We also show an example on a real imaging dataset, suggesting that TFCE does indeed provide not just improved sensitivity, but richer and more interpretable output than cluster-based thresholding.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1016/j.neuroimage.2008.12.039">Meta-analysis of neuroimaging data: A comparison of image-based and coordinate-based pooling of studies</a>.
			</span><br />
			<span class="authors">
				Gholamreza Salimi-Khorshidi, Stephen M. Smith, John R Keltner, Tor D. Wager, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 45(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://dx.doi.org/10.1016/j.neuroimage.2008.10.003">NeuroImage Anatomically-distinct genetic associations of APOE \epsilon 4 allele load with regional cortical atrophy in Alzheimer ' s disease</a>.
			</span><br />
			<span class="authors">
				Nicola Filippini, Anil Rao, Sally Wetten, Rachel A Gibson, Michael Borrie, Danilo Guzman, Andrew Kertesz, Inge Loy-english, Julie Williams, Thomas E. Nichols, Brandon Whitcher, and Paul M. Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 44(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				Striatal [11C] dihydrotetrabenazine and [11C] methylphenidate binding in Tourette syndrome.
			</span><br />
			<span class="authors">
				R L Albin, R A Koeppe, K Wernette, W Zhuang, Thomas E. Nichols, M R Kilbourn, and A Frey.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neurology</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811908009117">Cluster mass inference via random field theory</a>.
			</span><br />
			<span class="authors">
				Hui Zhang, Thomas E. Nichols, and Timothy D Johnson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neuroimage 44(1)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20090004 --><big><strong>techreport</strong></big><br />
			
			<span class="pubtitle">
				Bayesian Hierarchical Spatial Modeling of fMRI Multi-Subject Data, Web-based supplementary Materials for.
			</span><br />
			<span class="authors">
				Lei Xu, Timothy D Johnson, Thomas E. Nichols, and Derek E Nee.
			</span><br />
			<span class="venuetype">Technical Report </span><span class="venue"></span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2008</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20080008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Inferring individual differences in fMRI: finding brain regions with significant within subject correlation.
			</span><br />
			<span class="authors">
				Sumitra Purkayastha, Tor D. Wager, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Statistica Sinica</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20080008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2287206{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Guidelines for reporting an fMRI study.</a>.
			</span><br />
			<span class="authors">
				Russell A. Poldrack, Paul C Fletcher, Richard N Henson, Keith J Worsley, Matthew Brett, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 40(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Poldrack2008').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Poldrack2008" class="abstract">
			In this editorial, we outline a set of guidelines for the reporting of methods and results in functional magnetic resonance imaging studies and provide a checklist to assist authors in preparing manuscripts that meet these guidelines.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20080008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811907007100 http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2423281{\&amp;}tool=pmcentrez{\&amp;}rendertype=abstract">Power calculation for group fMRI studies accounting for arbitrary design and temporal autocorrelation.</a>.
			</span><br />
			<span class="authors">
				Jeanette A Mumford and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 39(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Mumford2008a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Mumford2008a" class="abstract">
			When planning most scientific studies, one of the first steps is to carry out a power analysis to define a design and sample size that will result in a well-powered study. There are limited resources for calculating power for group fMRI studies due to the complexity of the model. Previous approaches for group fMRI power calculation simplify the study design and/or the variance structure in order to make the calculation possible. These approaches limit the designs that can be studied and may result in inaccurate power calculations. We introduce a flexible power calculation model that makes fewer simplifying assumptions, leading to a more accurate power analysis that can be used on a wide variety of study designs. Our power calculation model can be used to obtain region of interest (ROI) summaries of the mean parameters and variance parameters, which can be use to increase understanding of the data as well as calculate power for a future study. Our example illustrates that minimizing cost to achieve 80{\%} power is not as simple as finding the smallest sample size capable of achieving 80{\%} power, since smaller sample sizes require each subject to be scanned longer.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20080008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/18718572">Optimizing the design and analysis of clinical functional magnetic resonance imaging research studies.</a>.
			</span><br />
			<span class="authors">
				Cameron S Carter, Stephan Heckers, Thomas Nichols, Daniel S Pine, and Stephen Strother.
			</span><br />
			<span class="venuetype"></span><span class="venue">Biological psychiatry 64(10)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Carter2008').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Carter2008" class="abstract">
			With the widespread availability of functional magnetic resonance imaging (fMRI), there has been rapid progress in identifying neural correlates of cognition and emotion in the human brain. In conjunction with basic research studies, fMRI has been increasingly applied in clinical disorders, making it a central research tool in human psychopathology, psychopharmacology, and genetics. In the present article, we discuss a number of conceptual and methodological challenges that confront the implementation of fMRI in clinical and translational research, and we offer a set of recommendations intended to enhance the interpretability and reproducibility of results in clinical fMRI.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2007</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20070008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/17406613">Acquisition and voxelwise analysis of multi-subject diffusion data with tract-based spatial statistics.</a>.
			</span><br />
			<span class="authors">
				Stephen M. Smith, Heidi Johansen-Berg, Mark Jenkinson, Daniel Rueckert, Thomas E. Nichols, Karla L Miller, Matthew D Robson, Derek K Jones, Johannes C Klein, Andreas J Bartsch, and Timothy E.J. Behrens.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature protocols 2(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Smith2007a').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Smith2007a" class="abstract">
			There is much interest in using magnetic resonance diffusion imaging to provide information on anatomical connectivity in the brain by measuring the diffusion of water in white matter tracts. Among the measures, the most commonly derived from diffusion data is fractional anisotropy (FA), which quantifies local tract directionality and integrity. Many multi-subject imaging studies are using FA images to localize brain changes related to development, degeneration and disease. In a recent paper, we presented a new approach, tract-based spatial statistics (TBSS), which aims to solve crucial issues of cross-subject data alignment, allowing localized cross-subject statistical analysis. This works by transforming the data from the centers of the tracts that are consistent across a study&#39;s subjects into a common space. In this protocol, we describe the MRI data acquisition and analysis protocols required for TBSS studies of localized change in brain connectivity across multiple subjects.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20070007 --><big><strong>Books & Chapters</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://books.google.com/books?hl=en{\&amp;}lr={\&amp;}id=G{\_}qdEsDlkp0C{\&amp;}oi=fnd{\&amp;}pg=PA253{\&amp;}dq=Non-parametric+procedures{\&amp;}ots=Xk1HDBU9QE{\&amp;}sig=d1aaZTDYvNhnnYFfbICVWuRF1d0">Non-parametric procedures</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and A Holmes.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=incollection)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20070007 -->
			<span class="pubtitle">
				False Discovery Rate procedures.
			</span><br />
			<span class="authors">
				Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=incollection)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2006</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/16786032">Impact of complex genetic variation in COMT on human brain function.</a>.
			</span><br />
			<span class="authors">
				a Meyer-Lindenberg, Thomas E. Nichols, J H Callicott, J Ding, B Kolachana, J Buckholtz, V S Mattay, Michael F Egan, and Daniel R. Weinberger.
			</span><br />
			<span class="venuetype"></span><span class="venue">Molecular psychiatry 11(9)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Meyer-Lindenberg2006').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Meyer-Lindenberg2006" class="abstract">
			Catechol-O-methyltransferase (COMT) has been shown to be critical for prefrontal dopamine flux, prefrontal cortex-dependent cognition and activation. Several potentially functional variants in the gene have been identified, but considerable controversy exists regarding the contribution of individual alleles and haplotypes to risk for schizophrenia, partly because clinical phenotypes are ill-defined and preclinical studies are limited by lack of adequate models. Here, we propose a neuroimaging approach to overcome these limitations by characterizing the functional impact of ambiguous haplotypes on a neural system-level intermediate phenotype in humans. Studying 126 healthy control subjects during a working-memory paradigm, we find that a previously described risk variant in a functional Val158Met (rs4680) polymorphism interacts with a P2 promoter region SNP (rs2097603) and an SNP in the 3&#39; region (rs165599) in predicting inefficient prefrontal working memory response. We report evidence that the nonlinear response of prefrontal neurons to dopaminergic stimulation is a neural mechanism underlying these nonadditive genetic effects. This work provides an in vivo approach to functional validation in brain of the biological impact of complex genetic variations within a gene that may be critical for its clinical association.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/16607471">Lesion probability maps of white matter hyperintensities in elderly individuals: results of the Austrian stroke prevention study.</a>.
			</span><br />
			<span class="authors">
				Christian Enzinger, Stephen M. Smith, Franz Fazekas, Gunther Drevin, Stefan Ropele, Thomas E. Nichols, Timothy E.J. Behrens, Reinhold Schmidt, and Paul M Matthews.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of neurology 253(8)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Enzinger2006').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Enzinger2006" class="abstract">
			White matter hyperintensities (WMH) are common on brain MRI of the elderly. Their size ranges from punctate to early confluent to confluent lesions. While this increase in extension is frequently seen as evidence for a continuum of changes, histological data and clinical follow-up suggest differences in underlying pathology and their progression.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/16099175">Non-white noise in fMRI: does modelling have an impact?</a>.
			</span><br />
			<span class="authors">
				Torben E Lund, Kristoffer H Madsen, Karam Sidaros, Wen-Lin Luo, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 29(1)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Lund2006').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Lund2006" class="abstract">
			The sources of non-white noise in Blood Oxygenation Level Dependent (BOLD) functional magnetic resonance imaging (fMRI) are many. Familiar sources include low-frequency drift due to hardware imperfections, oscillatory noise due to respiration and cardiac pulsation and residual movement artefacts not accounted for by rigid body registration. These contributions give rise to temporal autocorrelation in the residuals of the fMRI signal and invalidate the statistical analysis as the errors are no longer independent. The low-frequency drift is often removed by high-pass filtering, and other effects are typically modelled as an autoregressive (AR) process. In this paper, we propose an alternative approach: Nuisance Variable Regression (NVR). By inclusion of confounding effects in a general linear model (GLM), we first confirm that the spatial distribution of the various fMRI noise sources is similar to what has already been described in the literature. Subsequently, we demonstrate, using diagnostic statistics, that removal of these contributions reduces first and higher order autocorrelation as well as non-normality in the residuals, thereby improving the validity of the drawn inferences. In addition, we also compare the performance of the NVR method to the whitening approach implemented in SPM2.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 -->
			<span class="pubtitle">
				Diagnosis of Single-Subject and Group fMRI Data with SPMd.
			</span><br />
			<span class="authors">
				Hui Zhang, Wen-lin Luo, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Human Brain Mapping</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 -->
			<span class="pubtitle">
				Estimation efficiency and statistical power in arterial spin labeling fMRI.
			</span><br />
			<span class="authors">
				Jeanette A Mumford, Luis Hernandez-Garcia, Gregory R Lee, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811906001388">Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data</a>.
			</span><br />
			<span class="authors">
				Stephen M. Smith, Mark Jenkinson, Heidi Johansen-Berg, Daniel Rueckert, Thomas E. Nichols, Clare E Mackay, K.E. Watkins, Olga Ciccarelli, M.Z. Cader, Paul M. Matthews, and Timothy E.J. Behrens.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neuroimage 31(4)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://scholar.google.com/scholar?hl=en{\&amp;}btnG=Search{\&amp;}q=intitle:Modeling+and+Inference+of+Multisubject+fMRI+Data{\#}0">Modeling and inference of multisubject fMRI data</a>.
			</span><br />
			<span class="authors">
				Jeanette A Mumford and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">IEEE Engineering in Medicine and Biology Magazine 25(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20060007 --><big><strong>Books & Chapters</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.amazon.co.uk/Statistical-Parametric-Mapping-Analysis-Functional/dp/0123725607">Statistical Parametric Mapping: The Analysis of Functional Brain Images</a>.
			</span><br />
			<span class="authors">
				William D. Penny, Karl J. Friston, John T. Ashburner, Stefan J. Kiebel, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=book)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2005</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20050008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/16120776">Placebo effects mediated by endogenous opioid activity on mu-opioid receptors.</a>.
			</span><br />
			<span class="authors">
				Jon-Kar Zubieta, Joshua A Bueller, Lisa R Jackson, David J Scott, Yanjun Xu, Robert A Koeppe, Thomas E Nichols, and Christian S Stohler.
			</span><br />
			<span class="venuetype"></span><span class="venue">The Journal of Neuroscience 25(34)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Zubieta2005').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Zubieta2005" class="abstract">
			Reductions in pain ratings when administered a placebo with expected analgesic properties have been described and hypothesized to be mediated by the pain-suppressive endogenous opioid system. Using molecular imaging techniques, we directly examined the activity of the endogenous opioid system on mu-opioid receptors in humans in sustained pain with and without the administration of a placebo. Significant placebo-induced activation of mu-opioid receptor-mediated neurotransmission was observed in both higher-order and sub-cortical brain regions, which included the pregenual and subgenual rostral anterior cingulate, the dorsolateral prefrontal cortex, the insular cortex, and the nucleus accumbens. Regional activations were paralleled by lower ratings of pain intensity, reductions in its sensory and affective qualities, and in the negative emotional state of the volunteers. These data demonstrate that cognitive factors (e.g., expectation of pain relief) are capable of modulating physical and emotional states through the site-specific activation of mu-opioid receptor signaling in the human brain.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20050008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.ncbi.nlm.nih.gov/pubmed/15838181">Linkage disequilibrium: ancient history drives the new genetics.</a>.
			</span><br />
			<span class="authors">
				Gon{\c{c}}alo R Abecasis, Debashis Ghosh, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Human heredity 59(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Abecasis2005').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Abecasis2005" class="abstract">
			This brief review provides a summary of the biological causes of genetic association between tightly linked markers--termed linkage disequilibrium--and unlinked markers--termed population structure. We also review the utility of linkage disequilibrium data in gene mapping in isolated populations, in the estimation of recombination rates and in studying the history of particular alleles, including the detection of natural selection. We discuss current understanding of the extent and patterns of linkage disequilibrium in the genome, and its promise for genetic association studies in complex disease. Finally, we highlight the importance of using appropriate statistical procedures, such as the false discovery rate, to maximize the chances of success in large scale association studies.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20050008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811904005671">A comparison of random field theory and permutation methods for the statistical analysis of MEG data</a>.
			</span><br />
			<span class="authors">
				Dimitrios Pantazis, Thomas E. Nichols, Sylvain Baillet, and Richard M. Leahy.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20050008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811904007505">Valid conjunction inference with the minimum statistic</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, Matthew Brett, Jesper Andersson, Tor D. Wager, and Jean-Baptiste Poline.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neuroimage 25(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2004</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20040008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811904000862">Nonstationary cluster-size inference with random field and permutation methods</a>.
			</span><br />
			<span class="authors">
				Satoru Hayasaka, K Luan Phan, Israel Liberzon, Keith J Worsley, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 22(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20040008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811904002459">Combining voxel intensity and cluster extent with permutation test framework</a>.
			</span><br />
			<span class="authors">
				Satoru Hayasaka and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neuroimage 23(1)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20040005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				Statistical Surface-Based Morphometry Using a Non-parametric Approach.
			</span><br />
			<span class="authors">
				Dimitrios Pantazis, Richard M. Leahy, Thomas E. Nichols, and Martin Styner.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings IEEE International Symposium on Biomedical Imaging</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2003</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Temporal Resolution Properties of Dynamic PET Reconstructions.
			</span><br />
			<span class="authors">
				Evren Asma, Thomas E. Nichols, Richard M. Leahy, and A Log-likelihood Functions.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proc. IEEE Nuclear Science Symposium and Medical Imaging Conference</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 -->
			<span class="pubtitle">
				Switching attention and resolving interference: fMRI measures of executive functions.
			</span><br />
			<span class="authors">
				C Y Sylvester, Tor D. Wager, Steven C Lacey, Luis Hernandez-Garcia, Thomas E. Nichols, E.E. Smith, and John Jonides.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neuropsychologia 41(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811902000460">Optimization of experimental design in fMRI: a general framework using a genetic algorithm</a>.
			</span><br />
			<span class="authors">
				Tor D. Wager and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neuroimage 18(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.springerlink.com/index/FEALKLNX6BYQ3EYV.pdf">Spatiotemporal Localization of Significant Activation in MEG using Permutation Tests</a>.
			</span><br />
			<span class="authors">
				Dimitrios Pantazis, Thomas E. Nichols, Sylvain Baillet, and Richard M. Leahy.
			</span><br />
			<span class="venuetype"></span><span class="venue">Lecture Notes in Computer Science</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811903001496">Diagnosis and exploration of massively univariate neuroimaging models</a>.
			</span><br />
			<span class="authors">
				Wen-Lin Luo and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 19(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811903005020 http://www.ncbi.nlm.nih.gov/pubmed/14683734">Validating cluster size inference: random field and permutation methods.</a>.
			</span><br />
			<span class="authors">
				Satoru Hayasaka and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 20(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_haya:nich:2003').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_haya:nich:2003" class="abstract">
			Cluster size tests used in analyses of brain images can have more sensitivity compared to intensity based tests. The random field (RF) theory has been widely used in implementation of such tests, however the behavior of such tests is not well understood, especially when the RF assumptions are in doubt. In this paper, we carried out a simulation study of cluster size tests under varying smoothness, thresholds, and degrees of freedom, comparing RF performance to that of the permutation test, which is known to be exact. For Gaussian images, we find that the RF methods are generally conservative, especially for low smoothness and low threshold. For t images, the RF tests are found to be conservative at lower thresholds and do not perform well unless the threshold is high and images are sufficiently smooth. The permutation test performs well for any settings though the discreteness in cluster size must be accounted for. We make specific recommendations on when permutation tests are to be preferred to RF tests.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030008 -->
			<span class="pubtitle">
				Controlling the familywise error rate in functional neuroimaging: a comparative review.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and Satoru Hayasaka.
			</span><br />
			<span class="venuetype"></span><span class="venue">Statistical Methods in Medical Research 12(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20030007 --><big><strong>inbook</strong></big><br />
			
			<span class="pubtitle">
				7, ``Modul.
			</span><br />
			<span class="authors">
				John Jonides, C Y Sylvester, Steven C Lacey, Tor D. Wager, Thomas E. Nichols, and E Awh.
			</span><br />
			<span class="venuetype">Chapter in </span><span class="venue">{Principles of Learning and Memory}</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2002</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20020008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Statistical analysis of multiplex brain gene expression datasets.
			</span><br />
			<span class="authors">
				A Ossadtchi, V M Brown, A H Khan, S R Cherry, Thomas E. Nichols, Richard M. Leahy, and D J Smith.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neurochemical Research 27(10)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20020008 -->
			<span class="pubtitle">
				Spatiotemporal Reconstruction of List Mode PET Data.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, J Qi, Evren Asma, and Richard M. Leahy.
			</span><br />
			<span class="venuetype"></span><span class="venue">IEEE Transactions on Medical Imaging 21(4)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20020008 -->
			<span class="pubtitle">
				Variable Threshold Methods for Functional Neuroimaging.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and Randy L. Buckner.
			</span><br />
			<span class="venuetype"></span><span class="venue">Presented at the 8th International Conference on Functional Mapping of the Human Brain, June 2-6, 2002, Sendai, Japan. Available on CD-Rom in NeuroImage</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20020008 -->
			<span class="pubtitle">
				Gender differences in patterns of cerebral activation during equal experience of painful laser stimulation..
			</span><br />
			<span class="authors">
				S W G Derbyshire, Thomas E. Nichols, Anthony Jones, D. W. Townsend, and L Firestone.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Pain 3(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20020008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://linkinghub.elsevier.com/retrieve/pii/S1053811901910377">Thresholding of Statistical Maps in Functional Neuroimaging Using the False Discovery Rate</a>.
			</span><br />
			<span class="authors">
				Christopher R Genovese, Nicole A Lazar, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 15(4)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20020008 -->
			<span class="pubtitle">
				Nonparametric permutation tests for functional neuroimaging: A primer with examples.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and Andrew P Holmes.
			</span><br />
			<span class="venuetype"></span><span class="venue">Human Brain Mapping 15(1)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>2001</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 20010008 -->
			<span class="pubtitle">
				4D PET Image Reconstruction from List Mode Data.
			</span><br />
			<span class="authors">
				Evren Asma, Thomas E. Nichols, and Richard M. Leahy.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proc. IEEE Nuclear Science Symposium and Medical Imaging Conference</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20010008 -->
			<span class="pubtitle">
				Data Exploration Through Model Diagnosis.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and Wen-Lin Luo.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 13(6 (2/2))</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 20010008 -->
			<span class="pubtitle">
				Activation of the Medial Prefrontal Cortex and Extended Amygdala by Individual Ratings of Emotional Arousal: An f\MRI\ Study.
			</span><br />
			<span class="authors">
				K Luan Phan, Stephan F Taylor, R C Welsh, L R Decker, D C Noll, Thomas E. Nichols, J C Britton, and Israel Liberzon.
			</span><br />
			<span class="venuetype"></span><span class="venue">Biological Psychiatry</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1999</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19990008 -->
			<span class="pubtitle">
				Comparative Evaluation of \MR\-Based Partial Volume Correction Schemes for \PET\.
			</span><br />
			<span class="authors">
				C C Meltzer, Paul E. Kinahan, P J Greer, Thomas E. Nichols, C Comtat, M N Cantwell, and J C Price.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 40(12)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19990008 -->
			<span class="pubtitle">
				Changes in forebrain function from Waking to REM sleep in depression: Preliminary Analyses of [18F]-FDG PET studies.
			</span><br />
			<span class="authors">
				E A Nofzinger, Thomas E. Nichols, D J Kupfer, and R Y Moore.
			</span><br />
			<span class="venuetype"></span><span class="venue">Psychiatry Research: Neuroimaging 91(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19990008 -->
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1692643">Statistical limitations in functional neuroimaging II. Signal detection and statistical inference</a>.
			</span><br />
			<span class="authors">
				Karl Magnus Petersson, Thomas E. Nichols, Jean-Baptiste Poline, and Andrew P Holmes.
			</span><br />
			<span class="venuetype"></span><span class="venue">Philosophical Transactions of the Royal Society B: Biological Sciences 354(354)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19990008 -->
			<span class="pubtitle">
				Statistical limitations in functional neuroimaging I. Non-inferential methods and statistical models.
			</span><br />
			<span class="authors">
				Karl Magnus Petersson, Thomas E. Nichols, Jean-Baptiste Poline, and Andrew P Holmes.
			</span><br />
			<span class="venuetype"></span><span class="venue">Philosophical Transactions of the Royal Society B: Biological Sciences</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19990005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				Continuous Time Dynamic PET Imaging Using List Mode Data.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, J Qi, and Richard M. Leahy.
			</span><br />
			<span class="venuetype"></span><span class="venue">Information Processing in Medical Imaging (Lecture Notes in Computer Science)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19990005 -->
			<span class="pubtitle">
				Continuous Time Dynamic Positron Emission Tomography (\PET\).
			</span><br />
			<span class="authors">
				Thomas E. Nichols, J Qi, and Richard M. Leahy.
			</span><br />
			<span class="venuetype"></span><span class="venue">Joint Statistical Meetings Abstracts</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19990005 -->
			<span class="pubtitle">
				Extensible Framework for Nonparametric Analysis of Neuroimaging Data.
			</span><br />
			<span class="authors">
				Thomas E. Nichols and Andrew P Holmes.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1998</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19980008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Test-retest variability of positron emission tomography (PET) and [18F]-altanserin for the in vivo imaging of serotonin (5-HT2A) receptors in the human brain.
			</span><br />
			<span class="authors">
				G S Smith, J C Price, B J Lopresti, Y Huang, N Simpson, D. Holt, N S Mason, C C Meltzer, R Sweet, Thomas E. Nichols, D Sashin, and C Mathis.
			</span><br />
			<span class="venuetype"></span><span class="venue">Synapse</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19980008 -->
			<span class="pubtitle">
				Holmes and Watson on 'Sherlock'.
			</span><br />
			<span class="authors">
				Andrew P Holmes, J D G Watson, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Cerebral Blood Flow {\&} Metabolism 18(6)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1997</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19970008 -->
			<span class="pubtitle">
				PET brain mapping study of auditory verbal supraspan memory versus visual fixation in schizophrenia.
			</span><br />
			<span class="authors">
				R Ganguli, C S Carter, Mark a Mintun, J Brar, J T Becker, R Sarma, Thomas E. Nichols, and E Bennington.
			</span><br />
			<span class="venuetype"></span><span class="venue">Biological Psychiatry 41(1)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19970008 -->
			<span class="pubtitle">
				Anterior Cingulate Gyrus Dysfunction And Selective Attentional Dysfunction in Schizophrenia: An [15O]-H2O PET Study During Stroop Task Performance.
			</span><br />
			<span class="authors">
				C S Carter, Mark a Mintun, Thomas E. Nichols, and Jonathan D Cohen.
			</span><br />
			<span class="venuetype"></span><span class="venue">American Journal of Psychiatry 154(12)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19970005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				Whole Brain Model Selection With \PET\ Activation Data.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, S Land, and William F Eddy.
			</span><br />
			<span class="venuetype"></span><span class="venue">Joint Statistical Meetings Abstracts</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1996</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Alterations in Functional Neuroanatomical Connectivity in Alzheimer's Disease - Positron Emission Tomography of Auditory Verbal Short-Term Memory.
			</span><br />
			<span class="authors">
				J T Becker, Mark a Mintun, K Aleva, M B Wiseman, Thomas E. Nichols, and S T Dekosky.
			</span><br />
			<span class="venuetype"></span><span class="venue">Annals of the New York Academy of Sciences 777(412)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				Positron emission tomographic imaging of serotonin activation effects on prefrontal cortex in healthy volunteers.
			</span><br />
			<span class="authors">
				J J Mann, K M Malone, D J Diehl, J Perel, Thomas E. Nichols, and Mark a Mintun.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Cerebral Blood Flow {\&} Metabolism 16(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				Functional connectivity in auditory-verbal short-term memory in Alzheimer's disease.
			</span><br />
			<span class="authors">
				A N Herbster, Thomas E. Nichols, M B Wiseman, Mark a Mintun, S T DeKosky, and J T Becker.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 4(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				The effect of intra-study alignment on model fit for dynamic neuroreceptor ligand images.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, F Jadali, A.M. Smith, G Smith, and Mark a Mintun.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 37(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				Working Towards an Automatic and Accurate Method for \PET\-\MR\ Alignment.
			</span><br />
			<span class="authors">
				M B Wiseman, Thomas E. Nichols, M A Dachille, and Mark a Mintun.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 37(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				Alterations in functional neuroanatomical connectivity in Alzheimer's disease. Positron emission tomography of auditory verbal short-term memory.
			</span><br />
			<span class="authors">
				J T Becker, Mark a Mintun, K Aleva, M B Wiseman, Thomas E. Nichols, and S T Dekosky.
			</span><br />
			<span class="venuetype"></span><span class="venue">Annals of the New York Academy of Sciences</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				The State of the \ART\.
			</span><br />
			<span class="authors">
				D. W. Townsend, S W G Derbyshire, S Clark, Thomas E. Nichols, M A Dachille, J A Ruszkiewicz, D L Milko, F Gyulai, L Firestone, Mark a Mintun, and Anthony Jones.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 37(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				Compensatory reallocation of brain resources supporting verbal episodic memory in Alzheimer's disease.
			</span><br />
			<span class="authors">
				J T Becker, Mark a Mintun, K Aleva, M B Wiseman, Thomas E. Nichols, and S T DeKosky.
			</span><br />
			<span class="venuetype"></span><span class="venue">Neurology 46(3)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960008 -->
			<span class="pubtitle">
				Anterior Cingulate Gyrus Dysfunction And Attentional Pathology in Schizophrenia.
			</span><br />
			<span class="authors">
				C S Carter, Mark a Mintun, Jonathan D Cohen, Thomas E. Nichols, and M B Wiseman.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 3(3,{\~{}}2/2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://link.aip.org/link/?PSISDG/2629/197/1">Laser stimulation for pain research</a>.
			</span><br />
			<span class="authors">
				Stuart Clark, Mark R. Dickinson, Terence A. King, Anthony Jones, Andrew Chen, Stuart Derbyshire, D. W. Townsend, Paul E. Kinahan, Mark a Mintun, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings of SPIE</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Clark1996').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Clark1996" class="abstract">
			Pain is a serious medical problem; it inflicts huge economic loss and personal suffering. Pain signals are conducted via small, non- and partially myelinated A-delta and C nerve fibers and lasers are particularly well suited to stimulating these fibers. Large myelinated fibers convey touch and vibration information and these fibers are also discharged when contact thermodes and other touch pain stimuli are used and this would give a more muddled signal for functional imaging experiments. The advantages of lasers over conventional methods of pain stimulation are good temporal resolution, no variable parameters are involved such as contact area and they give very reproducible results. Accurate inter-stimulus changes can be achieved by computer control of the laser pulse duration, pulse height and repetition rate and this flexibility enables complex stimulation paradigms to be realized. We present a flexible carbon dioxide laser system designed to generate these stimuli for the study of human cerebral pain responses. We discuss the advantages within research of this system over other methods of pain stimulation such as thermal, electrical and magnetic. The stimulator is used in conjunction with functional magnetic resonance imaging, positron emission tomography and electrophysiological methods of imaging the brain&#39;s activity. This combination is a powerful tool for the study of pain-induced activity in different areas of the brain. An accurate understanding of the brain&#39;s response to pain will help in research into the areas of rheumatoid arthritis and chronic back pain.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19960005 -->
			<span class="pubtitle">
				Meta-Analysis of Auditory Verbal Short-Term Memory Using Positron Emission Tomography.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, A Palmer, and J T Becker.
			</span><br />
			<span class="venuetype"></span><span class="venue">48th Annual Meeting of the American Academy of Neurology</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1995</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19950008 --><big><strong>Articles</strong></big><br />
			
			<span class="pubtitle">
				Stereotaxic techniques comparing foci intensity and location of activation areas in the brain as obtained using positron emission tomography (\PET\).
			</span><br />
			<span class="authors">
				M B Wiseman, Thomas E. Nichols, R P Woods, John A Sweeney, and Mark a Mintun.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 36(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19950008 -->
			<span class="pubtitle">
				Effect of resolution on recovery of gray matter activity in \FDG\ \PET\.
			</span><br />
			<span class="authors">
				C C Meltzer, Thomas E. Nichols, and Mark a Mintun.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 36(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19950008 -->
			<span class="pubtitle">
				Scatter Correction in \PET\ Activation Studies.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, D. W. Townsend, T Braver, Jonathan D Cohen, and Mark a Mintun.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage 2(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19950008 -->
			<span class="pubtitle">
				Use of principal component analysis in \PET\ cognitive activation studies.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, M B Wiseman, Mark a Mintun, and John A Sweeney.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Nuclear Medicine 36(5)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19950008 -->
			<span class="pubtitle">
				Comparison of Functional Magnetic Resonance Imaging with Positron Emission Tomography and Magnetoencephalography to Identify the Motor Cortex in a Patient with an Arteriovenous Malformation.
			</span><br />
			<span class="authors">
				S B Baumann, D C Noll, D S Kondziolka, W Schneider, Thomas E. Nichols, Mark a Mintun, J D Lewine, H Yonas, W W Orrison, and R J Sclabassi.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of Image Guided Surgery 1(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_Baumann:95').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_Baumann:95" class="abstract">
			Alterations in gyral contour made it difficult to identify the motor cortex thought to be near an arteriovenous malformation (AVM) in a 24-year-old man considered for stereotactic radiosurgery. Functional imaging in three modalities was performed preoperatively to compare the reliability of localization using functional magnetic resonance imaging (fMRI) on a conventional scanner with positron emission tomography (PET) and magnetoencephalography (MEG). Similar tasks were used for each imaging modality in an attempt to activate and identify the sensory and motor cortex. Data from all three modalities converged for the sensory task, and fMRI and PET data converged for the motor task. The right hemisphere motor strip was localized adjacent and anterior to the AVM. These data were used in planning the radiosurgery isodose configuration to the AVM in order to reduce the irradiation of motor cortex parenchyma. A postoperative fMRI study was also performed using newer techniques to reduce head motion artifact and to improve signal-to-noise ratio. The data confirmed the conclusions derived from the preoperative evaluations. This study demonstrates how conventional MRI scanners can be used for functional studies of use in surgical planning.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19950008 -->
			<span class="pubtitle">
				Diurnal variation in baseline human regional cerebral blood flow demonstrated by positron emission tomography: Preliminary findings.
			</span><br />
			<span class="authors">
				D J Diehl, Mark a Mintun, R Y Moore, Thomas E. Nichols, and D J Kupfer.
			</span><br />
			<span class="venuetype"></span><span class="venue">Psychiatry Research: Neuroimaging</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1994</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19940008 -->
			<span class="pubtitle">
				Post-ECT increases in MRI regional T2 relaxation times and their relationship to cognitive side effects: a pilot study.
			</span><br />
			<span class="authors">
				D J Diehl, M S Keshavan, E Kanal, R D Nebes, Thomas E. Nichols, and J S Gillen.
			</span><br />
			<span class="venuetype"></span><span class="venue">Psychiatry Research 54(2)</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date"><big><strong>1993</strong></big><br />
						
			</span>
		</td>
		<td class="publication">
			<!-- 19930005 --><big><strong>inproceedings</strong></big><br />
			
			<span class="pubtitle">
				PET imaging of serotonin activation of prefrontal cortex.
			</span><br />
			<span class="authors">
				K M Malone, Mark a Mintun, D J Diehl, Thomas E. Nichols, and J J Mann.
			</span><br />
			<span class="venuetype"></span><span class="venue">American College of Neuropsychopharmacology 32nd Annual Meeting</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">		
			</span>
		</td>
		<td class="publication">
			<!-- 19930005 -->
			<span class="pubtitle">
				Brain glucose uptake changes with total sleep deprivation in normals.
			</span><br />
			<span class="authors">
				S Dube, Mark a Mintun, Thomas E. Nichols, D Mesiano, P Houck, D J Kupfer, and C F {Reynolds III}.
			</span><br />
			<span class="venuetype"></span><span class="venue">American College of Neuropsychopharmacology 32nd Annual Meeting</span>.
			<br />
			<span class="links">
			</span>
		</td>
	</tr>
</tbody>
</table>
