<table class="table">
<tbody>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2017</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/84709/">Reply to Brown and Behrmann, Cox, et al., and Kessler et al. : Data and code sharing is the way forward for fMRI</a>.
			</span><br />
			<span class="authors">
				A. Eklund, Thomas E. Nichols, and H. Knutsson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=misc)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap84709').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap84709" class="abstract">
			We are glad that our paper (1) has generated intense discussions in the fMRI field (2??4), on how to analyze fMRI data, and how to correct for multiple comparisons. The goal of the paper was not to disparage any specific fMRI software, but to point out that parametric statistical methods are based on a number of assumptions that are not always valid for fMRI data, and that nonparametric statistical methods (5) are a good alternative. Through AFNI?s introduction of nonparametric statistics in the function 3dttest++ (3, 6), the three most common fMRI softwares now all support nonparametric group inference [SPM through the toolbox SnPM (www2.warwick.ac.uk/fac/sci/statistics/staff/academic-research/nichols/software/snpm), and FSL through the function randomise].
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/85042/">Best practices in data analysis and sharing in neuroimaging using MRI</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, Samir Das, Simon B. Eickhoff, Alan C. Evans, Tristan Glatard, Michael Hanke, Nikolaus Kriegeskorte, Michael P. Milham, Russell A. Poldrack, Jean-Baptiste Poline, Erika Proal, Bertrand Thirion, David C. Van Essen, Tonya White, and B.T. Thomas Yeo.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature Neuroscience 20(3)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap85042').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap85042" class="abstract">
			Given concerns about the reproducibility of scientific findings, neuroimaging must define best 28 practices for data analysis, results reporting, and algorithm and data sharing to promote 29 transparency, reliability and collaboration. We describe insights from developing a set of 30 recommendations on behalf of the Organization for Human Brain Mapping, and identify barriers 31 that impede these practices, including how the discipline must change to fully exploit the 32 potential of the world?s neuroimaging data.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/85365/">A defense of using resting state fMRI as null data for estimating false positive rates</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols, Anders Eklund, and Hans Knutsson.
			</span><br />
			<span class="venuetype"></span><span class="venue">Cognitive Neuroscience</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap85365').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap85365" class="abstract">
			A recent Editorial by Slotnick (2017) reconsiders the findings of our paper on the accuracy of false positive rate control with cluster inference in fMRI (Eklund et al, 2016), in particular criticising our use of resting state fMRI data as a source for null data in the evaluation of task fMRI methods. We defend this use of resting fMRI data, as while there is much structure in this data, we argue it is representative of task data noise and as such analysis software should be able to accommodate this noise. We also discuss a potential problem with Slotnick?s own method.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/84873/">Scanning the horizon : towards transparent and reproducible neuroimaging research</a>.
			</span><br />
			<span class="authors">
				Russell A. Poldrack, Chris I. Baker, Joke Durnez, Krzysztof J. Gorgolewski, Paul M. Matthews, Marcus R. Munaf{\`o}, Thomas E. Nichols, Jean-Baptiste Poline, Edward Vul, and Tal Yarkoni.
			</span><br />
			<span class="venuetype"></span><span class="venue">Nature Reviews Neuroscience</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap84873').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap84873" class="abstract">
			Functional neuroimaging techniques have transformed our ability to probe the neurobiological basis of behaviour and are increasingly being applied by the wider neuroscience community. However, concerns have recently been raised that the conclusions that are drawn from some human neuroimaging studies are either spurious or not generalizable. Problems such as low statistical power, flexibility in data analysis, software errors and a lack of direct replication apply to many fields, but perhaps particularly to functional MRI. Here, we discuss these problems, outline current and suggested best practices, and describe how we think the field should evolve to produce the most meaningful and reliable answers to neuroscientific questions.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2016</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/84382/">Sharing brain mapping statistical results with the neuroimaging data model</a>.
			</span><br />
			<span class="authors">
				Camille Maumet, Tibor Auer, Alexander Bowring, Gang Chen, Samir Das, Guillaume Flandin, Satrajit Ghosh, Tristan Glatard, Krzysztof J. Gorgolewski, Karl G. Helmer, Mark Jenkinson, David B. Keator, B. Nolan Nichols, Jean-Baptiste Poline, Richard Reynolds, Vanessa Sochat, Jessica Turner, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Scientific Data</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap84382').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap84382" class="abstract">
			Only a tiny fraction of the data and metadata produced by an fMRI study is finally conveyed to the community. This lack of transparency not only hinders the reproducibility of neuroimaging results but also impairs future meta-analyses. In this work we introduce NIDM-Results, a format specification providing a machine-readable description of neuroimaging statistical results along with key image data summarising the experiment. NIDM-Results provides a unified representation of mass univariate analyses including a level of detail consistent with available best practices. This standardized representation allows authors to relay methods and results in a platform-independent regularized format that is not tied to a particular neuroimaging software package. Tools are available to export NIDM-Result graphs and associated files from the widely used SPM and FSL software packages, and the NeuroVault repository can import NIDM-Results archives. The specification is publically available at: http://nidm.nidash.org/specs/nidm-results.html.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/83529/">Variance decomposition for single-subject task-based fMRI activity estimates across many sessions</a>.
			</span><br />
			<span class="authors">
				Javier Gonzalez-Castillo, Gang Chen, Thomas E. Nichols, and Peter A. Bandettini.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap83529').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap83529" class="abstract">
			Here we report an exploratory within-subject variance decomposition analysis conducted on a task-based fMRI dataset with an unusually large number of repeated measures (i.e., 500 trials in each of three different subjects) distributed across 100 functional scans and 9 to 10 different sessions. Within-subject variance was segregated into four primary components: variance across-sessions, variance across-runs within a session, variance across-blocks within a run, and residual measurement/modeling error. Our results reveal inhomogeneous and distinct spatial distributions of these variance components across significantly active voxels in grey matter. Measurement error is dominant across the whole brain. Detailed evaluation of the remaining three components shows that across-session variance is the second largest contributor to total variance in occipital cortex, while across-runs variance is the second dominant source for the rest of the brain. Network-specific analysis revealed that across-block variance contributes more to total variance in higher-order cognitive networks than in somatosensory cortex. Moreover, in some higher-order cognitive networks across-block variance can exceed across-session variance. These results help us better understand the temporal (i.e., across blocks, runs and sessions) and spatial distributions (i.e., across different networks) of within-subject natural variability in estimates of task responses in fMRI. They also suggest that different brain regions will show different natural levels of test-retest reliability even in the absence of residual artifacts and sufficiently high contrast-to-noise measurements. Further confirmation with a larger sample of subjects and other tasks is necessary to ensure generality of these results.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/81170/">Behavior, sensitivity, and power of activation likelihood estimation characterized by massive empirical simulation</a>.
			</span><br />
			<span class="authors">
				Simon B. Eickhoff, Thomas E. Nichols, Angela R. Laird, Felix Hoffstaedter, Katrin Amunts, Peter T. Fox, Danilo Bzdok, and Claudia R. Eickhoff.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap81170').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap81170" class="abstract">
			Given the increasing number of neuroimaging publications, the automated knowledge extraction on brain-behavior associations by quantitative meta-analyses has become a highly important and rapidly growing field of research. Among several methods to perform coordinate-based neuroimaging meta-analyses, Activation Likelihood Estimation (ALE) has been widely adopted. In this paper, we addressed two pressing questions related to ALE meta-analysis: i) Which thresholding method is most appropriate to perform statistical inference? ii) Which sample size, i.e., number of experiments, is needed to perform robust meta-analyses? We provided quantitative answers to these questions by simulating more than 120,000 meta-analysis datasets using empirical parameters (i.e., number of subjects, number of reported foci, distribution of activation foci) derived from the BrainMap database. This allowed to characterize the behavior of ALE analyses, to derive first power estimates for neuroimaging meta-analyses, and to thus formulate recommendations for future ALE studies. We could show as a first consequence that cluster-level family-wise error (FWE) correction represents the most appropriate method for statistical inference, while voxel-level FWE correction is valid but more conservative. In contrast, uncorrected inference and false-discovery rate correction should be avoided. As a second consequence, researchers should aim to include at least 20 experiments into an ALE meta-analysis to achieve sufficient power for moderate effects. We would like to note, though, that these calculations and recommendations are specific to ALE and may not be extrapolated to other approaches for (neuroimaging) meta-analysis.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/78399/">Modelling the penumbra in computed tomography</a>.
			</span><br />
			<span class="authors">
				Audrey Kueh, Jason M. Warnett, Gregory John Gibbons, Julia Brettschneider, Thomas E. Nichols, M. A. Williams, and Wilfrid S. Kendall.
			</span><br />
			<span class="venuetype"></span><span class="venue">Journal of X-Ray Science and Technology 24(4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap78399').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap78399" class="abstract">
			Background: In computed tomography (CT), the spot geometry is one of the main sources of error in CT images. Since X-rays do not arise from a point source, artefacts are produced. In particular there is a penumbra effect, leading to poorly defined edges within a reconstructed volume. Penumbra models can be simulated given a fixed spot geometry and the known experimental setup. Objective: This paper proposes to use a penumbra model, derived from Beer?s law, both to confirm spot geometry from penumbra data, and to quantify blurring in the image. Methods: Two models for the spot geometry are considered; one consists of a single Gaussian spot, the other is a mixture model consisting of a Gaussian spot together with a larger uniform spot. Results: The model consisting of a single Gaussian spot has a poor fit at the boundary. The mixture model (which adds a larger uniform spot) exhibits a much improved fit. The parameters corresponding to the uniform spot are similar across all powers, and further experiments suggest that the uniform spot produces only soft X-rays of relatively low-energy. Conclusions: Thus, the precision of radiographs can be estimated from the penumbra effect in the image. The use of a thin copper filter reduces the size of the effective penumbra.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/80232/">Exploring fMRI Results Space : 31 Variants of an fMRI Analysis in AFNI, FSL, and SPM</a>.
			</span><br />
			<span class="authors">
				Ruth Pauli, Alexander Bowring, Richard Reynolds, Gang Chen, Thomas E. Nichols, and Camille Maumet.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in neuroinformatics</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap80232').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap80232" class="abstract">
			Data sharing is becoming a priority in functional Magnetic Resonance Imaging (fMRI) research, but the lack of a standard format for shared data is an obstacle (Poline et al., 2012; Poldrack and Gorgolewski, 2014). This is especially true for information about data provenance, including auxiliary information such as participant characteristics and task descriptions. The three most commonly used analysis software packages [AFNI1 (Cox, 1996), FSL2 (Jenkinson et al., 2012), and SPM3 (Penny et al., 2011)] broadly conduct the same analysis, but differ in how fundamental concepts are described, and have a myriad of differences in the pre-processing and modeling steps. The practical consequence is that sharing analyzed data is further complicated by the idiosyncrasies of the particular software used.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/80355/">The brain imaging data structure, a format for organizing and describing outputs of neuroimaging experiments</a>.
			</span><br />
			<span class="authors">
				Krzysztof J. Gorgolewski, Tibor Auer, Vince D. Calhoun, R. Cameron Craddock, Samir Das, Eugene P. Duff, Guillaume Flandin, Satrajit S. Ghosh, Tristan Glatard, Yaroslav O. Halchenko, Daniel A. Handwerker, Michael Hanke, David Keator, Xiangrui Li, Zachary Michael, Camille Maumet, B. Nolan Nichols, Thomas E. Nichols, John Pellman, Jean-Baptiste Poline, Ariel Rokem, Gunnar Schaefer, Vanessa V. Sochat, William Triplett, Jessica A. Turner, Ga{\"e}l Varoquaux, and Russell A. Poldrack.
			</span><br />
			<span class="venuetype"></span><span class="venue">Scientific Data</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap80355').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap80355" class="abstract">
			The development of magnetic resonance imaging (MRI) techniques has defined modern neuroimaging. Since its inception, tens of thousands of studies using techniques such as functional MRI and diffusion weighted imaging have allowed for the non-invasive study of the brain. Despite the fact that MRI is routinely used to obtain data for neuroscience research, there has been no widely adopted standard for organizing and describing the data collected in an imaging experiment. This renders sharing and reusing data (within or between labs) difficult if not impossible and unnecessarily complicates the application of automatic pipelines and quality assurance protocols. To solve this problem, we have developed the Brain Imaging Data Structure (BIDS), a standard for organizing and describing MRI datasets. The BIDS standard uses file formats compatible with existing software, unifies the majority of practices already common in the field, and captures the metadata necessary for most common data processing operations.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/76502/">Non-parametric combination and related permutation tests for neuroimaging</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Matthew A. Webster, Jonathan C. Brooks, Irene Tracey, Stephen M. Smith, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Human Brain Mapping</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap76502').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap76502" class="abstract">
			In this work, we show how permutation methods can be applied to combination analyses such as those that include multiple imaging modalities, multiple data acquisitions of the same modality, or simply multiple hypotheses on the same data. Using the well-known definition of union-intersection tests and closed testing procedures, we use synchronized permutations to correct for such multiplicity of tests, allowing flexibility to integrate imaging data with different spatial resolutions, surface and/or volume-based representations of the brain, including non-imaging data. For the problem of joint inference, we propose and evaluate a modification of the recently introduced non-parametric combination (NPC) methodology, such that instead of a two-phase algorithm and large data storage requirements, the inference can be performed in a single phase, with reasonable computational demands. The method compares favorably to classical multivariate tests (such as MANCOVA), even when the latter is assessed using permutations. We also evaluate, in the context of permutation tests, various combining methods that have been proposed in the past decades, and identify those that provide the best control over error rate and power across a range of situations. We show that one of these, the method of Tippett, provides a link between correction for the multiplicity of tests and their combination. Finally, we discuss how the correction can solve certain problems of multiple comparisons in one-way ANOVA designs, and how the combination is distinguished from conjunctions, even though both can be assessed using permutation tests. We also provide a common algorithm that accommodates combination and correction.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2015</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/67560/">NeuroVault.org : a web-based repository for collecting and sharing unthresholded statistical maps of the human brain</a>.
			</span><br />
			<span class="authors">
				Krzysztof J. Gorgolewski, Gael Varoquaux, Gabriel Rivera, Yannick Schwarz, Satrajit S. Ghosh, Camille Maumet, Vanessa V. Sochat, Thomas E. Nichols, Russell A. Poldrack, Jean-Baptiste Poline, Tal Yarkoni, and Daniel Margulies.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in neuroinformatics</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap67560').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap67560" class="abstract">
			Here we present NeuroVault-a web based repository that allows researchers to store, share, visualize, and decode statistical maps of the human brain. NeuroVault is easy to use and employs modern web technologies to provide informative visualization of data without the need to install additional software. In addition, it leverages the power of the Neurosynth database to provide cognitive decoding of deposited maps. The data are exposed through a public REST API enabling other services and tools to take advantage of it. NeuroVault is a new resource for researchers interested in conducting meta- and coactivation analyses.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/65654/">Massively expedited genome-wide heritability analysis (MEGHA)</a>.
			</span><br />
			<span class="authors">
				Tian Ge, Thomas E. Nichols, Phil H. Lee, Avram J. Holmes, Joshua L. Roffman, Randy L. Buckner, Mert R. Sabuncu, and Jordan W. Smoller.
			</span><br />
			<span class="venuetype"></span><span class="venue">Proceedings of the National Academy of Sciences of the United States of America Volume 112(Number 8)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap65654').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap65654" class="abstract">
			The discovery and prioritization of heritable phenotypes is a computational challenge in a variety of settings, including neuroimaging genetics and analyses of the vast phenotypic repositories in electronic health record systems and population-based biobanks. Classical estimates of heritability require twin or pedigree data, which can be costly and difficult to acquire. Genome-wide Complex Trait Analysis (GCTA) is an alternative tool to compute heritability estimates from unrelated individuals, using genome-wide data that is increasingly ubiquitous, but is computationally demanding and becomes difficult to apply in evaluating very large numbers of phenotypes. Here we present a novel, fast and accurate statistical method for high-dimensional heritability analysis using genome-wide single nucleotide polymorphism (SNP) data from unrelated individuals, termed Massively Expedited Genome-wide Heritability Analysis (MEGHA), and accompanying nonparametric sampling techniques that enable flexible inferences for arbitrary statistics of interest. MEGHA produces estimates and significance measures of heritability with several orders of magnitude less computational time than existing methods, making heritability-based prioritization of millions of phenotypes based on data from unrelated individuals tractable for the first time. As a demonstration of application, we conducted heritability analyses on global and local morphometric measurements derived from brain structural magnetic resonance imaging (MRI) scans, using genome-wide SNP data from 1,320 unrelated young healthy adults of non-Hispanic European ancestry. We also computed surface maps of heritability for cortical thickness measures and empirically localized cortical regions where thickness measures were significantly heritable. Our analyses demonstrate the unique capability of MEGHA for large-scale heritability-based screening and high-dimensional heritability profile construction.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/65285/">Searching multiregression dynamic models of resting-state fMRI networks using integer programming</a>.
			</span><br />
			<span class="authors">
				Lilia Costa, James Q. Smith, Thomas E. Nichols, James Cussens, Eugene P. Duff, and Tamar R. Makin.
			</span><br />
			<span class="venuetype"></span><span class="venue">Bayesian Analysis 10(2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap65285').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap65285" class="abstract">
			A Multiregression Dynamic Model (MDM) is a class of multivariate time series that represents various dynamic causal processes in a graphical way. One of the advantages of this class is that, in contrast to many other Dynamic Bayesian Networks, the hypothesised relationships accommodate conditional conjugate inference. We demonstrate for the first time how straightforward it is to search over all possible connectivity networks with dynamically changing intensity of transmission to find the MAP model within this class. This search method is made feasible by using a novel application of an Integer Programming algorithm. The efficacy of applying this particular class of dynamic models to this domain is shown and more specifically the computational efficiency of a corresponding search of 11-node DAG model space. We proceed to show how diagnostic methods, analogous to those defined for static Bayesian Networks, can be used to suggest embellishment of the model class to extend the process of model selection. All methods are illustrated using simulated and real resting-state functional Magnetic Resonance Imaging (fMRI) data.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2014</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/63790/">Correction: Global genetic variations predict brain response to faces</a>.
			</span><br />
			<span class="authors">
				Erin W. Dickie, Amir Tahmasebi, Leon French, Natasa Kovacevic, Tobias Banaschewski, Gareth J. Barker, Arun Bokde, Christian B{\"u}chel, Patricia J. Conrod, Herta Flor, Hugh Garavan, Ju?rgen Gallinat, Penny Gowland, Andreas Heinz, Bernd Ittermann, Claire Lawrence, Karl Mann, Jean-Luc Martinot, Frauke Nees, Thomas E. Nichols, Mark Lathrop, Eva Loth, Zdenka Pausova, Marcela Rietschel, Michal N. Smolka, Andreas Str{\"o}hle, Roberto Toro, Gunter Schumann, and Tom{\'a}{\vs} Paus.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS Genetics Volume 10(Number 10)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap63790').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap63790" class="abstract">
			Face expressions are a rich source of social signals. Here we estimated the proportion of phenotypic variance in the brain response to facial expressions explained by common genetic variance captured by {\texttt{\char126}}500,000 single nucleotide polymorphisms. Using genomic-relationship-matrix restricted maximum likelihood (GREML), we related this global genetic variance to that in the brain response to facial expressions, as assessed with functional magnetic resonance imaging (fMRI) in a community-based sample of adolescents (n = 1,620). Brain response to facial expressions was measured in 25 regions constituting a face network, as defined previously. In 9 out of these 25 regions, common genetic variance explained a significant proportion of phenotypic variance (40?50\%) in their response to ambiguous facial expressions; this was not the case for angry facial expressions. Across the network, the strength of the genotype-phenotype relationship varied as a function of the inter-individual variability in the number of functional connections possessed by a given region (R2 = 0.38, p{\ensuremath{&lt;}}0.001). Furthermore, this variability showed an inverted U relationship with both the number of observed connections (R2 = 0.48, p{\ensuremath{&lt;}}0.001) and the magnitude of brain response (R2 = 0.32, p{\ensuremath{&lt;}}0.001). Thus, a significant proportion of the brain response to facial expressions is predicted by common genetic variance in a subset of regions constituting the face network. These regions show the highest inter-individual variability in the number of connections with other network nodes, suggesting that the genetic model captures variations across the adolescent brains in co-opting these regions into the face network.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/63626/">Stochastic blockmodeling of the modules and core of the caenorhabditis elegans connectome</a>.
			</span><br />
			<span class="authors">
				Dragana M. Pavlovic, Petra E. V{\'e}rtes, Edward T. Bullmore, William R. Schafer, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS One Volume 9(Number 7)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap63626').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap63626" class="abstract">
			Recently, there has been much interest in the community structure or mesoscale organization of complex networks. This structure is characterised either as a set of sparsely inter-connected modules or as a highly connected core with a sparsely connected periphery. However, it is often difficult to disambiguate these two types of mesoscale structure or, indeed, to summarise the full network in terms of the relationships between its mesoscale constituents. Here, we estimate a community structure with a stochastic blockmodel approach, the Erd{\Ho}s-R{\&#39;e}nyi Mixture Model, and compare it to the much more widely used deterministic methods, such as the Louvain and Spectral algorithms. We used the Caenorhabditis elegans (C. elegans) nervous system (connectome) as a model system in which biological knowledge about each node or neuron can be used to validate the functional relevance of the communities obtained. The deterministic algorithms derived communities with 4?5 modules, defined by sparse inter-connectivity between all modules. In contrast, the stochastic Erd{\Ho}s-R{\&#39;e}nyi Mixture Model estimated a community with 9 blocks or groups which comprised a similar set of modules but also included a clearly defined core, made of 2 small groups. We show that the ?core-in-modules? decomposition of the worm brain network, estimated by the Erd{\Ho}s-R{\&#39;e}nyi Mixture Model, is more compatible with prior biological knowledge about the C. elegans nervous system than the purely modular decomposition defined deterministically. We also show that the blockmodel can be used both to generate stochastic realisations (simulations) of the biological connectome, and to compress network into a small number of super-nodes and their connectivity. We expect that the Erd{\Ho}s-R{\&#39;e}nyi Mixture Model may be useful for investigating the complex community structures in other (nervous) systems.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/59944/">Fast and accurate modelling of longitudinal and repeated measures neuroimaging data</a>.
			</span><br />
			<span class="authors">
				Bryan Guillaume, Xue Hua, Paul M. Thompson, Lourens Waldorp, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap59944').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap59944" class="abstract">
			Despite the growing importance of longitudinal data in neuroimaging, the standard analysis methods make restrictive or unrealistic assumptions (e.g., assumption of Compound Symmetry{--}the state of all equal variances and equal correlations{--}or spatially homogeneous longitudinal correlations). While some new methods have been proposed to more accurately account for such data, these methods are based on iterative algorithms that are slow and failure-prone. In this article, we propose the use of the Sandwich Estimator (SwE) method which first estimates the parameters of interest with a simple Ordinary Least Square model and second estimates variances/covariances with the ?so-called? SwE which accounts for the within-subject correlation existing in longitudinal data. Here, we introduce the SwE method in its classic form, and we review and propose several adjustments to improve its behaviour, specifically in small samples. We use intensive Monte Carlo simulations to compare all considered adjustments and isolate the best combination for neuroimaging data. We also compare the SwE method to other popular methods and demonstrate its strengths and weaknesses. Finally, we analyse a highly unbalanced longitudinal dataset from the Alzheimer&#39;s Disease Neuroimaging Initiative and demonstrate the flexibility of the SwE method to fit within- and between-subject effects in a single model. Software implementing this SwE method has been made freely available at http://warwick.ac.uk/tenichols/SwE.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/59139/">The ENIGMA Consortium : large-scale collaborative analyses of neuroimaging and genetic data</a>.
			</span><br />
			<span class="authors">
				Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Brain Imaging and Behavior Volume 8(Number 2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap59139').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap59139" class="abstract">
			The Enhancing NeuroImaging Genetics through Meta-Analysis (ENIGMA) Consortium is a collaborative network of researchers working together on a range of large-scale studies that integrate data from 70 institutions worldwide. Organized into Working Groups that tackle questions in neuroscience, genetics, and medicine, ENIGMA studies have analyzed neuroimaging data from over 12,826 subjects. In addition, data from 12,171 individuals were provided by the CHARGE consortium for replication of findings, in a total of 24,997 subjects. By meta-analyzing results from many sites, ENIGMA has detected factors that affect the brain that no individual site could detect on its own, and that require larger numbers of subjects than any individual neuroimaging study has currently collected. ENIGMA?s first project was a genome-wide association study identifying common variants in the genome associated with hippocampal volume or intracranial volume. Continuing work is exploring genetic associations with subcortical volumes (ENIGMA2) and white matter microstructure (ENIGMA-DTI). Working groups also focus on understanding how schizophrenia, bipolar illness, major depression and attention deficit/hyperactivity disorder (ADHD) affect the brain. We review the current progress of the ENIGMA Consortium, along with challenges and unexpected discoveries made on the way.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/65670/">Permutation inference for the general linear model</a>.
			</span><br />
			<span class="authors">
				Anderson M. Winkler, Gerard R. Ridgway, Matthew A. Webster, Stephen M. Smith, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap65670').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap65670" class="abstract">
			Permutation methods can provide exact control of false positives and allow the use of non-standard statistics, making only weak assumptions about the data. With the availability of fast and inexpensive computing, their main limitation would be some lack of flexibility to work with arbitrary experimental designs. In this paper we report on results on approximate permutation methods that are more flexible with respect to the experimental design and nuisance variables, and conduct detailed simulations to identify the best method for settings that are typical for imaging research scenarios. We present a generic framework for permutation inference for complex general linear models (glms) when the errors are exchangeable and/or have a symmetric distribution, and show that, even in the presence of nuisance effects, these permutation inferences are powerful while providing excellent control of false positives in a wide range of common and relevant imaging research scenarios. We also demonstrate how the inference on glm parameters, originally intended for independent data, can be used in certain special but useful cases in which independence is violated. Detailed examples of common neuroimaging applications are provided, as well as a complete algorithm ? the ?randomise? algorithm ? for permutation inference with the glm.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/65069/">Spatial analysis of dead pixels</a>.
			</span><br />
			<span class="authors">
				Julia Brettschneider, John Albert Thornby, Thomas E. Nichols, and Wilfrid S. Kendall.
			</span><br />
			<span class="venuetype"></span><span class="venue">Unknown venue (type=unpublished)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap65069').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap65069" class="abstract">
			Considering a rectangular panel of pixels arranged in a grid, we introduce a taxonomy of damages based on spatial arrangements of dysfunctional pixels. We detect these different types of damage in experimental data obtained from a detector from an X-ray machine used for additive layer manufacturing object visualisation. We model the spatial distribution of dysfunctional pixels using point pattern analysis including intensity estimation and checking for CSR. As a practical application, we propose a protocol for performance monitoring for detector panels.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2013</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/56409/">Multi-site genetic analysis of diffusion images and voxelwise heritability analysis : a pilot project of the ENIGMA?DTI working group</a>.
			</span><br />
			<span class="authors">
				Neda Jahanshad, Peter V. Kochunov, Emma Sprooten, Ren{\'e} C. Mandl, Thomas E. Nichols, Laura Almasy, John Blangero, Rachel M. Brouwer, Joanne E. Curran, Greig I. Zubicaray, Ravi Duggirala, Peter T. Fox, L. Elliot Hong, Bennett A. Landman, Nicholas G. Martin, Katie L. McMahon, Sarah E. Medland, Braxton D. Mitchell, Rene L. Olvera, Charles P. Peterson, John M. Starr, Jessika E. Sussmann, Arthur W. Toga, Joanna M. Wardlaw, Margaret J. Wright, Hilleke E. Hulshoff Pol, Mark E. Bastin, Andrew M. McIntosh, Ian J. Deary, Paul M. Thompson, and David C. Glahn.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap56409').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap56409" class="abstract">
			The ENIGMA (Enhancing NeuroImaging Genetics through Meta-Analysis) Consortium was set up to analyze brain measures and genotypes from multiple sites across the world to improve the power to detect genetic variants that influence the brain. Diffusion tensor imaging (DTI) yields quantitative measures sensitive to brain development and degeneration, and some common genetic variants may be associated with white matter integrity or connectivity. DTI measures, such as the fractional anisotropy (FA) of water diffusion, may be useful for identifying genetic variants that influence brain microstructure. However, genome-wide association studies (GWAS) require large populations to obtain sufficient power to detect and replicate significant effects, motivating a multi-site consortium effort. As part of an ENIGMA?DTI working group, we analyzed high-resolution FA images from multiple imaging sites across North America, Australia, and Europe, to address the challenge of harmonizing imaging data collected at multiple sites. Four hundred images of healthy adults aged 18?85 from four sites were used to create a template and corresponding skeletonized FA image as a common reference space. Using twin and pedigree samples of different ethnicities, we used our common template to evaluate the heritability of tract-derived FA measures. We show that our template is reliable for integrating multiple datasets by combining results through meta-analysis and unifying the data through exploratory mega-analyses. Our results may help prioritize regions of the FA map that are consistently influenced by additive genetic factors for future genetic discovery studies. Protocols and templates are publicly available at (http://enigma.loni.ucla.edu/ongoing/dti-working-group/).
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/56495/">Boosting BOLD fMRI by K-Space density weighted echo planar imaging</a>.
			</span><br />
			<span class="authors">
				Mario Zeller, Alexander M{\"u}ller, Marcel Gutberlet, Thomas E. Nichols, Dietbert Hahn, Herbert K{\"o}stler, and Andreas J. Bartsch.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS One Volume 8(Number 9)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap56495').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap56495" class="abstract">
			Functional magnetic resonance imaging (fMRI) has become a powerful and influential method to non-invasively study neuronal brain activity. For this purpose, the blood oxygenation level-dependent (BOLD) effect is most widely used. T2* weighted echo planar imaging (EPI) is BOLD sensitive and the prevailing fMRI acquisition technique. Here, we present an alternative to its standard Cartesian recordings, i.e. k-space density weighted EPI, which is expected to increase the signal-to-noise ratio in fMRI data. Based on in vitro and in vivo pilot measurements, we show that fMRI by k-space density weighted EPI is feasible and that this new acquisition technique in fact boosted spatial and temporal SNR as well as the detection of local fMRI activations. Spatial resolution, spatial response function and echo time were identical for density weighted and conventional Cartesian EPI. The signal-to-noise ratio gain of density weighting can improve activation detection and has the potential to further increase the sensitivity of fMRI investigations.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/55346/">Dynamic filtering of static dipoles in magnetoencephalography</a>.
			</span><br />
			<span class="authors">
				Alberto Sorrentino, Adam M. Johansen, John A. D. Aston, Thomas E. Nichols, and Wilfrid S. Kendall.
			</span><br />
			<span class="venuetype"></span><span class="venue">Annals of Applied Statistics Volume 7(Number 2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap55346').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap55346" class="abstract">
			We consider the problem of estimating neural activity from measurements of the magnetic fields recorded by magnetoencephalography. We exploit the temporal structure of the problem and model the neural current as a collection of evolving current dipoles, which appear and disappear, but whose locations are constant throughout their lifetime. This fully reflects the physiological interpretation of the model. In order to conduct inference under this proposed model, it was necessary to develop an algorithm based around state-of-the-art sequential Monte Carlo methods employing carefully designed importance distributions. Previous work employed a bootstrap filter and an artificial dynamic structure where dipoles performed a random walk in space, yielding nonphysical artefacts in the reconstructions; such artefacts are not observed when using the proposed model. The algorithm is validated with simulated data, in which it provided an average localisation error which is approximately half that of the bootstrap filter. An application to complex real data derived from a somatosensory experiment is presented. Assessment of model fit via marginal likelihood showed a clear preference for the proposed model and the associated reconstructions show better localisation.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2012</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/50672/">Increasing power for voxel-wise genome-wide association studies : the random field theory, least square kernel machines and fast permutation procedures</a>.
			</span><br />
			<span class="authors">
				Tian Ge, Jianfeng Feng, Derrek P. Hibar, Paul M. Thompson, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">NeuroImage Vol.63(No.2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap50672').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap50672" class="abstract">
			Imaging traits are thought to have more direct links to genetic variation than diagnostic measures based on cognitive or clinical assessments and provide a powerful substrate to examine the influence of genetics on human brains. Although imaging genetics has attracted growing attention and interest, most brain-wide genome-wide association studies focus on voxel-wise single-locus approaches, without taking advantage of the spatial information in images or combining the effect of multiple genetic variants. In this paper we present a fast implementation of voxel- and cluster-wise inferences based on the random field theory to fully use the spatial information in images. The approach is combined with a multi-locus model based on least square kernel machines to associate the joint effect of several single nucleotide polymorphisms (SNP) with imaging traits. A fast permutation procedure is also proposed which significantly reduces the number of permutations needed relative to the standard empirical method and provides accurate small p-value estimates based on parametric tail approximation. We explored the relation between 448,294 single nucleotide polymorphisms and 18,043 genes in 31,662 voxels of the entire brain across 740 elderly subjects from the Alzheimer&#39;s Disease Neuroimaging Initiative (ADNI). Structural MRI scans were analyzed using tensor-based morphometry (TBM) to compute 3D maps of regional brain volume differences compared to an average template image based on healthy elderly subjects. We find method to be more sensitive compared with voxel-wise single-locus approaches. A number of genes were identified as having significant associations with volumetric changes. The most associated gene was GRIN2B, which encodes the N-methyl-d-aspartate (NMDA) glutamate receptor NR2B subunit and affects both the parietal and temporal lobes in human brains. Its role in Alzheimer&#39;s disease has been widely acknowledged and studied, suggesting the validity of the approach. The various advantages over existing approaches indicate a great potential offered by this novel framework to detect genetic influences on human brains.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2011</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/39087/">Genetic analysis of cortical thickness and fractional anisotropy of water diffusion in the brain</a>.
			</span><br />
			<span class="authors">
				Peter Kochunov, David C. Glahn, Thomas E. Nichols, Anderson M. Winkler, Elliot L. Hong, Henry H. Holcomb, Jason L. Stein, Paul M. Thompson, Joanne E. Curran, Melanie A. Carless, Rene L. Olvera, Matthew P. Johnson, Shelley A. Cole, Valeria Kochunov, Jack Kent, and John Blangero.
			</span><br />
			<span class="venuetype"></span><span class="venue">Frontiers in Neuroscience</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap39087').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap39087" class="abstract">
			Objectives: The thickness of the brain?s cortical gray matter (GM) and the fractional anisotropy (FA) of the cerebral white matter (WM) each follow an inverted U-shape trajectory with age. The two measures are positively correlated and may be modulated by common biological mechanisms. We employed four types of genetic analyses to localize individual genes acting pleiotropically upon these phenotypes. Methods: Whole-brain and regional GM thickness and FA values were measured from high-resolution anatomical and diffusion tensor MR images collected from 712, Mexican American participants (438 females, age=47.9{$\pm$}13.2 years) recruited from 73 (9.7{$\pm$}9.3 individuals/family) large families. The significance of the correlation between two traits was estimated using a bivariate genetic correlation analysis. Localization of chromosomal regions that jointly influenced both traits was performed using whole-genome quantitative trait loci (QTL) analysis. Gene localization was performed using SNP genotyping on Illumina 1M chip and correlation with leukocyte-based gene-expression analyses. The gene-expressions were measured using the Illumina BeadChip. These data were available for 371 subjects. Results: Significant genetic correlationwas observed amongGMthickness and FA values. Significant logarithm of odds (LOD?3.0) QTLs were localized within chromosome 15q22?23. More detailed localization reported no significant association (p {\ensuremath{&lt;}}5?10?5) for 1565 SNPs located within the QTLs. Post hoc analysis indicated that 40\% of the potentially significant (p {$\leq$}10?3) SNPs were localized to the related orphan receptor alpha (RORA) and NARG2 genes. A potentially significant association was observed for the rs2456930 polymorphism reported as a significant GWAS finding in Alzheimer?s disease neuroimaging initiative subjects. The expression levels for RORA and ADAM10 genes were significantly (p {\ensuremath{&lt;}}0.05) correlated with both FA and GM thickness. NARG2 expressions were significantly correlated with GM thickness (p {\ensuremath{&lt;}}0.05) but failed to show a significant correlation (p =0.09) with FA. Discussion: This study identified a novel, significant QTL at 15q22?23. SNP correlation with gene-expression analyses indicated that RORA, NARG2, and ADAM10 jointly influenceGM thickness and WM?FA values.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/36875/">Brain network analysis : separating cost from topology using cost-integration</a>.
			</span><br />
			<span class="authors">
				Cedric E. Ginestet, Thomas E. Nichols, Edward T. Bullmore, and Andrew Simmons.
			</span><br />
			<span class="venuetype"></span><span class="venue">PLoS ONE Vol.6(No.7)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap36875').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap36875" class="abstract">
			A statistically principled way of conducting brain network analysis is still lacking. Comparison of different populations of brain networks is hard because topology is inherently dependent on wiring cost, where cost is defined as the number of edges in an unweighted graph. In this paper, we evaluate the benefits and limitations associated with using cost-integrated topological metrics. Our focus is on comparing populations of weighted undirected graphs that differ in mean association weight, using global efficiency. Our key result shows that integrating over cost is equivalent to controlling for any monotonic transformation of the weight set of a weighted graph. That is, when integrating over cost, we eliminate the differences in topology that may be due to a monotonic transformation of the weight set. Our result holds for any unweighted topological measure, and for any choice of distribution over cost levels. Cost-integration is therefore helpful in disentangling differences in cost from differences in topology. By contrast, we show that the use of the weighted version of a topological metric is generally not a valid approach to this problem. Indeed, we prove that, under weak conditions, the use of the weighted version of global efficiency is equivalent to simply comparing weighted costs. Thus, we recommend the reporting of (i) differences in weighted costs and (ii) differences in cost-integrated topological measures with respect to different distributions over the cost domain. We demonstrate the application of these techniques in a re-analysis of an fMRI working memory task. We also provide a Monte Carlo method for approximating cost-integrated topological measures. Finally, we discuss the limitations of integrating topology over cost, which may pose problems when some weights are zero, when multiplicities exist in the ranks of the weights, and when one expects subtle cost-dependent topological differences, which could be masked by cost-integration.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2009</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/708/">Impacts of Climate Change on indirect human exposure to pathogens and chemicals from agriculture</a>.
			</span><br />
			<span class="authors">
				Alistair B.A. Boxall, Anthony Hardy, Sabine Beulke, Tatiana Boucard, Laura Burgin, Peter D. Falloon, Philip M. Haygarth, Thomas Hutchinson, R. Sari Kovats, Giovanni Leonardi, Leonard S. Levy, Gordon Nichols, Simon Parsons, Laura Potts, David Stone, Edward Topp, David B. Turley, Kerry Walsh, E. M. H. Wellington, and Richard J. Williams.
			</span><br />
			<span class="venuetype"></span><span class="venue">Environmental Health Perspectives Vol.11(No.4)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap708').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap708" class="abstract">
			Objective: Climate change is likely to affect the nature of pathogens and chemicals in the environment and their fate and transport. Future risks of pathogens and chemicals could therefore be very different from those of today. In this review, we assess the implications of climate change for changes in human exposures to pathogens and chemicals in agricultural systems in the United Kingdom and discuss the subsequent effects on health impacts. Data sources: In this review, we used expert input and considered literature on climate change ; health effects resulting from exposure to pathogens and chemicals arising from agriculture ; inputs of chemicals and pathogens to agricultural systems ; and human exposure pathways for pathogens and chemicals in agricultural systems. Data synthesis: We established the current evidence base for health effects of chemicals and pathogens in the agricultural environment ; determined the potential implications of climate change on chemical and pathogen inputs in agricultural systems ; and explored the effects of climate change on environmental transport and fate of different contaminant types. We combined these data to assess the implications of climate change in terms of indirect human exposure to pathogens and chemicals in agricultural systems. We then developed recommendations on future research and policy changes to manage any adverse increases in risks. Conclusions: Overall, climate change is likely to increase human exposures to agricultural contaminants. The magnitude of the increases will be highly dependent on the contaminant type. Risks from many pathogens and particulate and particle-associated contaminants could increase significantly. These increases in exposure can, however, be managed for the most part through targeted research and policy changes.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2008</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/38355/">Inferring individual differences in fMRI : finding brain regions with significant within subject correlation</a>.
			</span><br />
			<span class="authors">
				Sumitra Purkayastha, Tor D. Wager, and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">Statistica Sinica</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap38355').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap38355" class="abstract">
			Functional magnetic resonance imaging studies answer questions about activation effects in populations of subjects. To begin with, this involves appropriate modelling of the fMRI data at the within-subject level. This is followed by extending the model to multiple subjects. There have been several attempts toward this extension, all of which have focused on inference on a single effect of interest (e.g., fMRI response for one type of working memory). However, the existing literature does not seem to say much about the relevant inferential procedures when multiple effects are of interest (e.g., response for four different types of working memory). In particular, the within subject dependence of one activation effect on another is an important issue with a multivariate repeated measures model. While most standard statistical methods regard such correlation as a nuisance, to be adjusted for and then ignored, we develop two simple and intuitive tests to make inference on the existence of such correlation. We demonstrate use of these tests by application to an fMRI study of attention switching. These tests are different not only from conventional tests for sphericity but also, more importantly, from the likelihood ratio test (LRT) of the relevant hypothesis. We also discuss what prompts us to look for tests different from the LRT.
			</div>
		</td>
	</tr>
	<tr>
		<td>
			<span class="date">
				
				<big><strong>2006</strong></big><br />
				
				
			</span>
		</td>
		<td class="publication">
			<span class="pubtitle">
				<img src="img/pdficon_small.png" width="16"
					height="16" alt="PDF" />
				<a href="http://wrap.warwick.ac.uk/38356/">Modeling and inference of multisubject fMRI data</a>.
			</span><br />
			<span class="authors">
				Jeanette A. Mumford and Thomas E. Nichols.
			</span><br />
			<span class="venuetype"></span><span class="venue">IEEE Engineering in Medicine and Biology Magazine Vol.25(No.2)</span>.
			<br />
			<span class="links">
				[<a class="dropdown-toggle" data-toggle="dropdown" href="javascript:void(0);" onclick="$('#abstract_wrap38356').slideToggle('fast');return false;">abstract</a>]
			</span>
			<div id="abstract_wrap38356" class="abstract">
			Functional magnetic resonance imaging (fMRI) is a rapidly growing technique for studying the brain in action. Since its creation [1], [2], cognitive scientists have been using fMRI to understand how we remember, manipulate, and act on information in our environment. Working with magnetic resonance physicists, statisticians, and engineers, these scientists are pushing the frontiers of knowledge of how the human brain works. The design and analysis of single-subject fMRI studies has been well described. For example, [3], chapters 10 and 11 of [4], and chapters 11 and 14 of [5] all give accessible overviews of fMRI methods for one subject. In contrast, while the appropriate manner to analyze a group of subjects has been the topic of several recent papers, we do not feel it has been covered well in introductory texts and review papers. Therefore, in this article, we bring together old and new work on so-called group modeling of fMRI data using a consistent notation to make the methods more accessible and comparable.
			</div>
		</td>
	</tr>
</tbody>
</table>
